%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% This is just an example/guide for you to refer to when submitting manuscripts to Frontiers, it is not mandatory to use Frontiers .cls files nor frontiers.tex  %
% This will only generate the Manuscript, the final article will be typeset by Frontiers after acceptance.   
%                                              %
%                                                                                                                                                         %
% When submitting your files, remember to upload this *tex file, the pdf generated with it, the *bib file (if bibliography is not within the *tex) and all the figures.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%% Version 3.3 Generated 2016/11/10 %%%
%%% You will need to have the following packages installed: datetime, fmtcount, etoolbox, fcprefix, which are normally inlcuded in WinEdt. %%%
%%% In http://www.ctan.org/ you can find the packages and how to install them, if necessary. %%%
%%%  NB logo1.jpg is required in the path in order to correctly compile front page header %%%

\documentclass[utf8]{frontiersSCNS} % for Science, Engineering and Humanities and Social Sciences articles
%\documentclass[utf8]{frontiersHLTH} % for Health articles
%\documentclass[utf8]{frontiersFPHY} % for Physics and Applied Mathematics and Statistics articles

%\setcitestyle{square} % for Physics and Applied Mathematics and Statistics articles
\usepackage{url,hyperref,lineno,microtype,subcaption}
\usepackage[onehalfspacing]{setspace}

\linenumbers


% Leave a blank line between paragraphs instead of using \\


\def\keyFont{\fontsize{8}{11}\helveticabold }
\def\firstAuthorLast{Elliott {et~al.}} %use et al only if is more than 1 author
\def\Authors{David R. Elliott\,$^{1,*}$, Andrew D. Thomas\,$^{2}$, Craig L. Strong\,$^{3}$, Helene A. Aubault\,$^{4}$, Grant H. McTainsh\,$^{5}$ and Joanna Bullard\,$^{6}$}
% Affiliations should be keyed to the author's name with superscript numbers and be listed as follows: Laboratory, Institute, Department, Organization, City, State abbreviation (USA, Canada, Australia), and Country (without detailed address information such as city zip codes or street names).
% If one of the authors has a change of address, list the new address below the correspondence details using a superscript symbol and use the same symbol to indicate the author in the author list.
\def\Address{$^{1}$Environmental Sustainability Research Centre, University of Derby, Derby, UK \\
$^{2}$Department of Geography and Earth Sciences, Aberystwyth University, Aberystwyth, UK \\
$^{3}$The Fenner School of Environment \& Society, ANU College of Sciences, The Australian National University, Canberra   ACT 2601,  Australia \\
$^{4}$TBC\\
$^{5}$Griffith School of the Environment, Griffith University, Nathan, Queensland, Australia\\
$^{6}$Department of Geography, School of Social, Political and Geographical Sciences, Loughborough University, Leicestershire, UK
}
% The Corresponding Author should be marked with an asterisk
% Provide the exact contact address (this time including street name and city zip code) and email of the corresponding author
\def\corrAuthor{David R. Elliott}

\def\corrEmail{d.r.elliott@derby.ac.uk}


\begin{document}

<<include=FALSE>>=
library(knitr)
opts_chunk$set(
concordance=TRUE
)
@


\onecolumn
\firstpage{1}

\title[Microbes and landscape stability]{Landscape Stability in Drylands Influenced by Dispersal Strategy of Soil Bacteria}

\author[\firstAuthorLast ]{\Authors} %This field will be automatically populated
\address{} %This field will be automatically populated
\correspondance{} %This field will be automatically populated

\extraAuth{}% If there are more than 1 corresponding author, comment this line and uncomment the next one.
%\extraAuth{corresponding Author2 \\ Laboratory X2, Institute X2, Department X2, Organization X2, Street X2, City X2 , State XX2 (only USA, Canada and Australia), Zip Code2, X2 Country X2, email2@uni2.edu}


\maketitle


\begin{abstract}

%%% Leave the Abstract empty if your article does not require one, please see the Summary Table for full details.
\section{}
Dryland soils are particularly vulnerable to erosion because of low plant cover and low organic matter which is linked with water availability and biological constraints. It is increasingly recognised that dryland soils can be managed to minimise erosion, which has benefits such as increasing soil fertility, carbon storage, and prevention of dust storms. Management may include monitoring, reduction of trampling, and planting. Interventions can help maintain the integrity of biological soil crusts (biocrusts), which are ubiquitous microbial communities living in the top few millimetres of dryland soils. Biocrusts perform numerous functions, including adhesion of particles which protects the soil from erosion and is widely regarded as beneficial. Biocrusts also fix carbon and nitrogen from the air, enriching the soil and resulting in production of biomass. This living biomass is subject to natural selection pressures, and the soil-surface microbial community is expected to be adapted for life in this complex environment rather than being transient or opportunistic. In fact, biocrusts probably colonised the land long before plants, so we suggest that their survival and dispersal strategies are exquisitely adapted, and that they have had an ever-present influence on geomorphological processes for billions of years. 

Whilst it is recognised that biocrusts stabilise soils and that certain cyanobacteria contribute significantly to this, there has been little work to practically verify identities and ecological strategies of stabilising organisms. Biological adaptations for dispersal by wind, water, or vegetative growth are expected, and such adaptations are considered likely to differently influence geomorphic processes and ecosystem function. Adhesive adaptations may contribute to landscape stability, whilst adaptations for wind dispersal may provide a mechanism for nutrient transportation and biogenic dust production.

To investigate biological dispersal and related geomorphic processes associated with biocrust organisms, we used DNA sequencing to characterise microbial communities of various biocrusts and sediments in Diamantina National Park of western Queensland, Australia (23°36’44.8”S; 143°17’46.9”E). Climate in the region is semi-arid characterised by a summer-dominant rainfall pattern with a mean annual precipitation of 270 mm a$^{-1}$ and high inter-annual variation. Central to the study site is 25 Km$^{2}$ of erosion active claypan, bordered by sand dunes and a river. Aeolian activity constantly moves sediment from the dunes and claypan, whilst periodic flooding (interval approx. 3 years) brings fresh river sediment and biological inoculum to the claypan.


\tiny
 \keyFont{ \section{Keywords:} soil, Aeolian, dryland, biocrust, bacteria, microbe, wind, fluvial} %All article types: you may provide up to 8 keywords; at least 5 are mandatory.
\end{abstract}

\section{Introduction}

<<setup, include=FALSE, cache=FALSE>>=
options(width = 120)
library(phyloseq)
library("ggplot2")
library(plyr)
library(vegan)
library(grid)
library(reshape2)
library(scales)   # needed for formatting y-axis labels to non-scientific type
library(RColorBrewer)
library("DESeq2")
library(knitr)
# set global chunk options
opts_chunk$set(fig.path='figure/minimal-', fig.align='center', fig.show='hold', message=FALSE)
options(formatR.arrow=TRUE,width=90)
set.seed(42)

theme_set(theme_bw())

# setup grid
g0 <- grid.layout(nrow=2, ncol=1)
vp.01 <- viewport(layout.pos.col=1, layout.pos.row=1) 
vp.02 <- viewport(layout.pos.col=1, layout.pos.row=2,)

gl <- grid.layout(nrow=1, ncol=2)
vp.1 <- viewport(layout.pos.col=1, layout.pos.row=1) 
vp.2 <- viewport(layout.pos.col=2, layout.pos.row=1)
@

<<load,echo=FALSE,message=FALSE, warning=FALSE, cache=TRUE>>=
bac <- import_biom("data/otu_table_mc2_w_tax.biom", "data/rep_set.tre",parseFunction = parse_taxonomy_greengenes)

map <- import_qiime(mapfilename = "data/map.txt")
bac <- merge_phyloseq(bac,map)

# fix incorrect spelling of nebkha in the biom file.
levels(sample_data(bac)$zone)[levels(sample_data(bac)$zone)=="nebka"] <- "nebkha"

# remove chloroplast
  bac <- subset_taxa(bac,Class!="Chloroplast")

  # standardise unknown taxon entries
tax_table(bac)[is.na(tax_table(bac))] <- "Unknown"
tax_table(bac)[tax_table(bac) %in% c("", "unclassified", "Unclassified")] <- "Unknown"

# keep combined kingdom objects
expt.combined <- bac

# separate archaea and bacteria into different objects
arc <- subset_taxa(bac,Kingdom=="Archaea")
bac <- subset_taxa(bac,Kingdom=="Bacteria" )

expt.deseq <- subset_samples(bac,zone=="dune flank")
expt.deseq = prune_taxa(taxa_sums(expt.deseq) > 0, expt.deseq)
# add 1 to the whole OTU table, to avoid infinity problems in deseq analysis
# this will reduce the fold changes
otu_table(expt.deseq) <- otu_table(expt.deseq)+1
  
# convert OTU observations into relative abundance per sample
arc.rel <- transform_sample_counts(arc, function(x) 100*x/(sum(x)))
bac.rel <- transform_sample_counts(bac, function(x) 100*x/(sum(x)))
expt.combined.rel <- merge_phyloseq(otu_table(bac.rel),tax_table(bac.rel),otu_table(arc.rel),tax_table(arc.rel),sample_data(bac.rel))

expt.combined.rel.phylum <- tax_glom(expt.combined.rel,taxrank = "Phylum")
cyano.rel <- subset_taxa(bac.rel,Phylum=="Cyanobacteria")

# Identify top phyla and cyano genera - so they can be selectively plotted/analysed
top.phyla = names(sort(tapply(taxa_sums(bac.rel), tax_table(bac.rel)[, "Phylum"], sum), TRUE)[1:10])
top.cyano.genus = names(sort(tapply(taxa_sums(cyano.rel), tax_table(cyano.rel)[, "Genus"], sum), TRUE)[1:6])
top.cyano.genus <- top.cyano.genus[top.cyano.genus!="Unknown"]

# keep only the top phyla, but retain all in .all object for when needed
bac.rel.all <- bac.rel
bac.rel <- subset_taxa(bac.rel, Phylum%in%top.phyla)

# agglomerate cyanos by genus, used later on
cyano.rel.g <- tax_glom(cyano.rel,taxrank = "Genus")

# simplify data (speed up processing). If this isn't done it causes long calculation time and problems viewing the PDF. Need to check exactly which outputs this is affecting and whether it matters.
bac.rel <- prune_taxa(taxa_sums(bac.rel) > 0.2, bac.rel) 
#bac.rel <- prune_taxa(taxa_sums(bac.rel) > 0.1, bac.rel) 

# To simplify plots
# Remove phyla not seen at more than 1 % in at least 10% of the samples.
expt.combined.rel.phylum.rr = filter_taxa(expt.combined.rel.phylum, function(x) sum(x > 1) > (0.1*length(x)), TRUE)

# Save a .csv of all the phylum abundances
#phylumdf.all = psmelt(tax_glom(expt.combined.rel.phylum,taxrank = "Phylum"))
#write.csv(phylumdf.all,"phylum_breakdown.csv")

expt.combined.rel.phylum.merged <- merge_samples(subset_samples(expt.combined.rel.phylum,element!="air"),"zone")
otu_table(expt.combined.rel.phylum.merged) <- otu_table(expt.combined.rel.phylum.merged)/5

tp <- t(otu_table(expt.combined.rel.phylum.merged))
tp <- as.data.frame(tp)
phyla.df <- as.data.frame(cbind(tax_table(expt.combined.rel.phylum.merged),tp))
phyla.order <- taxa_names(expt.combined.rel.phylum.merged)[order(taxa_sums(expt.combined.rel.phylum.merged),decreasing = T)]
phyla.df <- phyla.df[phyla.order,]
write.csv(phyla.df[ ,c(rank_names(bac)[1:2],sample_names(expt.combined.rel.phylum.merged))],"phyla.csv")

cyano.rel.g.merged <- merge_samples(subset_samples(cyano.rel.g,element!="air"),"zone")
otu_table(cyano.rel.g.merged) <- otu_table(cyano.rel.g.merged)/5
tc <- t(otu_table(cyano.rel.g.merged))
tc <- as.data.frame(tc)
cyano.df <- as.data.frame(cbind(tax_table(cyano.rel.g.merged),tc))
cyano.order <- taxa_names(cyano.rel.g.merged)[order(taxa_sums(cyano.rel.g.merged),decreasing = T)]
cyano.df <- cyano.df[cyano.order,]
write.csv(cyano.df[ ,c(rank_names(bac)[1:6],sample_names(cyano.rel.g.merged))],"cyano.csv")

bac.earthair <- subset_samples(bac,element!="water")
arc.earthair <- subset_samples(arc,element!="water")
ab.earthair <- subset_samples(expt.combined,element!="water")
@

<<analyse1,echo=FALSE,message=FALSE, results='hide', warning=FALSE, cache=TRUE>>=
# plot phyla (combined archea, bacteria, and fungi)

rich.ab <- plot_richness(expt.combined,measures = "Chao1",color = "zone", shape="element",x="zone") + geom_point(size=5,alpha=0.7) 
ord.ab <- ordinate(ab.earthair, "NMDS", "bray")
NMDS.ab <- plot_ordination(ab.earthair, ord.ab, type = "sites", color = "zone", shape="element") + geom_point(size=5,alpha=0.7) + theme(aspect.ratio=1) + theme(legend.position="right")

m_reads_total <- sum(get_variable(bac,'n_reads'))/1000000
m_reads_bac_qc <- sum(otu_table(bac))/1000000
m_reads_arc_qc <- sum(otu_table(arc))/1000000

@

<<locations, echo=FALSE, message=FALSE, results='hide', warning=FALSE, eval=TRUE, cache=TRUE>>=
library(ggmap)
library(sp)
library(rgdal) # needed by sp
library(RColorBrewer)
# Coordinates conversion to locations on the Earth in the Universal Transverse Mercator (UTM) coordinate system using the sp (spatial points) package. 

# Calculate spatial points based on the lat/lon information
# based on http://grokbase.com/t/r/r-sig-geo/136h9k8e9y/lat-long-utm-again

samples_latlon <- sample_data(bac)[,c("lat","lon")]
samples_latlon <- data.frame(samples_latlon)
#convert to sp object:
coordinates(samples_latlon) <- ~ lon + lat #longitude first

#adding a coordinate reference system:
proj4string(samples_latlon) <- CRS("+proj=longlat +ellps=WGS84 +datum=WGS84")

#project using sp transform:
UTM <- spTransform(samples_latlon, CRS=CRS("+proj=utm +zone=53 +ellps=WGS84 +datum=WGS84 +units=m +no_defs +towgs84=0,0,0"))

# Now get the spatial points back into the main dataframe:
UTM.df <- as.data.frame(UTM)
UTM.df$easting <- UTM.df$lon
UTM.df$northing <- UTM.df$lat

sample_data(bac) <- cbind(sample_data(bac),UTM.df[,c("easting","northing")])
@

<<map2,echo=FALSE, message=FALSE, results='hide', warning=FALSE,eval=TRUE, cache=TRUE>>=

#Create a custom color scale
myColors <- brewer.pal(7,"Set1")[c(2,1,7,5,3)]
names(myColors) <- levels(get_variable(bac,"zone"))
colScale <- scale_colour_manual(name = "zone",values = myColors)

# plot using ggmap. This just uses lat/lon, not the spatial points.

loc <- c(lon = mean(get_variable(bac,"lon")), lat = mean(get_variable(bac,"lat")))
locZoom <- c(lon = mean(get_variable(bac,"lon")[get_variable(bac,"zone")!="channel"]), lat = mean(get_variable(bac,"lat")[get_variable(bac,"zone")!="channel"]))
locZoom2 <- c(lon = mean(get_variable(bac,"lon")[1:2]), lat = mean(get_variable(bac,"lat")[1:2] ))

locZoom2 <- c(lon = mean(samples_latlon$lon["dune4"]), lat = mean(samples_latlon$lat["nebka4"] ))

dMap <- get_map(location=loc, source="google", maptype = "satellite", zoom = 11)
dMapZoom <- get_map(location=locZoom, source="google", maptype = "satellite", zoom = 14)
dMapZoom2 <- get_map(location=locZoom2, source="google", maptype = "satellite", zoom = 17)

basemap <- ggmap(dMap)
basemapZoom <- ggmap(dMapZoom)
basemapZoom2 <- ggmap(dMapZoom2)

map1 <- basemap + geom_point(aes(x = lon, y = lat, colour=zone,shape=element),alpha=0.5, data = sample_data(bac), size = 4) + colScale

bac.zoommap <- sample_data(subset_samples(physeq = bac,zone!="channel"))
map2 <- basemapZoom + geom_point(aes(x = lon, y = lat, colour=zone,shape=element),alpha=0.5, data = bac.zoommap, size = 4) + geom_text(aes(label=transect),size=2.5, data = bac.zoommap) + colScale

# higher zoom map for Lyell meeting
map3 <- basemapZoom2 + geom_point(aes(x = lon, y = lat, colour=zone,shape=element),alpha=0.5, data = bac.zoommap, size = 8) + geom_text(aes(label=transect),size=4, data = bac.zoommap) + colScale

@

<<hyp_analysis,echo=FALSE, results='hide', cache=TRUE>>=
# no output from this block but kept for now as it calculates some useful results/objects.
# Might be suitable for output as small table or in text mention.

channel <- subset_samples(bac.rel,zone=="channel")
dune <- subset_samples(bac.rel,element=="air")
pan <- subset_samples(bac.rel,zone=="pan")

channel <- prune_taxa(taxa_sums(channel)>0,channel)
dune <- prune_taxa(taxa_sums(dune)>0,dune)
pan <- prune_taxa(taxa_sums(pan)>0,pan)

channel.ntaxa <- ntaxa(channel)
dune.ntaxa <- ntaxa(dune)
pan.ntaxa <- ntaxa(pan)

channel_dune.intersection <- intersect(taxa_names(channel),taxa_names(dune))
channel_dune.intersection.n <- length(channel_dune.intersection)  

channel_pan.intersection <- intersect(taxa_names(channel),taxa_names(pan))
channel_pan.intersection.n <- length(channel_pan.intersection)  

dune_pan.intersection <- intersect(taxa_names(dune),taxa_names(pan))
dune_pan.intersection.n <- length(dune_pan.intersection)  

all.intersection <- intersect(channel_dune.intersection, channel_pan.intersection)
all.intersection <- length(all.intersection)

# Now we have the intersections
# make objects for the pairs and subset to just the intersecting taxa
channel_pan <- subset_samples(bac.rel,zone%in%c("channel","pan"))
channel_pan <- prune_taxa(channel_pan.intersection, channel_pan)
otu_table(channel_pan) <- otu_table(channel_pan)/5

channel_dune <- subset_samples(bac.rel,zone%in%c("channel","dune"))
channel_dune <- prune_taxa(channel_dune.intersection, channel_dune)
otu_table(channel_dune) <- otu_table(channel_dune)/5

# create a common colour palette for plotting
# not used now but might be useful later
#allphyla <- unique(c(tax_table(channel_dune)[,"Phylum"],tax_table(channel_pan)[,"Phylum"]))
#allphyla.col <- as.vector(colorRampPalette(brewer.pal(12,"Set3"))(length(allphyla)))
#names(allphyla.col)  <- allphyla

# plot
#channel_pan.shared <- plot_bar(channel_pan,x = "zone",fill = "Phylum") + geom_bar(aes(color=Phylum, fill=Phylum), stat="identity", position="stack") + scale_y_continuous(name="relative abundance (%)") + scale_fill_manual("Phylum", values = allphyla.col) + scale_colour_manual("Phylum", values = allphyla.col)

# work out OTU abundance for those only in soil
bac.rel.earth <- subset_samples(bac.rel,element=="earth")
bac.rel.earth.merged <- merge_samples(bac.rel.earth,rep(T,nsamples(bac.rel.earth)))
otu_table(bac.rel.earth.merged) <- otu_table(bac.rel.earth.merged)/nsamples(bac.rel.earth)
inoc_source_otus <- unique(c(taxa_names(channel),taxa_names(dune)))

soil_only_otus.sum <- round(sum(otu_table(bac.rel.earth.merged)) -  sum(otu_table(bac.rel.earth.merged)[,inoc_source_otus]),1)

@

<<hyp_analysis2,echo=FALSE, results='hide', cache=TRUE>>=
# working with merged objects to reduce processing time
bac.rel.merged <- merge_samples(subset_samples(bac.rel,element!="air"),"zone")
otu_table(bac.rel.merged) <- otu_table(bac.rel.merged)/5
bac.rel.merged.phylum <- tax_glom(bac.rel.merged,"Phylum")

# make dataframes for output later
t <- t(otu_table(bac.rel.merged))
t <- as.data.frame(t)
otus.df <- as.data.frame(cbind(tax_table(bac.rel.merged),t))
otus.order <- taxa_names(bac.rel.merged)[order(taxa_sums(bac.rel.merged),decreasing = T)]
otus.df <- otus.df[otus.order,]
# more info will be added to this DF after DESEQ analysis, then it will be output

# Keep only phyla that are > 1% overall:
# to simplify plot and protect against unequal sampling bias

keep_phyla <- (taxa_sums(bac.rel.merged.phylum)/5)>1
keep_phyla <- as.character(tax_table(bac.rel.merged.phylum)[keep_phyla,"Phylum"])
keep_phyla_index <- tax_table(bac.rel.merged)[,"Phylum"]%in%keep_phyla

bac.rel.merged <- subset_taxa(bac.rel.merged, keep_phyla_index)
sample_data(bac.rel.merged)$zone <- as.factor(sample_names(bac.rel.merged))

airborne <- subset_samples(bac.rel,element=="air")
airborne <- prune_taxa(taxa_sums(airborne)>0,airborne)
waterborne <- subset_samples(bac.rel.merged,zone=="channel")
waterborne <- prune_taxa(taxa_sums(waterborne)>0,waterborne)

airborne_otus <- taxa_names(airborne)[taxa_sums(airborne)>0]
waterborne_otus <- taxa_names(waterborne)[taxa_sums(waterborne)>0]
both_otus <- intersect(airborne_otus,waterborne_otus)

# make an object to be modified for plots
bac.rel.ea <- subset_samples(bac.rel,element!="air")
otu_table(bac.rel.ea) <- otu_table(bac.rel.ea)/5

# keep only the more abundant phyla (for clarity of plots)
# this now done globally so shouldn't be needed here
# bac.rel.ea.p <- subset_taxa(bac.rel.ea, Phylum%in%top.phyla)
bac.rel.ea.p <- bac.rel.ea

# same again but looking at cyanobacterial genera
bac.rel.ea.cyano <- subset_taxa(bac.rel.ea, Phylum == "Cyanobacteria")
# select only the more abundant cyanobacterial genera
bac.rel.ea.cyano <- subset_taxa(bac.rel.ea.cyano, Genus%in%top.cyano.genus)

airborne_otus <- airborne_otus[airborne_otus%in%taxa_names(bac.rel.ea)]
waterborne_otus <- waterborne_otus[waterborne_otus%in%taxa_names(bac.rel.ea)]
both_otus <- both_otus[both_otus%in%taxa_names(bac.rel.ea)]

@

<<deseq_analysis1, echo=FALSE, message=FALSE, results='hide', cache=TRUE>>=

  typedds = phyloseq_to_deseq2(expt.deseq, ~ element)
  typedds = DESeq(typedds, test="Wald",fitType="parametric")
  # DESeq docs:
  #On p-values:
  # By default, independent filtering is performed to select a set of genes for multiple test correction which will optimize the number of adjusted p-values less than a given critical value alpha (by default 0.1). The adjusted p-values for the genes which do not pass the filter threshold are set to NA. 
  
# When using local regression fit, the user should examine plotDispEsts(dds)
# to make sure the fitted line is not sharply curving up or down based on
# the position of individual points.

  #  plotDispEsts(typedds,main="model check")

  res = results(typedds, cooksCutoff = FALSE, alpha = 0.05)
  alpha = 0.05
  sigtab = res

  rel.abundance <- apply(otu_table(bac)[rownames(sigtab), ],FUN = max,MARGIN = 1)
  sigtab = cbind(as(sigtab, "data.frame"), as(tax_table(expt.deseq)[rownames(sigtab), ], "matrix"))
  sigtab$max <- rel.abundance
  
  # record the OTU name
  sigtab$otu <- row.names(sigtab)
  
  ind <- sigtab$padj<0.05&!is.na(sigtab$padj)
  sigtab$otu.orig <- sigtab$otu
  sigtab[ind,"otu2"] <- gsub(pattern = "OTU_",replacement = "",x = sigtab[ind,"otu"])
  sigtab[ind,"otu"] <- paste("*",sigtab[ind,"otu"],"*",sep="")

  sigtab$sig <- ""
  sigtab[ind,"sig"] <- "*"
  
  sd <- as.data.frame(sample_data(expt.deseq))

# manipulations to the sigtab table - for later output as graphics and tables

wanted_cols <- c("Phylum",  "Class",  "Order",	"Family",	"Genus",	"Species","padj","log2FoldChange")
wanted_rows <- (sigtab$padj<0.05) & (!is.na(sigtab$padj))
wanted_otus <- sigtab[wanted_rows,"otu.orig"]

expt.rel.df <- subset_samples(bac.rel.all, zone=="dune flank")
expt.rel.df.merged <- merge_samples(expt.rel.df,group = "element")
otus.rel.df.by.type <- otu_table(expt.rel.df.merged)/5

orbt <- as.data.frame(t(otus.rel.df.by.type))
sigtab.subset <- sigtab[wanted_rows,wanted_cols]
sigtab.subset <- cbind(sigtab.subset[wanted_otus,],orbt[wanted_otus,])
# add a difference column
sigtab.subset$diff <- sigtab.subset$earth - sigtab.subset$air

# Sanity check on the significant OTUs
sane <- sign(sigtab.subset$log2FoldChange)==(sign(sigtab.subset$diff))

# identify the removed OTUs (none)
# sigtab.subset.removed <- sigtab.subset[!sane,]

# remove the OTUs not passing sanity check (none)
# sigtab.subset <- sigtab.subset[sane,]

# order by diff
sigtab.subset <- sigtab.subset[order(sigtab.subset$diff,decreasing = T),] 
# save to file
write.csv(sigtab.subset,"windtunnel_otus.csv")

outcols <- c("Phylum",		"Family",	"Genus","padj","air",	"earth") #"Class",	"Order",
outtab <- sigtab.subset[1:10,outcols]
outtab2 <- sigtab.subset[nrow(sigtab.subset):(nrow(sigtab.subset)-10),outcols]
# there are NAs in outtab2 because of something wrong in working out the mean abundances
# as can be seen by this line
# otu_table(bac)["4397479"]
# otu_table(expt.rel.df.merged)["4397479"]
# there are observations for filter but it comes out as 0 in the table.
# This is just because the OTU "4397479" is present in bac used for deseq analysis, but not in the subset object.

otus_earth <- row.names(sigtab.subset[sigtab.subset$diff>0,])
otus_air <- row.names(sigtab.subset[sigtab.subset$diff<0,])

n_otu_earth <- length(otus_earth)
n_otu_air <- length(otus_air)

# record significant OTUs in otus.df for output file / table
otus.df$sig <- NA
otus.df[otus_earth,"sig"] <- "soil"
otus.df[otus_air,"sig"] <- "air"
@


<<source_analysis1, echo=FALSE, message=FALSE, cache=TRUE>>=

colnames(tax_table(bac.rel.ea.p))[1] <- "source"
tax_table(bac.rel.ea.p)[,"source"] <- "not found in river"
tax_table(bac.rel.ea.p)[taxa_names(bac.rel.ea.p)%in%waterborne_otus,"source"] <- "found in river"

water_otus_plot <- plot_bar(bac.rel.ea.p,x = "zone",fill = "source")
water_otus_plot <- water_otus_plot + geom_bar(aes(color=source, fill=source), stat="identity", position="stack") 
water_otus_plot <- water_otus_plot + scale_y_continuous(name="relative abundance (%)") + scale_fill_brewer(palette = "Set1",direction = -1) + scale_color_brewer(palette = "Set1",direction = -1)
water_otus_plot <- water_otus_plot + facet_wrap(~Phylum, nrow=2) + theme(legend.position="bottom")

###

colnames(tax_table(bac.rel.ea.cyano))[1] <- "source"
tax_table(bac.rel.ea.cyano)[,"source"] <- "not found in river"
tax_table(bac.rel.ea.cyano)[taxa_names(bac.rel.ea.cyano)%in%waterborne_otus,"source"] <- "found in river"

water_otus_plot.c <- plot_bar(bac.rel.ea.cyano,x = "zone",fill = "source")
water_otus_plot.c <- water_otus_plot.c + geom_bar(aes(color=source, fill=source), stat="identity", position="stack") 
water_otus_plot.c <- water_otus_plot.c + scale_y_continuous(name="relative abundance (%)") + scale_fill_brewer(palette = "Set1",direction = -1) + scale_color_brewer(palette = "Set1",direction = -1)
water_otus_plot.c <- water_otus_plot.c + facet_wrap(~Genus, nrow=1, scales = "free") + theme(legend.position="bottom")

@

<<deseq_analysis2, echo=FALSE, message=FALSE, cache=TRUE>>=
# see whether OTU is easy or not to become airborne by wind tunnel treatment
# this is used later to make the wind_otus_plot
# using object bac.rel.ea.p - which has only the more abundant phyla

colnames(tax_table(bac.rel.ea.p))[1] <- "behaviour"
tax_table(bac.rel.ea.p)[,"behaviour"] <- "intermediate"
tax_table(bac.rel.ea.p)[row.names(tax_table(bac.rel.ea.p))%in%otus_air,"behaviour"] <- "easily becomes airborne"
tax_table(bac.rel.ea.p)[row.names(tax_table(bac.rel.ea.p))%in%otus_earth,"behaviour"] <- "resists becoming airborne"

colnames(tax_table(bac.rel.ea.cyano))[1] <- "behaviour"
tax_table(bac.rel.ea.cyano)[,"behaviour"] <- "intermediate"
tax_table(bac.rel.ea.cyano)[row.names(tax_table(bac.rel.ea.cyano))%in%otus_air,"behaviour"] <- "easily becomes airborne"
tax_table(bac.rel.ea.cyano)[row.names(tax_table(bac.rel.ea.cyano))%in%otus_earth,"behaviour"] <- "resists becoming airborne"

deseq_n_test <- nrow(sigtab)

deseq_sum_diff_air <- sum(sigtab.subset[sigtab.subset$diff<0,"diff"] )
deseq_sum_diff_earth <- sum(sigtab.subset[sigtab.subset$diff>0,"diff"] )
deseq_n_diff_air <- length(sigtab.subset[sigtab.subset$diff<0,"diff"] )
deseq_n_diff_earth <- length(sigtab.subset[sigtab.subset$diff>0,"diff"] )

@

<<source_analysis2, echo=FALSE, message=FALSE, cache=TRUE>>=
###

wind_otus_plot <- plot_bar(bac.rel.ea.p,x = "zone",fill = "behaviour")
wind_otus_plot <- wind_otus_plot + geom_bar(aes(color=behaviour, fill=behaviour), stat="identity", position="stack") 
wind_otus_plot <- wind_otus_plot + scale_y_continuous(name="relative abundance (%)") + scale_fill_brewer(palette = "Accent") + scale_color_brewer(palette = "Accent")
wind_otus_plot <- wind_otus_plot + facet_wrap(~Phylum, nrow=2) + theme(legend.position="bottom")

###

wind_otus_plot.c <- plot_bar(bac.rel.ea.cyano,x = "zone",fill = "behaviour")
wind_otus_plot.c <- wind_otus_plot.c + geom_bar(aes(color=behaviour, fill=behaviour), stat="identity", position="stack") 
wind_otus_plot.c <- wind_otus_plot.c + scale_y_continuous(name="relative abundance (%)") + scale_fill_brewer(palette = "Accent") + scale_color_brewer(palette = "Accent")
wind_otus_plot.c <- wind_otus_plot.c + facet_wrap(~Genus, nrow=1, scales = "free") + theme(legend.position="bottom")
@


The role of microbes in landscape scale processes is rarely acknowledged, yet there is mounting evidence that microscopic players have profound geomorphic influence \citep[e.g.][]{beraldi2013early,ferrenberg2017biological}. Pressures of climate change and land use change coupled with growing human populations require informed landscape management strategies to protect resources and minimise adverse impacts. Currently such strategies include actions such as controlling grazing and vegetation, but rarely are microbial players actively managed. We suggest that this is simply because the actions of microbes are less easily regognised, and that recent advances in molecular microbial ecology provide an opportunity to address this.

In drylands where vegetation is sparse, soil microbes compensate by providing some ecosystem functions that are normally dominated by plants in temperate systems. Microbes in the soil surface of drylands form biological soil crusts (biocrusts) which contribute to stabilisation of the soil, fixing of carbon and nitrogen, and modulation of hydrology \citep[e.g.][]{ferrenberg2017biological}. These biotic processes are of course inter-linked, and they are the direct result of microbial community physiology, ecology, and evolution over billions of years. Biocrusts are likely among the first organisms to colonise the land \citep{beraldi2013early}, therefore it is to be expected that they are well adapted to both exploit soil surface conditions, and modify their environment for their own advantage. This process of ecosystem engineering is widely recognised in larger organisms such as beavers \citep{jones1994organisms}, and although microbial players are less visible they certainly do modify the environment at landscape scale \citep[e.g][]{elliott2009dynamic}. By understanding biocrust ecology, we can gain insight into the biotic forces that influence geomorphic processes.

Biocrusts have been estimated to cover approximtely 50 \% of the soil surfaces in drylands \citep{garcia2003estimates}, where the relative lack of plants means that the role of microbes in environmental processes is likely to be proportionally higher than in termperate systems. Considering that drylands account for 45 \% of the Earth land area \citep{pruavualie2016drylands}, and this is expected to expand in the context of climate change scenarios \citep{huang2016accelerated} it seems crucial that understanding the most poorly characterised cover type on Earth, biocrusts (Ferrenberg et al., 2017) be improved. In particular it is important to increase understanding of the relationship of these organisms to stability of the landscape.”

%% section moved from here and integrated (reduced) further down to aims

%% old version: Landscape stability and dust production are of the utmost importance to communities in drylands and neighbouring areas who face hazards such as dust storms and mobile sand dunes in addition to suffering erosion which degrades the utility of land (REF). Specifically in central Australia ...\footnote{Craig/Grant please could you add something here to explain the hazards of dust etc. of relevance to the field site, including an introduction to crusts and how they are managed (if at all)}. Further afield from dust sources there are continental scale impacts including the movement of sediment which is of geomorphic significance, and the movement of nutrients which can have effects on the biota of water bodies in particular \citep{beraldi2013early}.

Landscape stability and dust production are of the utmost importance to communities in drylands and neighbouring areas who face hazards such as dust storms and mobile sand dunes. An unstable landscape is one where soil surface protection is no longer in place. Loss of vegetation, biocrusts and soil surface disturbance increases the susceptibility to wind or water erosion (Webb and Strong, 2011). Loss of soil at the site of erosion degrades the soil physical and chemical properties \citep{leys1994soil,leys2002erosion}. Transport of dust can result in downwind impacts to other terrestrial \citep{kohfeld2001dirtmap,reheis1995quaternary,herrmann1996identification,cattle2002aeolian}, aquatic \citep{leys1999dust,beraldi2013early}, marine \citep{Jickells2001atmospheric,shinn2000african} and human populations \citep{tozer2013dust,huszar1986estimating,miri2009environmental,saxton1995wind}. Further afield from dust sources there are continental scale impacts including the movement of sediment which is of geomorphic significance, and the movement of nutrients which can have effects on the biota of water bodies in particular \citep{beraldi2013early}.

% aims moved from here to further down

% section moved from here to methods 

The extent to which “microscopic ecosystems engineers” contribute to the geomorphic process and importantly if they rely on the geomorphology to drive their existence is explored further in this paper.  Landscape processes in many drylands is driven by the interplay between fluvial and Aeolian processes \citep{mctainsh1987desert,bullard2003aeolian,field2009toward,langford1989fluvial}. citet{mctainsh1987desert} proposed a model of sediment recycling that provided an integrated interpretation of these processes in the landscape. Rivers feeding the internal draining river basin such as the Lake Chad and Lake Eyre transport vast amounts of fine sediments feeding the large dust source regions out of Africa, China and Australia. By linking river systems and wind erosion, McTainsh proposed looking at arid geomorphology as a system of sediment recycling operating on a massive scale. 

Sediment recycling results in a constantly changing soil surface. A conceptual model (Figure \ref{fig:dispersal-hazards}) highlights the interaction between the fluvial and aeolian systems. Floods supply fresh sediment to the claypan surface, having the effect of ‘re-zeroing’ the claypan surface, burying the effects of previous wind erosion activity. Once dry, however, time and climatic variation impact the sediment creating a mosaic of surface types. Source boarding sand dunes provide new sediments, larger, heavier particles that have the capacity to break down fine crusts as they bounce across the claypan. The ‘living soils’—the cyanobacteria crusts whilst known to be very responsive to changes in climatic stimulus, and therefore change the soil surface layer through time, have not generally been considered as a component of this fluvial - aeolian interaction. \footnote{need to develop model further}

%% earlier version: A simplified model of disperal strategies and hazards for biocrust microbiota at the field site is presented in Figure \ref{fig:dispersal-hazards}, showing key features of the three system compartments: sand dune; clay pan; and river. Most of the time the region is dry and the soil surface is covered by biological soil crusts, the biota of which are mostly unknown although the phototrophs in the area have been identified by Strong et al... (presumably - please advise). On rare occasions the claypan is flooded by water carried by rivers from areas to the north which experience more rainfall. This environment presents considerable challenges to life at the soil surface, and these vary due to the different conditions in each compartment of the system. However, these compartments are linked by the actions of wind and water which can transfer sediment and inoculum (Figure \ref{fig:dispersal-hazards}), thus the system is highly dynamic and there is potential for microbes to be adapted either as specialists exploiting one compartment or generalists able to exploit multiple aspects of the same system. 

\begin{figure}
  \centering
      \includegraphics[width=1\textwidth]{img/dispersal-hazards-fig}
  \caption{Photographs of main system compartments with identification of key hazards for microbial life and illustration of expected linkages by Aeolian and hydrological transfer of sediment and inoculum. In our sampling the sand dune flank samples belong to the sand dune compartment and the nebkha samples belong to the claypan compartment - these are expected to be subject to the same respective hazards and linkages.}
  \label{fig:dispersal-hazards}
\end{figure}

Understanding erodibility, and hence sediment entrainment, within the claypan provides important insights into the triggers that occur through time responsible for landscape stability, and also the crucial triggers of cyanobacterial filament entrainment and possible transportation. At a broad scale, this paper seeks to identify if the microbial ecology of a dynamic dryland ecosystem in central Australia spatially structure differently according to the various geomorphic drivers and sediment types interacting within this field setting. 

A small number of previous studies have been published on the microbial content of airborne sediments, and there was recently a review on the subject by \citet{acosta2015microbiology}. This review pointed out the fact that most previous work has focused on long-distance transport and the movement of pathogens, which are quite different subjects compared to the present work. The only previous study to our knowledge combining use of a wind tunnel with collection of airborne particles and their microbial charaterisation is \citet{gardner2012pyrosequencing}, which used a pyrosequencing approach to determine the association of particular microbes or diversity metrics with organic matter and particle size fractions. Prior to wind-tunnel treatments the soils in \citet{gardner2012pyrosequencing} were tilled, raked, and flattened using a lawn roller, in contrast to the present study which tested soils in their natural condition. 

Therefore by linking the dust source explicitly to soils of known microbial composition, we are able to show for the first time definitive evidence identifying taxa that are mobilised by moderate wind from dryland soils, and interpret these findings in the wider geomorphic context.

To increase understanding of the relationship of biocrusts to landscape stability we aimed to firstly gain basic knowledge about the microbial composition of different dryland soil types, and secondly to investigate the possible ecological strategies of microbes present. By understanding the ecology of communities and their constituent individual taxa, it should be possible to predict responses for various scenarios and devise interventions to manipulate the biotic component of geomorphic processes. Our main hypothesis is that wind differentially mobilises certain microbes from biocrusts, and that other microbes resist wind erosion. Related to this, we also expected to find biocrust organisms in watercourses. 

In this paper we investigate the microbial ecology of a dynamic dryland ecosystem in central Australia comprising a 25 km$^{2}$ claypan bounded by sand dunes and ephemeral rivers... \footnote{ from Tom: expalin the extraordinary study site. DLNP has to be one of the best places in the world to study/test this hypothesis. Huge, ephemeral river system to bring in water, sediment and microbes.  Large pans for the water, seds and microbes to settle out and for crusts to form. They are also great places for wind to interact with sediment - thus forming the fringing dunes. The bigger landscape setting and the microbial-geo-hydro model are pretty integral to this paper and we could really make this clear in the intro. - would here be the best place for this?}


\section{Material and Methods}

We used high throughput DNA sequencing of the 16S rRNA gene from soils and sediments to determine the bacterial community members and relative abundances in the drylands of central Australia. In addition, we applied an experimental wind tunnel treatment to simulate the effect of wind upon sand dunes stabilised by biocrusts. By collecting the mobilised dust and comparing the dust community with that of other samples, we were able to identify possible dispersal strategies and suggest the relevance of these to sediment stability and dust production. Results were interrogated to test hypotheses relating to interactions of microbes in geomorphic processes.

\subsection{Field site}
The dune runs approximately north-south and the claypan is on the east side (Figure \ref{fig:map}). Soil was sampled from five transects perpendicular to the dune, and water/sediment was sampled from water channels in the surrounding area.

<<map_out, echo=FALSE,fig.height=4, warning=FALSE, cache=TRUE,fig.cap='Sampling locations. a. General area of claypan including surrounding water channels; b. Closer view of dune and pan area.\\label{fig:map}'>>=

#Extract Legend 
g_legend<-function(a.gplot){ 
  tmp <- ggplot_gtable(ggplot_build(a.gplot)) 
  leg <- which(sapply(tmp$grobs, function(x) x$name) == "guide-box") 
  legend <- tmp$grobs[[leg]] 
  return(legend)} 

maplegend <- map1 + theme(legend.position="bottom")
legend <- g_legend(maplegend) 


grid.newpage()
pushViewport(viewport(layout = grid.layout(6, 2)))
vplayout <- function(x, y) viewport(layout.pos.row = x, layout.pos.col = y)
print(map1 + theme(legend.position="none"), vp = vplayout(1:5, 1))  # key is to define vplayout
print(map2 + theme(legend.position="none"), vp = vplayout(1:5, 2))
legend$vp = viewport(layout.pos.row = 6, layout.pos.col = 1:2)
grid.draw(legend)

@

The following locations were targeted for soil sampling (See Figures \ref{fig:photo-area} and \ref{fig:photo-sampling} for photographs):

\begin{enumerate}
\item [D.] Dune. Dune biocrust
\item [DF.] Dune flank. Dune flank biocrust. These sites were also used for wind tunnel experiments.
\item [N.] Nebkha. Biocrust situated on nebkha in the claypan to the east of the dune
\item [P.] Pan. Biocrust platelets in vegetation interspaces on the claypan
\item [C.] River channel. Sediment and water from the banks of water channels surrounding the dune/claypan complex
\end{enumerate}

Transects were planned to be approximtely 500 m apart to achieve even spacing along the dune, however some zones to the south were quite heavily impacted by cattle. These impacted zones were excluded because the dune flanks did not have suitable areas to run the wind tunnel. The transects are numbered 1-5 starting at the south end. 

\subsection{Sampling}
Sampled areas were 10x10 cm and were selected to represent the diversity of biocrust in the area (Figure \ref{fig:photo-sampling}). The sampled depth was dictated by coherence of the biocrust. The area was cut out with a sterile scalpel and the crust was lifted off the looser soil beneath. In areas where differing crust types were noticeable the sampled areas were selected to have both types where applicable (e.g. biological/physical, light/dark). In visibly disturbed areas e.g. from animals, un-disturbed crusts were selected (2x dune sites).

\begin{figure}
  \centering
      \includegraphics[trim = 17mm 10mm 17mm 10mm, clip,width=1\textwidth]{photos/photos1}
  \caption{Photographs of sampling areas including detail of nebkha and crust formations on the claypan.}
  \label{fig:photo-area}
\end{figure}

\begin{figure}
  \centering
      \includegraphics[trim = 17mm 10mm 17mm 10mm, clip,width=1\textwidth]{photos/photos2}
  \caption{Photographs taken during soil sampling, showing cuts made to 10 cm x 10 cm square areas of crust prior to lifting aseptically.}
  \label{fig:photo-sampling}
\end{figure}

Water channels were sampled by dragging a 5 ml sterile bijoux along the bank where it meets the water. Approximately 2.5 ml bank sediment and 2.5 ml standing water were collected in each bottle from the same location. 

\subsection{Wind tunnel}
In addition to the soil crust samples a wind tunnel was run on the dune flank zones within 3 m of where the soil crust sampling was done. The wind tunnel design was a duct-type design equipped with a sediment collection trap in which sterile Whatman xx filter papers were fitted. Further details of design and operation have previously been described by \citet{strong2016development}\footnote{should provide a bit more detail here}. The wind tunnel was run on dune flanks because they are naturally eroded by saltation impact from the dune sand above when it is windy, therefore the potential as an inoculum source to surrounding areas is greater than the higher dune areas. A saltation source was not added to these experiments for the sake of simplicity, as there will be saltation occurring naturally within the wind tunnel anyway. At each site the wind tunnel was run three times on adjacent areas for 5 minutes at 10 m/s for 5 minutes\footnote{Helene collected data to determine the exact run conditions}. This is a moderate treatment intended to be realistic and to collect only the easily mobilised particles / organisms. 

\subsection{Soil properties}
Total carbon and nitrogen content of soils were determined using a CN element analyser (Leco TruSpec). Particle size distribution was determined by ...

\subsection{DNA sequencing}
DNA was extracted in the field within 24 hours of sample collection using a Mobio Powersoil DNA extraction kit according to the manufacturers instructions except for increasing the soil amount slightly to 0.4 g per sample as described previously \citet{elliott2014niche}. DNA sequencing targeted the prokaryotic 16S rRNA gene and was performed by the Centre for Genomic Research (CGR) NERC Facility at the University of Liverpool, using an Illumina MiSeq in a Paired end approach. Resultant FastQ files were processed using QIIME \citep{caporaso2010qiime} for clustering, OTU table generation, and taxonomy assignment. These results were analysed using R and the phyloseq package \citep{mcmurdie2013phyloseq} for R.

\section{Results}

\subsection{Field observations}
Biological crusts on the claypan have previously been described morphologically, and typically have a blue-green appearance due to the presence of photosynthetic organisms, and they have a platelet formation caused by physical weathering (Strong et al ... presumably - please advise). Blue-green biological crusts were common but not the dominant surface cover on the claypan as a whole. They appear to have once been very widespread but have now been largely eroded away. Underneath them was a similar looking crust which is more brown (Figure \ref{fig:photo-sampling}), and this was not sampled in the present study. In many places on the claypan there were nebkha on top of the blue-green platelet crusts and the platelets ran up the nebkha slightly, however the crusts on nebkha appeared distinct. Platelet crusts appeared to be more common near nebkha. 

In summary, three distinct surface layers were observed in the claypan:
\begin{enumerate}
\item deeper layer brown crust (not sampled)
\item platelet formation surface biological crust (sampled: code P)
\item nebkha (sampled: code N)
\end{enumerate}

In dune areas the crust morphology was simpler as the platelet formation surface crusts and deeper crusts of different type were not present. In these areas (both dune and dune flank) there was widespread continuous biological soil crust which held together the sand surface, with some variation in pigmentation. Sometimes there were deeper (probably buried) crust layers, but only surface exposed crusts were sampled in this work.

\subsection{Soil properties}
... (Figure \ref{fig:chem}).

<<chem_out, echo=FALSE,fig.height=2,fig.cap='Mean total carbon and nitrogen (percent w/w) and pH of soil in and around Lake Constance claypan at Diamantina National Park, Australia. Error bars represent the standard error of the mean for each soil type (n = 5). Sample coding for soil types: D sand dune, DF dune flank, N nebkha, P pan surface. \\label{fig:chem}', warning=FALSE, cache=TRUE>>=
# melt for plotting
chemistry.melt <- melt(sample_data(bac.earthair),measure.vars=c("N_pc","C_pc","pH"),)
chemistry.melt$variable <- revalue(chemistry.melt$variable, c("N_pc"="nitrogen", "C_pc"="carbon"))
# make the plot 
print(ggplot(chemistry.melt, aes(x=zone_code, y=value, shape=element, colour=zone)) + stat_summary(fun.data = "mean_se", size=1) + facet_wrap(~variable,scales = "free_y",nrow=1) + labs(x = "zone") + theme(aspect.ratio=1,legend.position="none") + colScale )
@

\subsection{DNA sequencing and general microbial community composition}
DNA was successfully extracted and sequenced from \Sexpr{nsamples(bac)} samples, yielding \Sexpr{round(m_reads_total,1)} million reads. After pairing reads and quality control \Sexpr{round(m_reads_bac_qc,1)} million bacterial reads and \Sexpr{round(m_reads_arc_qc,1)} million archaeal reads remained. 

There were \Sexpr{ntaxa(bac)} bacterial OTUs and \Sexpr{ntaxa(arc)} archael OTUs detected (OTUs defined by 97 \% similarity which is approximately species-level difference between OTUs). This analysis was planned to target bacteria, and as archaea make up only \Sexpr{round(100*m_reads_arc_qc / m_reads_bac_qc,1)} \% of sequences, bacteria are the main focus of this paper.

A total of \Sexpr{length(unique(tax_table(bac)[,"Phylum"]))} bacterial phyla and \Sexpr{length(unique(tax_table(arc)[,"Phylum"]))} archael phyla were identified. Frequencies of the most abundant phyla are shown in Figure \ref{fig:waterp}. It can be seen that water channel sampels are dissimilar to the soil-based samples at the phylum level although all abundant phyla are represented in both soil and water. Some phyla differed in abundance between the dune flank soil and airborne dust liberated from the same soil by wind tunnel treatment, for example \textit{Cyanobacteria} (more abundant in soil) and \textit{Firmicutes} (more abundant in dust).

\begin{figure}[]
\begin{center}
<<OTU_source_out, echo=FALSE,fig.height=8.5, cache=FALSE>>=
grid.newpage()
pushViewport(viewport(layout=g0))
print(water_otus_plot,vp=vp.01)
print(wind_otus_plot,vp=vp.02)
@
\caption{ Relative abundance of bacterial phyla. Source indicates whether individual OTUs (97 pc similarity) contributing to the phylum abundance were found in the water channels (top), and whether they were significantly more abundant in soil or wind tunnel collectors at the dune flank sites (bottom). Only the 10 most abundant phyla are shown for clarity. Source designation is not indicative of the ultimate origin of OTUs.}\label{fig:waterp}
\end{center}
\end{figure}

\textit{Phormidium} was by far the most abundant cyanobacterial genus (Figure \ref{fig:waterc}), in common with our previous findings from the Kalahari in southern Botswana \citep{elliott2014niche}. \textit{Phormidium} and \textit{Microcoleus} species appeared to be more common in soil compared to airborne dust, whereas \textit{Leptolyngbya} species appeared more common in the soil compared to dust. \textit{Nodularia} species and \textit{Synechococcus} species appeared to be more common in water channels compared to soils, whilst the other genera mentioned were more common in soils. 

\begin{figure}[]
\begin{center}
<<OTU_source_out2, echo=FALSE,fig.height=5.5, cache=FALSE>>=
grid.newpage()
pushViewport(viewport(layout=g0))
print(water_otus_plot.c,vp=vp.01)
print(wind_otus_plot.c,vp=vp.02)
@
\caption{ Relative abundance of cyanobacterial genera. Source indicates whether individual OTUs (97 pc similarity) contributing to the genus abundance were found in the water channels (top), and whether they were significantly more abundant in soil or wind tunnel collectors at the dune flank sites (bottom). Unidentified OTUs were excluded. Only the 5 most abundant identified cyanobacterial genera are shown for clarity. Source designation is not indicative of the ultimate origin of OTUs.}\label{fig:waterc}
\end{center}
\end{figure}

The Chao1 species richness estimation (Figure \ref{fig:rich}) indicated in excess of 10,000 species present in each sample, without striking differences with respect to sample type. 

<<rich_out, echo=FALSE,fig.width=5,fig.height=4,fig.cap='Chao1 richness estimate (all prokaryotes). \\label{fig:rich}', cache=TRUE>>=

print(rich.ab + colScale)
@

Microbial community structure was compared with respect to sample source using nonmetric multidimensional scaling (NMDS). Initial results showed that the water samples differed extremely compared to all other samples (see also Figure \ref{fig:waterp}), and this dominated all other inter-sample differences. The water samples were therefore excluded from the NMDS to reveal the community stucutre relationships between the soil and airborne dust samples as shown in Figure \ref{fig:nmds}. The dune community differs from the dune flank community, and the airborne dust community derived from the dune flank differs from the dune flank community. Notably the dust community has a community structure approximately intermediate between the dune and nebkha communities.

<<nmds_out, echo=FALSE, fig.height=4, fig.cap='NMDS visualisation of prokaryotic community structure in soil (sand dunes, claypan, and nebkha) and airborne dust (from sand dunes). \\label{fig:nmds}', cache=TRUE>>=

print(NMDS.ab + colScale)
@

Microbial composition of the dune flank, pan, and nebkha sites varied considerably as can be seen from Figure \ref{fig:nmds}. High abundance OTUs are likely to be important contributors to the overall community differences visualised in Figure \ref{fig:nmds}, and details of these can be found in Table \ref{tab:deseq3}. In fact many of the abundant OTUs were related to each other, meaning that community differences may be evident and easier to understand at higher taxonomic ranks as can be seen in Figures \ref{fig:waterp} and \ref{fig:waterc}. In summary of these results it is clear that \textit{Cyanobacteria} numerically dominate at the experimental sites (about 28 \% of bacterial sequences), the overall cyanobacterial abundance at different sites is similar, and the dominant cyanobacterial genus is \textit{Phormidium}. The degree of dominance by \textit{Phormidium} varies, and differences in less abundant cyanobacterial genera (e.g. \textit{Leptolyngbya} and \textit{Microcoleus}) are evident between sites. For example \textit{Microcoleus} is the second most abundant identified cyanobacterial genus, being quite abundant in dune flank (2.1 \%) and claypan (3.1 \%) but rare in nebkha (0.2 \%).

\subsection{Dispersal of microbes by fluvial and aeolian processes}
Of the \Sexpr{pan.ntaxa} OTUs found in the claypan, \Sexpr{channel_pan.intersection.n} were also found in water channels, \Sexpr{dune_pan.intersection.n} were found in airborne dust from the dune flank, and \Sexpr{all.intersection} were found in both of these hypothesised inoculum sources. The dune and channel samples had \Sexpr{channel_dune.intersection.n} OTUs in common.

Whilst many soil OTUs were not detected in the air or water samples, most of the OTUs found exclusively in soil were rare, collectively accounting for only \Sexpr{soil_only_otus.sum} \% of reads in the soil samples. A breakdown of OTU occurrence in all samples summarised by phylum read frequency is shown in Figure \ref{fig:waterp}, with detail on cyanobacterial genera given in Figure \ref{fig:waterc}. It can be seen that co-occurrence of bacteria between soils and hypothesised inoculum sources is very common (assessed by number of reads not number of OTUs), and the relative abundance of organisms from different inoculum sources varies with respect to soil type and phylum. 

The DESeq2 model based approach \citep{love2014moderated} was used to check if any OTUs had significantly different abundance between the dune flank soil and the collected dust derived from the same soil obtained by wind tunnel treatment. This technique avoids problems associated with uneven sampling effort and includes correction for multiple testing. The results (adjusted p<0.05) indicated that \Sexpr{deseq_n_diff_air} OTUs (\Sexpr{round(deseq_sum_diff_air*-1)} \% of reads) were  more abundant in air samples and \Sexpr{deseq_n_diff_earth} OTUs (\Sexpr{round(deseq_sum_diff_earth)} \% of reads) were more abundant in soil samples. Details for a subset of these OTUs are shown in Tables \ref{tab:deseq1} and \ref{tab:deseq2}.

<<deseq_out1, echo=FALSE, cache=FALSE>>=
kable(outtab, digits=2, caption = "OTUs which are significantly more abundant in soil compared to dust blown from the soil. This is a subset of the DESeq2 analysis output, sorted by magnitude of difference in abundance. \\label{tab:deseq1}")

kable(outtab2, digits=2,caption = "OTUs which are significantly more abundant in dust blown from the soil compared to the whole soil. This is a subset of the DESeq2 analysis output, sorted by magnitude of difference in abundance. \\label{tab:deseq2}")

otus.df.out <- otus.df[1:10,c(rank_names(bac)[c(2,6)],sample_names(bac.rel.merged),"sig")]
colnames(otus.df.out)[colnames(otus.df.out)=="dune flank"] <- "d.flank"
# class(otus.df$dune)
# otus.df.out <- as.numeric(otus.df.out)
kable(otus.df.out,row.names = F, digits=2,caption = "OTU abundance in each sampling zone (pc), in order of overall abundance. Sig column indicates if the OTU was significantly more abundant in soil or wind tunnel filter after running the wind tunnel on dune flanks. The 10 most abundant OTUs overall are shown, full table with all OTUs is available in supplementary... \\label{tab:deseq3}")

write.csv(otus.df[ ,c(rank_names(bac)[1:7],sample_names(bac.rel.merged))],"all_otus.csv")
@

\section{Discussion}\footnote{This section may be divided by subheadings. Discussions should cover the key findings of the study: discuss any prior art related to the subject so to place the novelty of the discovery in the appropriate context; discuss the potential short-comings and limitations on their interpretations; discuss their integration into the current understanding of the problem and how this advances the current views; speculate on the future direction of the research and freely postulate theories that could be tested in the future.}

We have performed the first detailed characterisation of the prokaryotic community in a lanscape-scale dryland system comprising terrestrial, fluvial, and aeolian components. This is also one of only a few studies to identify the microbes present in the central Australian drylands which cover over xx KM$^{2}$. Whilst determining the microbial composition of soils was not our primary objective, results in addition to those presented are available in sequence databases and supplementary materials. This is an important contribution as such data on drylands are lacking in the global literature \citep{ferrenberg2017biological}. Our novel approach used landscape sampling combined with high-sensitivity differential abundance analysis, enabling us to test hypotheses about the ecology of biocrust organisms in drylands, particularly focusing on the implications for landscape stability and dust production. 

\subsection{Co-occurrence of taxa in biocrusts and river channels}
The higher proportion of DNA sequence reads shared between dune and river compared to river and pan, may be explained by the fact that flooding of the pan is a rare event (last occurring x years previously) but mobilisation of dust from sand dunes is occurring continuously. Due to the dune being at higher elevation than all other locations, it might be expected to harbour fewer OTUs of aquatic origin, however this does not seem to be the case (Figure \ref{fig:waterp}). In fact our data are not informative with respect to the direction of dispersal. We expect, however, that co-occurrence between dune and river channel may be driven principally by wind dispersal (direction dune>river), whereas co-occurrence between river channel and claypan may be driven principally by flood events (direction river>claypan). The question of dispersal direction could be tackled in future work by monitoring microbial populations over time and relating this to weather conditions and occurrence of flood events. Whilst we are satisfied in the present work to uncover clues about microbial dispersal in dryland systems, the next challenge is explicitly linking this to function at landscape scale. 

\subsection{Wind preferentially lifts certain taxa from biocrusts}
We have found strong evidence that microbes are differentially liberated from the dryland soil surface by moderate wind, and we have identified the behaviour of taxa in respect of this. These results provide numderous opportunities to investigate whether microbes are specifically adapted in this regard or not, and explore the implications of this for ecosystem functioning. Our original hypotheses motivating this work are predicated on the assumption that microbes are adapted for their environment, thus whether they stick or fly is most likely a consequence of natural selection. The question of whether biocrust microbes are ecosystem engineers is just an alternative way of asking whether microbes are active players in geomorphic processes and landscape stability. If their actions change the environment, then the answer to both can be "yes". There is ample evidence for this in the literature \citep[e.g.][]{bowker2007biological}, so in the present work we sought to shed light on the selective pressures and biological adaptations which may be driving microbial phenotypes of relevance to dispersal at the soil surface in drylands. 

Dune and pan sites found throughout the world's drylands present similar harsh environments but different challenges for survival at the soil surface, especially in relation to disperal and colonisation. Claypans can be completely engulfed by flood events lasting ??\footnote{give range and citation if possible} and often leading to extensive replacement of the soil surface with new or overturned sediment, but in contrast dunes rarely if ever experience flooding. Due to the extensive vacant niche with abundant resources when floodwaters subside, organisms colonising the claypan soil will face extreme competition.  Vegetative growth is therefore not expected to be an effective strategy for colonising the recently flooded claypan because it would be too slow, hence we expected that more generalised taxa already established in the rivers are most likely to be primary colonisers of the pan surface. We also expect due to the relatively short flooding interval at our field site that succession will not proceed far, therefore the claypan biocrusts are most likely to conatain organisms that are adapted for this lifecycle including both aquatic and terrestrial phases. Whilst it is possible that the pan biocrusts are mostly composed of ill-adapted aquatic species, we find this unlikely given the long geological timescales and widespread long-term global presence of similar systems having provided ample opportunity for organisms to efficiently exploit their periodicity. 

Stabilised sand dunes do not experience wholesale disruption, therefore succession can take place over long timescales and un-crusted sites exposed by localised disturbances can likely be re-colonised by vegetative growth from nearby areas. Large scale burial of sand dunes does occur as a consequence of Aeolian deposition (REF), but these events do not disrupt the crust or bury it deeply, and it is known that biocrust biota can move vertically up to xx mm in a day (REF) so shallow burial is not expected to threaten biocrust survival. This occasional partial burial followed by upward migration of biocrust biota could contribute significantly to the stability of sand dunes. Our data support these ideas, showing that sand dunes have the highest proportion of taxa that resist wind erosion (Figure \ref{fig:waterp}), thus sand dune surface communities are adapted in a way that promotes stability in the dune. This stabilising effect of the microbiota can be regarded as ecosystem engineering, as it modifies the environment to provide the stable soil surface required for long-term biocrust survival. Many OTUs were found to resist becoming airborne\footnote{specify supplementary info for this}, including most \textit{Cyanobacteria} which have long been recognised for their stabilising role in biocrusts \citep{belnap1998vulnerability}. This kind of adaptation may also inhibit dispersal by the wind, limiting the ability of biocrusts to colonise new areas or regenerate after elimination for example as a result of over-grazing, because the regenerating inoculum must come from a local source (e.g. vegetative growth or transport by an animal). Possible difficulties for biocrusts to regenerate after extreme sustained disturbance is something we suggested as a risk in \citet{elliott2014niche}, and is be supported by the present results which confirm the resistance of many biocrust taxa to becoming airborne.

The nebkha sites within the claypan were the only place where a high proportion of the biota was identified as being easily mobilised by wind (xx \%; Figure \ref{fig:waterp}). This was not a direct measurement, but based upon taxa from the dune flank that were easily mobilised and measuring their abundance in the nebkha. We anticipate that a direct measurement on the nebkha sites would yield a higher proportion but this would be extremely difficult to do because the nebkha morphology is incompatible with wind tunnel apparatus. It should also be noted that our techniques do not establish direction of transport, therefore a proportion (possibly the majority) of the taxa mobilised from the dune flank by wind treatment are likely to have originated from the pan and nebkha sites. The reason for a high proportion of wind-mobile biota on the nebkha is unlikely to be a purely physical phenomenon, since the nebkha physico-chemical properties are similar\footnote{debatable though, await grain size results and any further info} to the dune and pan compartments (Figure \ref{fig:chem}). A biological explanation may be that nebkha are inhabited by biocrusts that rely principally on aerial disperal, because they are adapted as primary colonisers of newly exposed niches. Such organisms would need to have the capacity for adhesion to establish a colony in the new niche, but also an ability to disperse from the colony. This could be achieved by phenotypic switching which is a well known phenomenon in biofilms (REF). Furthermore, biocrust organisms adapted for aerial dispersal would also need adaptations for survival in the air, such as small size and resistance to dessication and radiation. Alternatively, it is possible that the main niche of these microbes is in fact the atmosphere, and their presence in biocrusts is transitory, for instance to acquire nutrients before moving on.

One example of possible adaptation for mobilisation is a \textit{Deinococcus} species which was found to be significantly more likely to become airborne than to stay in the soil during wind treatment (Table \ref{tab:deseq2}). This genus is well known for its remarkable ability to withstand dessication and all kinds of radiation \citep{makarova2001genome}. It has been found in desert soils before (e.g. \citet{rainey2005extensive}) but according to the latest version of Bergey's manual \citep{whitman2015bergey} the natural habitat of \textit{Deinococcus} species remains unknown. We suggest then that the habitat of this ancient lineage is in fact the biocrust top surface, and it is adapted for wind dispersal - a hypothesis supported by the present results. With such ecology \textit{Deinococcus} species may serve a protective function to the biocrust, and can also ensure that they quickly colonise any newly emerging biocrust which will immediately benefit from such protection early in their development. 

\subsection{Microbial dispersal and landscape stability}

In summary based on our results, we propose the hypothesis that there are at least three distinct dispersal strategies employed by biocrust inhabitants, each strategy having particular phenotypic features and related geomorphic potential (Table \ref{tab:concept}). Whilst we propose a typical niche associated with each dispersal strategy, we do not expect this to be exclusive. Rather, we suggest that these strategies all co-exist in each biocrust community, and the relative abundance of organisms exhibiting each phenotype will vary depending on the life history of each particular biocrust. 

\subsubsection{Vegetative dispersal strategy}
Firstly, the vegetative dispersal strategy relates to the classic understanding of biocrust upon which the majority of biocrust literature is focused, and sand dune biocrusts are an example of this. These organisms are highly invested in maintaining a stable soil surface which facilitates the establishment of long-term extensive colonies that are highly resilient to harsh environmental conditions. This investment includes the production of large amounts of EPS which stabilise the soil, and production of protective pigments in upper layers. They are extremely resource efficient and can maintain activity including primary production at very low water potential, thus providing the capacity for diverting resources to soil stabilisation.

\subsubsection{Aeolian dispersal strategy}
Biocrusts may also be established from airborne inocula, and the nebkha biocrusts are most likely an example of this. Because strongly adhesive biocrust taxa are less likely to become airborne, biocrusts established from airborne sediments may lack the strength of vegetatively established biocrusts at least initially. They must therefore rely on other stabilising landscape features such as stones and plants, enabling them to settle without investing strongly in promoting landscape stability although they probably do this incidentally. Taxa adapted for aerial dispersal must achieve high numbers in the atmosphere to be successful, thus they are likely to be R-strategists or copiotrophs that exploit islands of fertility to produce more airborne inocula. This means that although they have some capacity to stabilise soil, they may also contribute to dust production and export of nutrients from the soil.

\subsubsection{Fluvial dispersal strategy}
Periodic cycles of wetting and drying are a common feature of the dryland soil surface which all biocrust organisms must adapt to in order to exploit occasional rainfall and dew. Adaptation to flooding cycles requires in addition an ability to persist long-term in the aqueous environment, and the terrestrial environment. When floods subside in drylands the soil is rich in resources, and the aquatic organisms already present are likely to benefit from a founder effect that will prevent significant establishment of vegetatively or aerially dispersed microbes. This is likely to have happened in the claypan, where the aquatic organisms best placed to exploit the founder effect would be R strategists that can quickly consume the available resources before the soil dries out. In the dry times that follow these organisms are unlikely to be able to function effectively in the absence of abundant water and in the face of dwindling resources. If they are adapted for fluvial dispersal then their main task between floods is to survive and exclude competitors until further floodwaters arrive, therefore they are likely to lie dormant unless activated by prolonged exposure to water. In terms of landscape stability they will be an immediate benefit as floods subside due to the extensive production of biomass, however during dry times they may suffer erosion and are unlikely to have the capacity for repair.

\subsection{Geomorphic predictions from microbial ecology}
...
\begin{enumerate}
\item Nitrogen deposition - Stable dune crusts with vegetative dispersal mechanism will experience increased erosion because the oligotrophic crust will face competition from copiotrophic aerial taxa which deplete soil resources
\item More frequent flooding - Claypans will experience increased stability because the biocrust which is not well adapted for repair under dry conditions will be replenished more often
\item More small rainfall events - Claypan biocrusts will be degraded due to activation of heterotrophs but insufficient water to initiate photosynthesis in R strategist phototrophs, resulting in increased dust production from claypans
\end{enumerate}


\section{Conclusions}
In conclusion, we have shown that biocrust taxa vary in their capacity to be mobilised by the wind and we have suggested that this is inversely related to the taxon potential to promote landscape stability. We have also shown that biocrust microbes are easily found in watercourses, and proposed that some biocrust taxa are principally dispersed by fluvial deposition. These results provide new evidence supporting the microbial role in landscape stability and suppresion of dust production in drylands. By interpreting results from an ecological perspective, we provided a framework for predicting how microbes will influence landscape stability under different scenarios. This approach can be used to help predict responses to climate change and land use change, and also to plan mitigation strategies for maximiasing soil stability. 

% Table generated by Excel2LaTeX from sheet 'Sheet1'
\begin{table}[htbp]
  \centering
    \caption{Summary of hypothesised dispersal strategies for biocrust microbes and their possible geomorphic effects.}%
    \begin{tabular}{p{3.5cm}p{4cm}p{4cm}p{4.5cm}}
    \textbf{Dispersal strategy} & \textbf{Niche} & \textbf{Likely phenotype} & \textbf{Geomorphic effect} \\
    \hline
    Vegetative growth & Long-term stable soils\newline{}e.g. stabilised sand dune & oligotrophic\newline{}mutualistic\newline{}EPS production\newline{}Can recover from localised disturbance\newline{}Low water requirement & Strongly stabilise soil\newline{}Suppress dust production\newline{}Trapping dust\newline{}Enhance soil fertility \\
    \hline
    Aerial & Recently exposed soil and transient landscape features. Possibly active whilst airborne.\newline{}e.g. nebkha & copiotrophic\newline{}exploitative\newline{}Limited recovery from localised disturbance\newline{}Medium water requirement & Partially stabilise soil\newline{}Promote moderate dust production including removal of soil nutrients \\
    \hline
    Fluvial & Water bodies, flood plains and pans\newline{}e.g. claypan & Copiotrophic\newline{}Excludes aerial invaders\newline{}Poor ability to recover from localised disturbance\newline{}High water requirement & Partially stabilise soil\newline{}Suppress dust production\newline{}Neutral soil fertility influence \\
    \end{tabular}%
    \label{tab:concept}%
\end{table}%




\section*{Author Contributions}

The Author Contributions section is mandatory for all articles, including articles by sole authors. If an appropriate statement is not provided on submission, a standard one will be inserted during the production process. The Author Contributions statement must describe the contributions of individual authors referred to by their initials and, in doing so, all authors agree to be accountable for the content of the work. Please see  \href{http://home.frontiersin.org/about/author-guidelines#AuthorandContributors}{here} for full authorship criteria.

\section*{Funding}
Details of all funding sources should be provided, including grant numbers if applicable. Please ensure to add all necessary funding information, as after publication this is no longer possible.

\section*{Acknowledgments}
This is a short text to acknowledge the contributions of specific colleagues, institutions, or agencies that aided the efforts of the authors.


\bibliographystyle{frontiersinSCNS_ENG_HUMS} % for Science, Engineering and Humanities and Social Sciences articles, for Humanities and Social Sciences articles please include page numbers in the in-text citations
%\bibliographystyle{frontiersinHLTH&FPHY} % for Health, Physics and Mathematics articles
\bibliography{references}

\section*{Figure captions}

%%% Please be aware that for original research articles we only permit a combined number of 15 figures and tables, one figure with multiple subfigures will count as only one figure.
%%% Use this if adding the figures directly in the mansucript, if so, please remember to also upload the files when submitting your article
%%% There is no need for adding the file termination, as long as you indicate where the file is saved. In the examples below the files (logo1.jpg and logos.jpg) are in the Frontiers LaTeX folder
%%% If using *.tif files convert them to .jpg or .png
%%%  NB logo1.jpg is required in the path in order to correctly compile front page header %%%

\begin{figure}[h!]
\begin{center}
\includegraphics[width=10cm]{logo1}% This is a *.jpg file
\end{center}
\caption{ Enter the caption for your figure here.  Repeat as  necessary for each of your figures}\label{fig:1}
\end{figure}


\begin{figure}[h!]
\begin{center}
\includegraphics[width=15cm]{logos}
\end{center}
\caption{This is a figure with sub figures, (A) is one logo, (B) is a different logo.}\label{fig:2}
\end{figure}

%%% If you are submitting a figure with subfigures please combine these into one image file with part labels integrated.
%%% If you don't add the figures in the LaTeX files, please upload them when submitting the article.
%%% Frontiers will add the figures at the end of the provisional pdf automatically
%%% The use of LaTeX coding to draw Diagrams/Figures/Structures should be avoided. They should be external callouts including graphics.


\end{document}



