\documentclass[linenumbers]{agujournal}
%% To submit your paper:
%% \documentclass[linenumbers,draft]{agujournal}

%% remove this usepackage before submission.
\usepackage{url,hyperref}

\journalname{JGR-Biogeosciences}

%% ------------------------------------------------------------------------ %%
%
%  ENTER Title Page commands:
%
%% ------------------------------------------------------------------------ %%

% (A title should be specific, informative, and brief. Use
% abbreviations only if they are defined in the abstract. Titles that
% start with general terms then specific results are optimized in
% searches)

% Example: \title{This is a test title}

% (List authors by first name or initial followed by last name and
% separated by commas. Use \affil{} to number affiliations, and
% \thanks{} for author notes.
% Additional author notes should be indicated with \thanks{} (for
% example, for current addresses).

% Example: \authors{A. B. Author\affil{1}\thanks{Current address, Antartica}, B. C. Author\affil{2,3}, and D. E.
% Author\affil{3,4}\thanks{Also funded by Monsanto.}}

% (include name and email addresses of the corresponding author.  More
% than one corresponding author is allowed in this LaTeX file and for
% publication; but only one corresponding author is allowed in our
% editorial system.)

%% Corresponding Author:
% Corresponding author mailing address and e-mail address:

% Example: \correspondingauthor{First and Last Name}{email@address.edu}

% Authors are individuals who have significantly contributed to the
% research and preparation of the article. Group authors are allowed, if
% each author in the group is separately identified in an appendix.)

% \affiliation{1}{First Affiliation}
% \affiliation{2}{Second Affiliation}
% \affiliation{3}{Third Affiliation}
% \affiliation{4}{Fourth Affiliation}

%% Keypoints, final entry on title page.
% Example:
% \begin{keypoints}
% \item	List up to three key points (at least one is required)
% \item	Key Points summarize the main points and conclusions of the article
% \item	Each must be 100 characters or less with no special
% characters or punctuation
% \end{keypoints}

%% \begin{abstract} begins second page

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Track Changes:
% To add words, \added{<word added>}
% To delete words, \deleted{<word deleted>}
% To replace words, \replaced{<word to be replaced>}{<replacement word>}
% To explain why change was made: \explain{<explanation>}

% At the end of the document, use \listofchanges, which will list the
% changes and the page and line number where the change was made.

% When final version, \listofchanges will not produce anything,
% \added{} word will be printed, \deleted{} will take away the word,
% \replaced{}{} will print only the 2nd argument.
% \explain will not print anything.

% Optional argument:
% You can also add additional information to be printed with the list
% of changes, to indicate the initials of the person changing the text,
% and the time and/or date of the change, or any other comment by using
% the optional [] argument:
% \added[AH, 3:30pm, Feb 18, 2016]{added term}
% will yield
% [AH, 3:30pm, Feb 18, 2016] added term on page...
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{document}

<<include=FALSE>>=
library(knitr)
opts_chunk$set(
concordance=TRUE
)
@

%% ------------------------------------------------------------------------ %%
%
%  TITLE
%
%% ------------------------------------------------------------------------ %%


\title{Landscape Stability in Drylands Influenced by Dispersal Strategy of Soil Bacteria}


%% ------------------------------------------------------------------------ %%
%
%  AUTHORS AND AFFILIATIONS
%
%% ------------------------------------------------------------------------ %%

 \authors{David R. Elliott\affil{1},
 Andrew D. Thomas\affil{2}, Craig L. Strong\affil{3},
 Helene A. Aubault\affil{4}, Grant H. McTainsh\affil{5}, and Joanna Bullard\affil{6}}

\affiliation{1}{Environmental Sustainability Research Centre, University of Derby, Derby, UK.}
\affiliation{2}{Department of Geography and Earth Sciences, Aberystwyth University, Aberystwyth, UK.}
\affiliation{3}{The Fenner School of Environment \& Society, ANU College of Sciences, The Australian National University, Canberra   ACT 2601,  Australia.}
\affiliation{4}{TBC.}
\affiliation{5}{Griffith School of the Environment, Griffith University, Nathan, Queensland, Australia.}
\affiliation{6}{Department of Geography, School of Social, Political and Geographical Sciences, Loughborough University, Leicestershire, UK.}

%% Corresponding Author
%(include name and email addresses of the corresponding author.  More
%than one corresponding author is allowed in this Word file and for
%publication; but only one corresponding author is allowed in our
%editorial system.)

\correspondingauthor{D. R. Elliott}{d.r.elliott@derby.ac.uk}

%  List up to three key points (at least one is required)
%  Key Points summarize the main points and conclusions of the article
%  Each must be 100 characters or less with no special characters or punctuation

\begin{keypoints}
\item Microbial community of dryland ephemeral lake bed, dunes, and river channels characterised
\item Wind preferentially mobilises specific taxa from dryland biological soil crusts (biocrusts)
\item Microbial dispersal mechanisms may influence soil stability by promoting or inhibiting adhesion
\end{keypoints}

%% ------------------------------------------------------------------------ %%
%
%  ABSTRACT
%
%% ------------------------------------------------------------------------ %%


\begin{abstract}
The abstract (1) states the nature of the investigation and (2) summarizes the important conclusions. The abstract should be suitable for indexing. Your abstract should:

be set as a single paragraph.

be less than 250 words for all journals except GRL, for which the limit is 150 words.

not include table or figure mentions.

not include reference citations.

define all abbreviations.

\end{abstract}

%% ------------------------------------------------------------------------ %%
%
%  TEXT
%
%% ------------------------------------------------------------------------ %%

%%% Suggested section heads:
% \section{Introduction}
%
% The main text should start with an introduction. Except for short
% manuscripts (such as comments and replies), the text should be divided
% into sections, each with its own heading.

% Headings should be sentence fragments and do not begin with a
% lowercase letter or number. Examples of good headings are:

% \section{Materials and Methods}
% Here is text on Materials and Methods.

% \subsection{A descriptive heading about methods}
% More about Methods.
%
% \section{Data} (Or section title might be a descriptive heading about data)
%
% \section{Results} (Or section title might be a descriptive heading about the
% results)
%
% \section{Conclusions}


\section{Introduction}
\label{sec:intro}

<<setup, include=FALSE, cache=FALSE>>=
options(width = 120)
library(phyloseq)
library("ggplot2")
library(plyr)
library(vegan)
library(grid)
library(reshape2)
library(scales)   # needed for formatting y-axis labels to non-scientific type
library(RColorBrewer)
library("DESeq2")
library(knitr)
library(ggmap)
library(sp)
library(rgdal) # needed by sp

# set global chunk options
opts_chunk$set(fig.path='figure/minimal-', fig.align='center', fig.show='hold', message=FALSE)
options(formatR.arrow=TRUE,width=90)
set.seed(42)

theme_set(theme_bw())

# setup grid
g0 <- grid.layout(nrow=2, ncol=1)
vp.01 <- viewport(layout.pos.col=1, layout.pos.row=1) 
vp.02 <- viewport(layout.pos.col=1, layout.pos.row=2)

gl <- grid.layout(nrow=1, ncol=2)
vp.1 <- viewport(layout.pos.col=1, layout.pos.row=1) 
vp.2 <- viewport(layout.pos.col=2, layout.pos.row=1)
@

<<load,echo=FALSE,message=FALSE, warning=FALSE, cache=TRUE>>=

# set the relative % abundance below which we will discard otus for some
# processing (.rel objects). This affects the otus.df object which outputs to csv
# it does not affect richness, NMDS, deseq2 which use full data
prune_pc <- 0.2

bac <- import_biom("data/Elliott_Diamantina_Microbiome_Data_S1[otu-tax].biom", parseFunction = parse_taxonomy_greengenes)

map <- import_qiime(mapfilename = "data/Elliott_Diamantina_Microbiome_Table_S1[sample-metadata].txt")
bac <- merge_phyloseq(bac,map)

# fix incorrect spelling of nebkha in the biom file.
levels(sample_data(bac)$zone)[levels(sample_data(bac)$zone)=="nebka"] <- "nebkha"
sample_names(bac) <- gsub(pattern = "nebka" , x = sample_names(bac), replacement = "nebkha")


# remove chloroplast
  bac <- subset_taxa(bac,Class!="Chloroplast")

# shorten long OTU names
  taxa_names(bac) <- gsub("New.","N",taxa_names(bac))
  taxa_names(bac) <- gsub("Cleanup.","C",taxa_names(bac))
  taxa_names(bac) <- gsub("Reference","R.",taxa_names(bac))
  
  # standardise unknown taxon entries
tax_table(bac)[is.na(tax_table(bac))] <- "Unknown"
tax_table(bac)[tax_table(bac) %in% c("", "unclassified", "Unclassified")] <- "Unknown"

# keep combined kingdom objects
ab <- bac

# separate archaea and bacteria into different objects
arc <- subset_taxa(bac,Kingdom=="Archaea")
bac <- subset_taxa(bac,Kingdom=="Bacteria" )

expt.deseq <- subset_samples(ab,zone=="dune flank")
expt.deseq = prune_taxa(taxa_sums(expt.deseq) > 0, expt.deseq)
# add 1 to the whole OTU table, to avoid infinity problems in deseq analysis
# this will reduce the fold changes
otu_table(expt.deseq) <- otu_table(expt.deseq)+1
  
# convert OTU observations into relative abundance per sample
arc.rel <- transform_sample_counts(arc, function(x) 100*x/(sum(x)))
bac.rel <- transform_sample_counts(bac, function(x) 100*x/(sum(x)))
ab.rel <- transform_sample_counts(ab, function(x) 100*x/(sum(x)))
#ab.rel <- merge_phyloseq(otu_table(bac.rel),tax_table(bac.rel),otu_table(arc.rel),tax_table(arc.rel),sample_data(bac.rel))

ab.rel.phylum <- tax_glom(ab.rel,taxrank = "Phylum")
cyano.rel <- subset_taxa(bac.rel,Phylum=="Cyanobacteria")

# Identify top phyla and cyano genera - so they can be selectively plotted/analysed
top.phyla = names(sort(tapply(taxa_sums(ab.rel), tax_table(ab.rel)[, "Phylum"], sum), TRUE)[1:10])
top.cyano.genus = names(sort(tapply(taxa_sums(cyano.rel), tax_table(cyano.rel)[, "Genus"], sum), TRUE)[1:6])
top.cyano.genus <- top.cyano.genus[top.cyano.genus!="Unknown"]

# keep only the top phyla, but retain all in .all object for when needed
bac.rel.all <- bac.rel
ab.rel.all <- ab.rel
#bac.rel <- subset_taxa(bac.rel, Phylum%in%top.phyla)
#ab.rel <- subset_taxa(ab.rel, Phylum%in%top.phyla)
# commented above removal of rare phyla because similar is done below (ab.phylum.rr)
# and these objects are used to exporting results where this trimming was not expected (e.g. otus.df)

# agglomerate cyanos by genus, used later on
cyano.rel.g <- tax_glom(cyano.rel,taxrank = "Genus")

# simplify data (speed up processing). If this isn't done it causes long calculation time and problems viewing the PDF. Need to check exactly which outputs this is affecting and whether it matters:
# - affects the otus.df output to csv

bac.rel <- prune_taxa(taxa_sums(bac.rel) > prune_pc, bac.rel) 
ab.rel <- prune_taxa(taxa_sums(ab.rel) > prune_pc, ab.rel)
#bac.rel <- prune_taxa(taxa_sums(bac.rel) > 0.1, bac.rel) 
#unique(tax_table(ab.rel)[,1])

# To simplify plots
# Remove phyla not seen at more than 1 % in at least 10% of the samples.
ab.rel.phylum.rr = filter_taxa(ab.rel.phylum, function(x) sum(x > 1) > (0.1*length(x)), TRUE)

# Save a .csv of all the phylum abundances
#phylumdf.all = psmelt(tax_glom(ab.rel.phylum,taxrank = "Phylum"))
#write.csv(phylumdf.all,"phylum_breakdown.csv")

ab.rel.phylum.merged <- merge_samples(subset_samples(ab.rel.phylum,element!="air"),"zone")
otu_table(ab.rel.phylum.merged) <- otu_table(ab.rel.phylum.merged)/5

tp <- t(otu_table(ab.rel.phylum.merged))
tp <- as.data.frame(tp)
phyla.df <- as.data.frame(cbind(tax_table(ab.rel.phylum.merged),tp))
phyla.order <- taxa_names(ab.rel.phylum.merged)[order(taxa_sums(ab.rel.phylum.merged),decreasing = T)]
phyla.df <- phyla.df[phyla.order,]

write.csv(phyla.df[ ,c(rank_names(bac)[1:2],sample_names(ab.rel.phylum.merged))],"results/Elliott_Diamantina_Microbiome_Table_S2[phyla].csv", row.names = FALSE)

cyano.rel.g.merged <- merge_samples(subset_samples(cyano.rel.g,element!="air"),"zone")
otu_table(cyano.rel.g.merged) <- otu_table(cyano.rel.g.merged)/5
tc <- t(otu_table(cyano.rel.g.merged))
tc <- as.data.frame(tc)
cyano.df <- as.data.frame(cbind(tax_table(cyano.rel.g.merged),tc))
cyano.order <- taxa_names(cyano.rel.g.merged)[order(taxa_sums(cyano.rel.g.merged),decreasing = T)]
cyano.df <- cyano.df[cyano.order,]

write.csv(cyano.df[ ,c(rank_names(bac)[1:6],sample_names(cyano.rel.g.merged))],"results/Elliott_Diamantina_Microbiome_Table_S3[cyanobacteria].csv", row.names = FALSE)

bac.earthair <- subset_samples(bac,element!="water")
arc.earthair <- subset_samples(arc,element!="water")
ab.earthair <- subset_samples(ab,element!="water")
ab.earth.rel <- subset_samples(ab.rel,element=="earth")

#Create a custom color scale
myColors <- brewer.pal(7,"Set1")[c(2,1,7,5,3)]
names(myColors) <- levels(get_variable(bac,"zone"))
colScale <- scale_colour_manual(name = "zone",values = myColors)

@

<<analyse1,echo=FALSE,message=FALSE, results='hide', warning=FALSE, cache=TRUE>>=
rich.ab <- plot_richness(ab,measures = "Chao1",color = "zone", shape="element",x="zone") + geom_point(size=5,alpha=0.7) 

rich.ab <- plot_richness(ab,measures = "Chao1",color = "zone", shape="element",x="zone_code") + geom_point(size=5,alpha=0.7, position=position_dodge(width=1)) 
rich.ab$layers <- rich.ab$layers[3]
rich.ab$theme$axis.text.x$angle <- 0
rich.ab$theme$axis.text.x$hjust <- 0.4

ord.ab <- ordinate(ab.earthair, "NMDS", "bray")
NMDS.ab <- plot_ordination(ab.earthair, ord.ab, type = "sites", color = "zone", shape="element") + geom_point(size=5,alpha=0.7) + theme(aspect.ratio=1) + theme(legend.position="right")
NMDS.ab$layers <- NMDS.ab$layers[2]

m_reads_total <- sum(get_variable(ab,'n_reads'))/1000000
m_reads_bac_qc <- sum(otu_table(bac))/1000000
m_reads_arc_qc <- sum(otu_table(arc))/1000000

# ordinate again with all samples for supplementary data
ord.all <- ordinate(ab, "NMDS", "bray")
NMDS.all <- plot_ordination(ab, ord.all, type = "sites", color = "zone", shape="element") + geom_point(size=5,alpha=0.7) + theme(aspect.ratio=1) + theme(legend.position="right") + colScale
NMDS.all$layers <- NMDS.all$layers[2]

pdf("results/Elliott_Diamantina_Microbiome_Figure_S1[nmds].pdf")
print(NMDS.all)
dev.off()

# ordinate using phyloseq then stepwise model fitting
# define some limit models for automatic model building
mod0 <- ordinate(ab.earth.rel, "RDA", formula = ~ 1) # Model with intercept only
mod1 <- ordinate(ab.earth.rel, "RDA", formula = ~ N_pc + C_pc + pH + ps_d10 + ps_d50 + ps_d90 + ps_sp.surf.area + ps_lt4 + ps_4to16 + ps_16to63 + ps_63to250 + ps_gt250 ) # Model with all explanatory variables

# Use ordistep to select constraints for RDA
step.result <- ordistep(object=mod0,scope=formula(mod1),direction="both")
#step.result$call
#step.result$anova

# Plot the automatically selected model (for supplementary info)

pdf("results/Elliott_Diamantina_Microbiome_Figure_S1A[RDA].pdf")
cols <- myColors[c("dune","dune flank", "nebkha","pan")]
plot(step.result,type = "none")
points(step.result,pch=as.numeric(sample_data(ab.earth.rel)$zone_code), col=cols[as.numeric(sample_data(ab.earth.rel)$zone_code)])
points(step.result, display = "cn", lwd=1.5,head.arrow = 0.15 )
text(step.result, display = "cn")
text(step.result, col=cols[as.numeric(sample_data(ab.earth.rel)$zone_code)])
dev.off()


@

<<locations, echo=FALSE, message=FALSE, results='hide', warning=FALSE, eval=TRUE, cache=TRUE>>=

# Coordinates conversion to locations on the Earth in the Universal Transverse Mercator (UTM) coordinate system using the sp (spatial points) package. 

# Calculate spatial points based on the lat/lon information
# based on http://grokbase.com/t/r/r-sig-geo/136h9k8e9y/lat-long-utm-again

samples_latlon <- sample_data(bac)[,c("lat","lon")]
samples_latlon <- data.frame(samples_latlon)
#convert to sp object:
coordinates(samples_latlon) <- ~ lon + lat #longitude first

#adding a coordinate reference system:
proj4string(samples_latlon) <- CRS("+proj=longlat +ellps=WGS84 +datum=WGS84")

#project using sp transform:
UTM <- spTransform(samples_latlon, CRS=CRS("+proj=utm +zone=53 +ellps=WGS84 +datum=WGS84 +units=m +no_defs +towgs84=0,0,0"))

# Now get the spatial points back into the main dataframe:
UTM.df <- as.data.frame(UTM)
UTM.df$easting <- UTM.df$lon
UTM.df$northing <- UTM.df$lat

sample_data(bac) <- cbind(sample_data(bac),UTM.df[,c("easting","northing")])
@

<<map2,echo=FALSE, message=FALSE, results='hide', warning=FALSE,eval=TRUE, cache=TRUE>>=

# plot using ggmap. This just uses lat/lon, not the spatial points.

loc <- c(lon = mean(get_variable(bac,"lon")), lat = mean(get_variable(bac,"lat")))
locZoom <- c(lon = mean(get_variable(bac,"lon")[get_variable(bac,"zone")!="channel"]), lat = mean(get_variable(bac,"lat")[get_variable(bac,"zone")!="channel"]))
locZoom2 <- c(lon = mean(get_variable(bac,"lon")[1:2]), lat = mean(get_variable(bac,"lat")[1:2] ))

locZoom2 <- c(lon = mean(samples_latlon$lon["dune4"]), lat = mean(samples_latlon$lat["nebkha4"] ))

dMap <- get_map(location=loc, source="google", maptype = "satellite", zoom = 11)
dMapZoom <- get_map(location=locZoom, source="google", maptype = "satellite", zoom = 14)
dMapZoom2 <- get_map(location=locZoom2, source="google", maptype = "satellite", zoom = 17)

basemap <- ggmap(dMap)
basemapZoom <- ggmap(dMapZoom)
basemapZoom2 <- ggmap(dMapZoom2)

map1 <- basemap + geom_point(aes(x = lon, y = lat, colour=zone,shape=element),alpha=0.5, data = sample_data(bac), size = 4) + colScale

bac.zoommap <- sample_data(subset_samples(physeq = bac,zone!="channel"))
map2 <- basemapZoom + geom_point(aes(x = lon, y = lat, colour=zone,shape=element),alpha=0.5, data = bac.zoommap, size = 4) + geom_text(aes(label=transect),size=2.5, data = bac.zoommap) + colScale

# higher zoom map for Lyell meeting
map3 <- basemapZoom2 + geom_point(aes(x = lon, y = lat, colour=zone,shape=element),alpha=0.5, data = bac.zoommap, size = 8) + geom_text(aes(label=transect),size=4, data = bac.zoommap) + colScale

@

<<hyp_analysis,echo=FALSE, results='hide', cache=TRUE>>=
# no output from this block but kept for now as it calculates some useful results/objects.
# Might be suitable for output as small table or in text mention.

channel <- subset_samples(ab.rel,zone=="channel")
dune <- subset_samples(ab.rel,element=="air")
pan <- subset_samples(ab.rel,zone=="pan")

channel <- prune_taxa(taxa_sums(channel)>0,channel)
dune <- prune_taxa(taxa_sums(dune)>0,dune)
pan <- prune_taxa(taxa_sums(pan)>0,pan)

channel.ntaxa <- ntaxa(channel)
dune.ntaxa <- ntaxa(dune)
pan.ntaxa <- ntaxa(pan)

channel_dune.intersection <- intersect(taxa_names(channel),taxa_names(dune))
channel_dune.intersection.n <- length(channel_dune.intersection)  

channel_pan.intersection <- intersect(taxa_names(channel),taxa_names(pan))
channel_pan.intersection.n <- length(channel_pan.intersection)  

dune_pan.intersection <- intersect(taxa_names(dune),taxa_names(pan))
dune_pan.intersection.n <- length(dune_pan.intersection)  

all.intersection <- intersect(channel_dune.intersection, channel_pan.intersection)
all.intersection <- length(all.intersection)

# Now we have the intersections
# make objects for the pairs and subset to just the intersecting taxa
channel_pan <- subset_samples(ab.rel,zone%in%c("channel","pan"))
channel_pan <- prune_taxa(channel_pan.intersection, channel_pan)
otu_table(channel_pan) <- otu_table(channel_pan)/5

channel_dune <- subset_samples(ab.rel,zone%in%c("channel","dune"))
channel_dune <- prune_taxa(channel_dune.intersection, channel_dune)
otu_table(channel_dune) <- otu_table(channel_dune)/5

# create a common colour palette for plotting
# not used now but might be useful later
#allphyla <- unique(c(tax_table(channel_dune)[,"Phylum"],tax_table(channel_pan)[,"Phylum"]))
#allphyla.col <- as.vector(colorRampPalette(brewer.pal(12,"Set3"))(length(allphyla)))
#names(allphyla.col)  <- allphyla

# plot
#channel_pan.shared <- plot_bar(channel_pan,x = "zone",fill = "Phylum") + geom_bar(aes(color=Phylum, fill=Phylum), stat="identity", position="stack") + scale_y_continuous(name="relative abundance (%)") + scale_fill_manual("Phylum", values = allphyla.col) + scale_colour_manual("Phylum", values = allphyla.col)

# work out OTU abundance for those only in soil
ab.rel.earth <- subset_samples(ab.rel,element=="earth")
ab.rel.earth.merged <- merge_samples(ab.rel.earth,rep(T,nsamples(ab.rel.earth)))
otu_table(ab.rel.earth.merged) <- otu_table(ab.rel.earth.merged)/nsamples(ab.rel.earth)
inoc_source_otus <- unique(c(taxa_names(channel),taxa_names(dune)))

soil_only_otus.sum <- round(sum(otu_table(ab.rel.earth.merged)) -  sum(otu_table(ab.rel.earth.merged)[,inoc_source_otus]),1)

@

<<hyp_analysis2,echo=FALSE, results='hide', cache=TRUE>>=
# working with merged objects to reduce processing time
# air samples merged separately because of shared zone name
ab.rel.merged <- merge_samples(subset_samples(ab.rel,element!="air"),"zone")
ab.rel.merged.air <- merge_samples(subset_samples(ab.rel,element=="air"),"zone")
sample_names(ab.rel.merged.air) <- "dune flank air"
ab.rel.merged <- merge_phyloseq(ab.rel.merged,ab.rel.merged.air)

otu_table(ab.rel.merged) <- otu_table(ab.rel.merged)/5
ab.rel.merged.phylum <- tax_glom(ab.rel.merged,"Phylum")

# make dataframes for output later
t <- t(otu_table(ab.rel.merged))
t <- as.data.frame(t)
otus.df <- as.data.frame(cbind(tax_table(ab.rel.merged),t))
otus.df$OTU <- row.names(otus.df)
otus.order <- taxa_names(ab.rel.merged)[order(taxa_sums(ab.rel.merged),decreasing = T)]
otus.df <- otus.df[otus.order,]
# more info will be added to this DF after DESEQ analysis, then it will be output

# Keep only phyla that are > 1% overall:
# to simplify plot and protect against unequal sampling bias

keep_phyla <- (taxa_sums(ab.rel.merged.phylum)/5)>1
keep_phyla <- as.character(tax_table(ab.rel.merged.phylum)[keep_phyla,"Phylum"])
keep_phyla_index <- tax_table(ab.rel.merged)[,"Phylum"]%in%keep_phyla

ab.rel.merged <- subset_taxa(ab.rel.merged, keep_phyla_index)
sample_data(ab.rel.merged)$zone <- as.factor(sample_names(ab.rel.merged))

airborne <- subset_samples(ab.rel,element=="air")
airborne <- prune_taxa(taxa_sums(airborne)>0,airborne)
waterborne <- subset_samples(ab.rel.merged,zone=="channel")
waterborne <- prune_taxa(taxa_sums(waterborne)>0,waterborne)

airborne_otus <- taxa_names(airborne)[taxa_sums(airborne)>0]
waterborne_otus <- taxa_names(waterborne)[taxa_sums(waterborne)>0]
both_otus <- intersect(airborne_otus,waterborne_otus)

# make an object to be modified for plots
ab.rel.ea <- subset_samples(ab.rel,element!="air")
otu_table(ab.rel.ea) <- otu_table(ab.rel.ea)/5
bac.rel.ea <- subset_samples(bac.rel,element!="air")
otu_table(bac.rel.ea) <- otu_table(bac.rel.ea)/5

# keep only the more abundant phyla (for clarity of plots)
# this now done globally so shouldn't be needed here
# ab.rel.ea.p <- subset_taxa(ab.rel.ea, Phylum%in%top.phyla)
bac.rel.ea.p <- bac.rel.ea
ab.rel.ea.p <- ab.rel.ea

# same again but looking at cyanobacterial genera
bac.rel.ea.cyano <- subset_taxa(bac.rel.ea, Phylum == "Cyanobacteria")
# select only the more abundant cyanobacterial genera
bac.rel.ea.cyano <- subset_taxa(bac.rel.ea.cyano, Genus%in%top.cyano.genus)

airborne_otus <- airborne_otus[airborne_otus%in%taxa_names(ab.rel.ea)]
waterborne_otus <- waterborne_otus[waterborne_otus%in%taxa_names(ab.rel.ea)]
both_otus <- both_otus[both_otus%in%taxa_names(ab.rel.ea)]

@

<<deseq_analysis1, echo=FALSE, message=FALSE, results='hide', cache=TRUE>>=

  typedds = phyloseq_to_deseq2(expt.deseq, ~ element)
  typedds = DESeq(typedds, test="Wald",fitType="parametric")
  # DESeq docs:
  #On p-values:
  # By default, independent filtering is performed to select a set of genes for multiple test correction which will optimize the number of adjusted p-values less than a given critical value alpha (by default 0.1). The adjusted p-values for the genes which do not pass the filter threshold are set to NA. 
  
# When using local regression fit, the user should examine plotDispEsts(dds)
# to make sure the fitted line is not sharply curving up or down based on
# the position of individual points.

  #  plotDispEsts(typedds,main="model check")

  res = results(typedds, cooksCutoff = FALSE, alpha = 0.05)
  alpha = 0.05
  sigtab = res

  rel.abundance <- apply(otu_table(ab)[rownames(sigtab), ],FUN = max,MARGIN = 1)
  sigtab = cbind(as(sigtab, "data.frame"), as(tax_table(expt.deseq)[rownames(sigtab), ], "matrix"))
  sigtab$max <- rel.abundance
  
  # record the OTU name
  sigtab$otu <- row.names(sigtab)
  
  ind <- sigtab$padj<0.05&!is.na(sigtab$padj)
  sigtab$otu.orig <- sigtab$otu
  sigtab[ind,"otu2"] <- gsub(pattern = "OTU_",replacement = "",x = sigtab[ind,"otu"])
  sigtab[ind,"otu"] <- paste("*",sigtab[ind,"otu"],"*",sep="")

  sigtab$sig <- ""
  sigtab[ind,"sig"] <- "*"
  
  sd <- as.data.frame(sample_data(expt.deseq))

# manipulations to the sigtab table - for later output as graphics and tables

wanted_cols <- c("Phylum",  "Class",  "Order",	"Family",	"Genus",	"Species","padj","log2FoldChange")
wanted_rows <- (sigtab$padj<0.05) & (!is.na(sigtab$padj))
wanted_otus <- sigtab[wanted_rows,"otu.orig"]

expt.rel.df <- subset_samples(ab.rel.all, zone=="dune flank")
expt.rel.df.merged <- merge_samples(expt.rel.df,group = "element")
otus.rel.df.by.type <- otu_table(expt.rel.df.merged)/5

orbt <- as.data.frame(t(otus.rel.df.by.type))
sigtab.subset <- sigtab[wanted_rows,wanted_cols]
sigtab.subset <- cbind(sigtab.subset[wanted_otus,],orbt[wanted_otus,])
# add a difference column
sigtab.subset$diff <- sigtab.subset$earth - sigtab.subset$air

# Sanity check on the significant OTUs
sane <- sign(sigtab.subset$log2FoldChange)==(sign(sigtab.subset$diff))

# identify the removed OTUs (none)
# sigtab.subset.removed <- sigtab.subset[!sane,]

# remove the OTUs not passing sanity check (none)
# sigtab.subset <- sigtab.subset[sane,]

# order by diff
sigtab.subset <- sigtab.subset[order(sigtab.subset$diff,decreasing = T),] 
# save to file
# not needed any more - now added this to otus.df
# write.csv(sigtab.subset,"results/Elliott_Diamantina_Microbiome_Table_S3[wind].csv")

outcols <- c("Phylum",		"Family",	"Genus","padj","air",	"earth") #"Class",	"Order",
outtab <- sigtab.subset[1:10,outcols]
outtab2 <- sigtab.subset[nrow(sigtab.subset):(nrow(sigtab.subset)-10),outcols]
# there are NAs in outtab2 because of something wrong in working out the mean abundances
# as can be seen by this line
# otu_table(bac)["4397479"]
# otu_table(expt.rel.df.merged)["4397479"]
# there are observations for filter but it comes out as 0 in the table.
# This is just because the OTU "4397479" is present in bac used for deseq analysis, but not in the subset object.

otus_earth <- row.names(sigtab.subset[sigtab.subset$diff>0,])
otus_air <- row.names(sigtab.subset[sigtab.subset$diff<0,])

n_otu_earth <- length(otus_earth)
n_otu_air <- length(otus_air)

# record significant OTUs in otus.df for output file / table
otus.df$sig <- "-"
#otus.df[otus_earth,"sig"] <- "soil"
#otus.df[otus_air,"sig"] <- "air"
otus.df[row.names(otus.df)%in%otus_earth,"sig"] <- "soil"
otus.df[row.names(otus.df)%in%otus_air,"sig"] <- "air"
@


<<source_analysis1, echo=FALSE, message=FALSE, cache=TRUE>>=

colnames(tax_table(ab.rel.ea.p))[1] <- "source"
tax_table(ab.rel.ea.p)[,"source"] <- "not found in river"
tax_table(ab.rel.ea.p)[taxa_names(ab.rel.ea.p)%in%waterborne_otus,"source"] <- "found in river"

ab.rel.ea.p.top <- subset_taxa(ab.rel.ea.p, Phylum%in%top.phyla)

water_otus_plot <- plot_bar(ab.rel.ea.p.top,x = "zone",fill = "source")
water_otus_plot <- water_otus_plot + geom_bar(aes(color=source, fill=source), stat="identity", position="stack") 
water_otus_plot <- water_otus_plot + scale_y_continuous(name="relative abundance (%)") + scale_fill_brewer(palette = "Set1",direction = -1) + scale_color_brewer(palette = "Set1",direction = -1)
water_otus_plot <- water_otus_plot + facet_wrap(~Phylum, nrow=2) + theme(legend.position="bottom")

###

colnames(tax_table(bac.rel.ea.cyano))[1] <- "source"
tax_table(bac.rel.ea.cyano)[,"source"] <- "not found in river"
tax_table(bac.rel.ea.cyano)[taxa_names(bac.rel.ea.cyano)%in%waterborne_otus,"source"] <- "found in river"

water_otus_plot.c <- plot_bar(bac.rel.ea.cyano,x = "zone",fill = "source")
water_otus_plot.c <- water_otus_plot.c + geom_bar(aes(color=source, fill=source), stat="identity", position="stack") 
water_otus_plot.c <- water_otus_plot.c + scale_y_continuous(name="relative abundance (%)") + scale_fill_brewer(palette = "Set1",direction = -1) + scale_color_brewer(palette = "Set1",direction = -1)
water_otus_plot.c <- water_otus_plot.c + facet_wrap(~Genus, nrow=1, scales = "free") + theme(legend.position="bottom")

@

<<deseq_analysis2, echo=FALSE, message=FALSE, cache=TRUE>>=
# see whether OTU is easy or not to become airborne by wind tunnel treatment
# this is used later to make the wind_otus_plot
# using object ab.rel.ea.p - which has only the more abundant phyla

colnames(tax_table(ab.rel.ea.p))[1] <- "behaviour"
tax_table(ab.rel.ea.p)[,"behaviour"] <- "intermediate"
tax_table(ab.rel.ea.p)[row.names(tax_table(ab.rel.ea.p))%in%otus_air,"behaviour"] <- "easily becomes airborne"
tax_table(ab.rel.ea.p)[row.names(tax_table(ab.rel.ea.p))%in%otus_earth,"behaviour"] <- "resists becoming airborne"

ab.rel.ea.p.top <- subset_taxa(ab.rel.ea.p, Phylum%in%top.phyla)

colnames(tax_table(bac.rel.ea.cyano))[1] <- "behaviour"
tax_table(bac.rel.ea.cyano)[,"behaviour"] <- "intermediate"
tax_table(bac.rel.ea.cyano)[row.names(tax_table(bac.rel.ea.cyano))%in%otus_air,"behaviour"] <- "easily becomes airborne"
tax_table(bac.rel.ea.cyano)[row.names(tax_table(bac.rel.ea.cyano))%in%otus_earth,"behaviour"] <- "resists becoming airborne"

deseq_n_test <- nrow(sigtab)

deseq_sum_diff_air <- sum(sigtab.subset[sigtab.subset$diff<0,"diff"] )
deseq_sum_diff_earth <- sum(sigtab.subset[sigtab.subset$diff>0,"diff"] )
deseq_n_diff_air <- length(sigtab.subset[sigtab.subset$diff<0,"diff"] )
deseq_n_diff_earth <- length(sigtab.subset[sigtab.subset$diff>0,"diff"] )

@

<<source_analysis2, echo=FALSE, message=FALSE, cache=TRUE>>=
###

wind_otus_plot <- plot_bar(ab.rel.ea.p.top,x = "zone",fill = "behaviour")
wind_otus_plot <- wind_otus_plot + geom_bar(aes(color=behaviour, fill=behaviour), stat="identity", position="stack") 
wind_otus_plot <- wind_otus_plot + scale_y_continuous(name="relative abundance (%)") + scale_fill_brewer(palette = "Accent") + scale_color_brewer(palette = "Accent")
wind_otus_plot <- wind_otus_plot + facet_wrap(~Phylum, nrow=2) + theme(legend.position="bottom")

###

wind_otus_plot.c <- plot_bar(bac.rel.ea.cyano,x = "zone",fill = "behaviour")
wind_otus_plot.c <- wind_otus_plot.c + geom_bar(aes(color=behaviour, fill=behaviour), stat="identity", position="stack") 
wind_otus_plot.c <- wind_otus_plot.c + scale_y_continuous(name="relative abundance (%)") + scale_fill_brewer(palette = "Accent") + scale_color_brewer(palette = "Accent")
wind_otus_plot.c <- wind_otus_plot.c + facet_wrap(~Genus, nrow=1, scales = "free") + theme(legend.position="bottom")
@

% There is mounting evidence that microscopic organisms have profound geomorphic influence at landscape scale \citep[e.g.][]{beraldi2013early,ferrenberg2017biological,chen2017stabilizing}. Pressures of climate change and land use change coupled with growing human populations require informed landscape management strategies to protect resources and minimise adverse impacts. Currently such strategies include actions such as controlling grazing and vegetation, but rarely are microbial players actively managed. We suggest that this is because the actions of microbes are less easily regognised, and we believe that recent advances in molecular microbial ecology provide an opportunity to address this.

In drylands where vegetation is sparse, soil microbes provide some ecosystem functions that are normally delivered by plants in temperate vegetated systems \citep[e.g.][]{lange1992taxonomic, ferrenberg2017biological}. Microbes in the soil surface of drylands form biological soil crusts (biocrusts) which contribute to stabilisation of the soil, fixing of carbon and nitrogen, and affecting hydrology \footnote{Tom you commented that hydrology topic could be expanded and that you have suitable refs for this. Please advise or draft adjustment}. These biotic processes are inter-linked with abiotic processes, and they are the direct result of microbial community physiology, ecology, and evolution over billions of years. Biocrusts were likely among the first organisms to colonise the land \citep{beraldi2013early}, therefore it is to be expected that they are well adapted to both exploit soil surface conditions, and modify their environment for their own advantage. This process of ecosystem engineering is widely recognised in larger organisms such as beavers \citep{jones1994organisms}, and although microbes are less visible they still have the ability to modify surface characteristics and processes on a very large scale \citep[e.g][]{viles2008understanding,chen2017stabilizing}. By identifying and understanding the adaptations of biocrust organisms, we can gain insight into the biotic forces that influence geomorphic processes, which has potential applications in landscape management e.g. for promotion of stability and preservation of soil fertility. It has long been recognised that cyanobacteria in biocrusts perform a stabilising function in drylands by secreting sticky extracellular polysaccharides (EPS) and because of their filamentous structure \citep{belnap1998vulnerability,ferrenberg2017biological}. In this paper we investigate how individual microbial taxa contribute to landscape stability or dust production in drylands and we relate this to hypothesised ecological strategies for survival.

%% consider deletion or move
% Biocrusts have been estimated to cover approximtely 50 \% of the soil surfaces in drylands \citep{garcia2003estimates}, where the relative lack of plants means that the role of microbes in environmental processes is likely to be proportionally higher than in termperate systems. Considering that drylands account for 45 \% of the Earth land area \citep{pruavualie2016drylands}, and this is expected to expand in the context of climate change scenarios \citep{huang2016accelerated} it seems crucial that understanding the most poorly characterised cover type on Earth, biocrusts (Ferrenberg et al., 2017) be improved. In particular it is important to increase understanding of the relationship of these organisms to stability of the landscape.”

% from Tom: expalin the extraordinary study site. DLNP has to be one of the best places in the world to study/test this hypothesis. Huge, ephemeral river system to bring in water, sediment and microbes.  Large pans for the water, seds and microbes to settle out and for crusts to form. They are also great places for wind to interact with sediment - thus forming the fringing dunes. The bigger landscape setting and the microbial-geo-hydro model are pretty integral to this paper and we could really make this clear in the intro. - would here be the best place for this?}

Landscape processes in some drylands are driven by the interplay between fluvial and aeolian processes \citep{mctainsh1987desert,bullard2003aeolian,field2009toward,langford1989fluvial}. \citet{mctainsh1987desert} proposed a model of sediment recycling that provides an integrated interpretation of fluvial and aeolian processes in the landscape. Rivers feeding internal draining endorheic basins such as Lake Chad, the Tarim Basin, and Lake Eyre transport vast amounts of fine sediments, feeding the large dust source regions out of Africa, China and Australia respectively. By linking river systems and wind erosion, McTainsh proposed looking at dryland geomorphology as a system of sediment recycling operating on a massive scale. Microbiota have not generally been considered as a component of this fluvial - aeolian interaction, however it has been noted by \citet{viles2008understanding} and others that biocrust organisms actively modify their environment sufficiently that they can be called ecosystem engineers, so consequently the role of biocrusts in large scale desert geomorphology is an area requiring further research activity. 

The extent to which “microscopic ecosystem engineers” contribute to interacting fluvial-aeolian geomorphic processes is explored in this paper. We investigated the microbial ecology of a dynamic dryland ecosystem in central Australia comprising a 25 km$^{2}$ claypan in the Lake Eyre basin bounded by sand dunes and ephemeral river channels. Fluvial-aeolian interactions are thought to have been occurring at least since the last glacial maximum about 10 kBp \citep{bullard2003aeolian}. Before that structural controls for internally draining rivers were established 20 –- 24 Ma \citep{simon1996evolution}, establishing an enclosed geological system of fluvial-aeolian sediment movement that has persisted in some form to the present day. 

Sediment recycling in fluvial-aeolian systems like this results in a constantly changing soil surface, presenting considerable challenges to life. Due to the long term existence of the Lake Eyre basin system and others like it, we expect that microscopic organisms will have evolved adaptations to survive and exploit predictable system features. To investigate this, we characterised the microbial communities in different parts of the system including sand dunes, pan sediments, river channels, and airborne dust liberated by \textit{in-situ} wind tunnel treatment on sand dunes. By comparing the microbial composition of different system parts we evaluated possible dispersal strategies of individual taxa and interpreted these findings in relation to landscape stability and dust production.

A conceptual model shown in Figure \ref{fig:dispersal-hazards} illustrates interactions between fluvial and aeolian processes in the Lake Eyre basin, and some of the threats and opportunities these present for the microbiota. Broadly speaking, microbes living in ephemeral lakes may be adapted for life in water, life on land, or for a life-cycle incorporating adaptations for both situations. Organisms with these (and other) different life strategies can co-exist in the same system, occupying different niches through different adaptations - some of which may have geomorphic relevance, for instance promotion of landscape stability or promotion of dust production. Microbial modification of the environment in this way my relate to a survival outcome for the microbe and thus would be positively seleceted by natural selection, or it may be incidental and not directly influence survival prospects. 

\begin{figure}
  \centering
      \includegraphics[width=1\textwidth]{figure/dispersal-hazards-fig}
  \caption{Photographs of main system compartments with identification of key hazards for microbial life and illustration of expected linkages by Aeolian and hydrological transfer of sediment and inoculum. In our sampling the sand dune flank samples belong to the sand dune compartment and the nebkha samples belong to the claypan compartment - these are expected to be subject to the same respective hazards and linkages.}
  \label{fig:dispersal-hazards}
\end{figure}

Very little is known about the microbial ecology of ephemeral lakes, especially during the dry phase when the lake is a pan and an active dust source. Previous research has understandably focused on photosynthetic microbes especially cyanobacteria which create blooms during flooding \citep[e.g.][]{woodhouse2016microbial}, and there has also been some interest in the anaerobic microbiota of sediments under water \citep[e.g][]{wright1999role}. To our knowledge, however, there has been no previous attempt to describe the microbial community of a dry ephemeral lake bed using a high-throughput DNA sequencing approach such as used here. We are also not aware of any attempt to identify microbes which inhabit different compartments of a fluvial-aeolian sediment transport system, or microbes associated with aeolian dust released from dryland soils. 

% from Craig:
% Understanding erodibility, and hence sediment entrainment, within the claypan provides important insights into the triggers that occur through time responsible for landscape stability, and also the crucial triggers of cyanobacterial filament entrainment and possible transportation. At a broad scale, this paper seeks to identify if the microbial ecology of a dynamic dryland ecosystem in central Australia spatially structure differently according to the various geomorphic drivers and sediment types interacting within this field setting. 

A few previous studies have been published on the microbial content of airborne sediments, and a recent review by \citet{acosta2015microbiology} identified that most of these focused on long-distance transport and the movement of pathogens of relevance to human health. The only previous study to our knowledge combining use of a wind tunnel with collection of airborne particles and their microbial charaterisation is \citet{gardner2012pyrosequencing}, which used a pyrosequencing approach to determine the association of particular microbes or diversity metrics with organic matter and particle size fractions. Prior to wind-tunnel treatments the soils in \citet{gardner2012pyrosequencing} were tilled, raked, and flattened using a lawn roller, in contrast to the present study which used soils in their natural condition. 

To increase understanding of the relationship of biocrusts to landscape stability we aimed to firstly gain basic knowledge about the microbial composition of different dryland soils and sediments, and secondly to identify possible ecological strategies (especially dispersal mechanisms) of microbes present. By developing hypotheses about the ecology of communities and their constituent individual taxa (see Figure \ref{fig:dispersal-hazards}), it should be possible to predict responses for various scenarios and devise interventions to manipulate the biotic component of geomorphic processes. Such an achievement would have a wide range of potential applications in land management, for example to enhance carbon storage, suppress dust emissions, and make predictions of future geomorphic activity under a changing climate.

Our main hypothesis is that different compartments in fluvial-aeolian systems harbour distinctive microbial communities, and that wind erosion selectively mobilises certain taxa from biocrusts. We expect this because it is known that biocrusts stabilise dryland soils and this stability is clearly linked to their survival prospects because shifting soils will prevent the establishment and succession of biocrusts. So whilst we expect a strong selective pressure in biocrust organisms for promotion of soil stability, it should be recognised that this is a benefit to the community not just the individual taxa which invest considerable resource to stabilise the soil (e.g. EPS production). Microbes may employ a variety of strategies in biocrusts and we expect that they include the full range of symbioses from mutualism to parasitism although these are not directly evaluated in the present work. If some biocrust organisms are exploiting community resources rather than contributing to the biocrust survival, then it can be hypothesised that such taxa may reduce soil stability and contribute to dust production. If this is the case then one might expect these "cheaters" to be adapted for dispersal on the produced dust so that they can rapidly find new biocrusts to exploit after depleting resources in a particular location.

To test our hypothesis we used high throughput DNA sequencing of the 16S rRNA gene from soils and sediments to determine the bacterial community members and relative abundances in the drylands of central Australia. In addition, we applied an experimental wind tunnel treatment to simulate the effect of wind upon sand dunes stabilised by biocrusts. By collecting the mobilised dust and comparing the dust community with that of other samples, we were able to identify possible dispersal strategies and suggest the relevance of these to sediment stability and dust production. 

\section{Methods}
\label{sec:meth}

\subsection{Field site and sampling}
Work was carried out at Diamantina National Park of western Queensland, Australia (23$^{\circ}$36'44.8" S; 143$^{\circ}$17'46.9" E). Climate in the region is semi-arid characterised by a summer-dominant rainfall pattern with a mean annual precipitation of 270 mm a$^{-1}$ and high inter-annual variation. Central to the study site is 25 Km$^{2}$ of claypan, bordered by sand dunes and multiple channels of the Diamantina river. Aeolian activity moves sediment from the dunes and claypan, whilst periodic flooding (interval approx. 3 years\footnote{Craig can you add any more on e.g. typical inundation frequencies/depths? }) brings fresh river sediment to the claypan. A linear dune runs approximately north-south on the west side of the claypan, and this was the location from which sampling transects were established (Figure \ref{fig:map}). Soil was sampled from five transects perpendicular to the dune, and water/sediment was sampled from water channels in the surrounding area. Transects were approximtely 500 m apart to achieve even spacing along the dune extending into the claypan, however some zones to the south were impacted by unauthorised cattle trampling. These impacted zones were excluded because the dune flanks did not have suitable areas to run the wind tunnel. The transects are numbered 1-5 starting at the south end. 

Biological crusts on the claypan typically have a blue-green appearance due to the presence of photosynthetic organisms, and they have a platelet formation caused by physical weathering. Blue-green biological crusts were common but not the dominant surface cover on the claypan as a whole. They appear to have once been widespread but had been largely eroded away at the time of sampling in July 2015. Underneath claypan platelet crusts was a similar looking crust which is brown (Figure \ref{fig:photo-area}), and this was not sampled in the present study. In many places on the claypan there were nebkha on top of the blue-green platelet crusts and the platelets ran up the nebkha slightly, however the crusts on nebkha had distinct morphology. Platelet crusts appeared to be more common near nebkha. 

% Floods supply fresh sediment and inoculum to the claypan, having the effect of ‘re-zeroing’ the claypan surface, burying the effects of previous biotic and wind erosion activity. Once dry, however, time and climatic variation impact the sediment creating a mosaic of surface types. Source boarding sand dunes provide new sediments, larger, heavier particles that have the capacity to break down fine crusts as they bounce across the claypan. 

<<map_out, echo=FALSE,fig.height=3.8, warning=FALSE, cache=TRUE,fig.cap='Sampling locations. a. General area of claypan including surrounding water channels; b. Closer view of dune and pan area.\\label{fig:map}'>>=

#Extract Legend 
g_legend<-function(a.gplot){ 
  tmp <- ggplot_gtable(ggplot_build(a.gplot)) 
  leg <- which(sapply(tmp$grobs, function(x) x$name) == "guide-box") 
  legend <- tmp$grobs[[leg]] 
  return(legend)} 

maplegend <- map1 + theme(legend.position="bottom")
legend <- g_legend(maplegend) 


grid.newpage()
pushViewport(viewport(layout = grid.layout(6, 2)))
vplayout <- function(x, y) viewport(layout.pos.row = x, layout.pos.col = y)
print(map1 + theme(legend.position="none"), vp = vplayout(1:5, 1))  # key is to define vplayout
print(map2 + theme(legend.position="none"), vp = vplayout(1:5, 2))
legend$vp = viewport(layout.pos.row = 6, layout.pos.col = 1:2)
grid.draw(legend)

@

The following locations were targeted for soil sampling (See Figures \ref{fig:photo-area} and \ref{fig:photo-sampling} for photographs; Table S1 for coordinates):

\begin{enumerate}
\item [D.] Dune. Dune biocrust
\item [DF.] Dune flank. Dune flank biocrust. These sites were also used for wind tunnel experiments.
\item [N.] Nebkha. Biocrust situated on nebkha in the claypan to the east of the dune
\item [P.] Pan. Biocrust platelets in vegetation interspaces on the claypan
\item [C.] River channel. Sediment and water from the banks of water channels surrounding the dune/claypan complex
\end{enumerate}

Sampled areas were 10x10 cm and were selected to represent the diversity of biocrust in the area (Figure \ref{fig:photo-sampling}). The sampled depth was dictated by coherence of the biocrust. The area was cut out with a sterile scalpel and the crust was lifted off the looser soil beneath. In areas where differing crust types were noticeable the sampled areas were selected to have both types where applicable (e.g. biological/physical, light/dark). In visibly disturbed areas e.g. from animals, un-disturbed crusts were selected (2x dune sites).

\begin{figure}
  \centering
      \includegraphics[trim = 17mm 10mm 17mm 10mm, clip,width=1\textwidth]{figure/photos1}
  \caption{Photographs of sampling areas including detail of nebkha and crust formations on the claypan.}
  \label{fig:photo-area}
\end{figure}

\begin{figure}
  \centering
      \includegraphics[trim = 17mm 10mm 17mm 10mm, clip,width=1\textwidth]{figure/photos2}
  \caption{Photographs taken during soil sampling, showing cuts made to 10 cm x 10 cm square areas of crust prior to lifting aseptically.}
  \label{fig:photo-sampling}
\end{figure}

Three distinct surface layers were observed in the claypan:\footnote{Craig please could you check if these descriptions are accurate and easy enough to understand?}
\begin{enumerate}
\item deeper layer brown crust, possibly older buried crust (not sampled)
\item platelet formation surface biological crust (sampled: code P)
\item nebkha (sampled: code N)
\end{enumerate}

In dune areas (both dune and dune flank) there was widespread continuous biological soil crust which held together the sand surface, with some variation in pigmentation. Sometimes there were deeper (probably buried) crust layers, but only surface exposed crusts were sampled in this work.

Water channels were sampled by dragging a 5 ml sterile bijoux along the bank where it meets the water. Approximately 2.5 ml bank sediment and 2.5 ml standing water were collected in each bottle from the same location. 

\subsection{Wind tunnel}
In addition to the soil crust samples a wind tunnel was run on the dune flank zones within 3 m of where the soil crust sampling was done. The wind tunnel was a duct-type design equipped with a sediment collection trap in which sterile Whatman xx\footnote{need filter paper type} filter papers were fitted. Further details of design and operation have previously been described by \citet{strong2016development}\footnote{should provide a bit more detail here}. The wind tunnel was run on dune flanks because they are naturally eroded by saltation impact from the dune sand above when it is windy, therefore the potential as an inoculum source to surrounding areas is greater than the higher dune areas. At each site the wind tunnel was run three times on adjacent areas for 5 minutes at 10 m/s for 5 minutes\footnote{Helene collected data to determine the exact run conditions}. This is a moderate treatment intended to be realistic and to collect only the easily mobilised particles / organisms. 

\subsection{Soil properties}
Total carbon and nitrogen content of soils were determined using a CN element analyser (Leco TruSpec). Particle size distribution was determined using a Beckman-Coulter LS280 laser-sizer in the range 0.375-2000 um with 93 class intervals. 
Soil pH was determined using a Sartorius PY-P10 probe after mixing soil in water according to the method of \citet{rowell1994soil}, scaled down for small samples. 

\subsection{DNA sequencing}
DNA was extracted in the field within 24 hours of sample collection using a Mobio Powersoil DNA extraction kit according to the manufacturers instructions except for increasing the soil amount slightly to 0.4 g per sample as described previously \citet{elliott2014niche}. DNA sequencing targeted the prokaryotic 16S rRNA gene and was performed by the Centre for Genomic Research (CGR) NERC Facility at the University of Liverpool, using an Illumina MiSeq in a Paired end approach. 

\subsection{Bioinformatics}
FastQ files from DNA sequencing were processed using QIIME \citep{caporaso2010qiime} with default settings. Clustering used the UCLUST method \citep{edgar2010search} and open-reference OTU picking using the Greengenes 16S rRNA reference database \citep{desantis2006greengenes}. OTUs were defined at 97 \% similarity which typically approximates to species-level differences \citep{stackebrandt1994taxonomic}. 

\subsection{Statistics}
OTU tables with assigned taxonomy from the bioinformatics pipeline were analysed using R \citep{R2017} with the phyloseq \citep{mcmurdie2013phyloseq} and vegan packages \citep{oksanen2017vegan}. Sequences identified as originating from chloroplasts or  mitochondria were removed. Microbial community structure was visualised with respect to sample source using nonmetric multidimensional scaling (NMDS) based on Bray-Curtis dissimilarity. Possible correlation of measured variables with microbial community structure was evaluated by stepwise model building to select constraining variables for redundancy analysis (RDA), and using anova to test significance of variables to the model. All measured variables were used in this process including pH, C, N, and a range of particle size parameters (variables are available in supp....). Alpha diversity was evaluated by calculating the Chao1 richness estimate for each sample.

To identify OTUs with different relative abundance in the wind tunnel collection filter compared to the source dune flank soil we used the model based DESeq2 approach \citep{love2014moderated} as implemented in the phyloseq package. The DESeq2 technique includes correction for multiple testing and is highly efficient on data preservation because it eliminates the need to perform rarefaction or normalization of count data \citep{mcmurdie2014waste}. 

\section{Results}
\label{sec:res}

\subsection{Soil properties}
abc (Figure \ref{fig:chem})


<<chem_out, echo=FALSE,fig.height=2,fig.cap='Mean total carbon and nitrogen (percent w/w) and pH of soil in and around Lake Constance claypan at Diamantina National Park, Australia. Error bars represent the standard error of the mean for each soil type (n = 5). Sample coding for soil types: D sand dune, DF dune flank, N nebkha, P pan surface. \\label{fig:chem}', warning=FALSE, cache=TRUE>>=
# melt for plotting
chemistry.melt <- melt(sample_data(bac.earthair),measure.vars=c("N_pc","C_pc","pH"))
chemistry.melt$variable <- revalue(chemistry.melt$variable, c("N_pc"="nitrogen", "C_pc"="carbon"))
# make the plot 
print(ggplot(chemistry.melt, aes(x=zone_code, y=value, shape=element, colour=zone)) + stat_summary(fun.data = "mean_se", size=1) + facet_wrap(~variable,scales = "free_y",nrow=1) + labs(x = "zone") + theme(aspect.ratio=1,legend.position="none") + colScale )
@

The dune and dune flank sediments are very similar in terms of particle-size distribution.  They are dominated by very fine to fine sands (46-48 \%) with the modal particle-size in the range 185-270 $\mu$m.  The dune sediments are slightly coarser overall containing a higher percentage of medium and coarse sands and lower percentages of silt and clay than the dune flank (Table \ref{tab:partsize}).  The nebkha are also dominated by very fine to fine sands (60 \%; mode 169-185 $\mu$m) and the sediments are well-sorted.  The pan sites are less well sorted than any of the sites dominated by wind-blown material.  The pan sites include the highest proportion of clay (c. 10 \%) and silts (55 \% total) and lowest percentage of sand-sized material ($<$35 \%).

<<partsize_out, echo=FALSE, cache=FALSE>>=


partsizetab <- data.frame(Clay = c(4.39,5.10,3.98,9.52),
                          Very_fine_silt = c(7.59,8.67,6.99,21.10),
                          Medium_coarse_silt = c(17.95,23.82,17.09,35.54),
                          Very_fine_fine_sand = c(48.17,46.33,59.72,28.70),
                          Medium_coarse_sand = c(21,16.09,12.23,6.14),
                          row.names = c("Dune","Dune Flank","Nebkha","Pan"), check.names="F")

kable(partsizetab, digits=2, caption = "Mean pc volume of sediment by size class for all sites. \\label{tab:partsize}")
@



\subsection{DNA sequencing and general microbial community composition}
DNA was successfully extracted and sequenced from \Sexpr{nsamples(bac)} samples, yielding \Sexpr{round(m_reads_total,1)} million reads. After pairing reads and quality control \Sexpr{round(m_reads_bac_qc,1)} million bacterial reads and \Sexpr{round(m_reads_arc_qc,1)} million archaeal reads remained. 

There were \Sexpr{ntaxa(bac)} bacterial OTUs and \Sexpr{ntaxa(arc)} archael OTUs detected (OTUs defined by 97 \% similarity which is approximately species-level difference between OTUs). This analysis was planned to target bacteria, and as archaea make up only \Sexpr{round(100*m_reads_arc_qc / m_reads_bac_qc,1)} \% of sequences bacteria are the main focus of this paper, however archaeal reads were retained for the analyses. The Chao1 species richness estimation (Figure \ref{fig:community}) indicated in excess of 10,000 species present in each sample, without striking differences with respect to sample type. 

Initial visualisation of microbial community structure by NMDS (Figure S1) showed that the water samples differed extremely compared to all other samples, and this dominated all other inter-sample differences. The water samples were therefore excluded from the NMDS to reveal the community stucutre relationships between the soil and airborne dust samples as shown in Figure \ref{fig:community}. The dune community differs from the dune flank community, and the airborne dust community derived from the dune flank differs from the dune flank community. Notably the dust community has a community structure approximately intermediate between the dune and nebkha communities.

<<community_out, echo=FALSE,fig.height=3.8, warning=FALSE, cache=TRUE,fig.cap='Prokaryotic microbial community diversity of soil (sand dunes, claypan, and nebkha), water channels, and airborne dust (from sand dunes). a. Chao1 richness estimate (alpha diversity); b. NMDS visualisation of community structure (beta diversity). Water channel data are not shown on the NMDS plot because of extreme dissimilarity to other samples.\\label{fig:community}'>>=

#Extract Legend 
g_legend<-function(a.gplot){ 
  tmp <- ggplot_gtable(ggplot_build(a.gplot)) 
  leg <- which(sapply(tmp$grobs, function(x) x$name) == "guide-box") 
  legend <- tmp$grobs[[leg]] 
  return(legend)} 

communitylegend <- rich.ab + colScale + theme(legend.position="bottom")
legend <- g_legend(communitylegend) 


grid.newpage()
pushViewport(viewport(layout = grid.layout(6, 2)))
vplayout <- function(x, y) viewport(layout.pos.row = x, layout.pos.col = y)
print(rich.ab + colScale + labs(y="number of OTUs", x="zone") + theme(legend.position="none",strip.background = element_blank(),
  strip.text.x = element_blank()), vp = vplayout(1:5, 1))  # key is to define vplayout
print(NMDS.ab + colScale + theme(legend.position="none"), vp = vplayout(1:5, 2))
legend$vp = viewport(layout.pos.row = 6, layout.pos.col = 1:2)
grid.draw(legend)

@

Microbial composition of the dune flank, pan, and nebkha sites varied considerably (Figure \ref{fig:community}). High abundance OTUs are likely to be important contributors to the overall community differences (Figure \ref{fig:community}), and many of the abundant OTUs were related to each other (Table S4), meaning that community differences may be evident and easier to understand at higher taxonomic ranks (Figures \ref{fig:waterp} and \ref{fig:waterc}). A total of \Sexpr{length(unique(tax_table(bac)[,"Phylum"]))} bacterial phyla and \Sexpr{length(unique(tax_table(arc)[,"Phylum"]))} archael phyla were identified. Frequencies of the most abundant phyla are shown in Figure \ref{fig:waterp} and are provided in full in Table S2. It can be seen that water channel samples are dissimilar to the soil-based samples at the phylum level although all abundant phyla are represented in both soil and water. \textit{Cyanobacteria} numerically dominate at the experimental sites (about 28 \% of prokaryotic sequences), the overall cyanobacterial abundance at different sites is similar, and the dominant cyanobacterial genus is \textit{Phormidium}. The degree of dominance by \textit{Phormidium} varies, and differences in less abundant cyanobacterial genera (e.g. \textit{Leptolyngbya} and \textit{Microcoleus}) are evident between sites. For example \textit{Microcoleus} is the second most abundant identified cyanobacterial genus, being quite abundant in dune flank (2.1 \%) and claypan (3.1 \%) but rare in nebkha (0.2 \%). \footnote{could note here potential limitations of cyanobacterial taxonomy and ID etc.}

\begin{figure}[]
\begin{center}
<<OTU_source_out, echo=FALSE,fig.height=8.5, cache=FALSE>>=
grid.newpage()
pushViewport(viewport(layout=g0))
print(water_otus_plot,vp=vp.01)
print(wind_otus_plot,vp=vp.02)
@
\caption{ Relative abundance of bacterial phyla. Source indicates whether individual OTUs (97 pc similarity) contributing to the phylum abundance were found in the water channels (top), and whether they were significantly more abundant in soil or wind tunnel collectors at the dune flank sites (bottom). Only the 10 most abundant phyla are shown for clarity (full data provided in Table S2). Source designation is not indicative of the ultimate origin of OTUs.}\label{fig:waterp}
\end{center}
\end{figure}

\begin{figure}[]
\begin{center}
<<OTU_source_out2, echo=FALSE,fig.height=5.5, cache=FALSE>>=
grid.newpage()
pushViewport(viewport(layout=g0))
print(water_otus_plot.c,vp=vp.01)
print(wind_otus_plot.c,vp=vp.02)
@
\caption{ Relative abundance of cyanobacterial genera. Source indicates whether individual OTUs (97 pc similarity) contributing to the genus abundance were found in the water channels (top), and whether they were significantly more abundant in soil or wind tunnel collectors at the dune flank sites (bottom). Unidentified OTUs were excluded. Only the 5 most abundant identified cyanobacterial genera are shown for clarity (full data provided in Table S3). Source designation is not indicative of the ultimate origin of OTUs.}\label{fig:waterc}
\end{center}
\end{figure}


\subsection{Dispersal of microbes by fluvial and aeolian processes}
Of the \Sexpr{pan.ntaxa} OTUs found in the claypan, \Sexpr{channel_pan.intersection.n} were also found in water channels, \Sexpr{dune_pan.intersection.n} were found in airborne dust from the dune flank, and \Sexpr{all.intersection} were found in both of these hypothesised inoculum sources. The dune and channel samples had \Sexpr{channel_dune.intersection.n} OTUs in common. Whilst many soil OTUs were not detected in the air or water samples, most of the OTUs found exclusively in soil were rare, collectively accounting for only \Sexpr{soil_only_otus.sum} \% of reads in the soil samples. 

The DESeq2 model based approach \citep{love2014moderated} was used to identify OTUs with significantly different abundance between the dune flank soil and the collected dust derived from the same soil obtained by wind tunnel treatment. This technique avoids problems associated with uneven sampling effort and includes correction for multiple testing. The results (adjusted p \textless 0.05) indicated that \Sexpr{deseq_n_diff_air} OTUs (\Sexpr{round(deseq_sum_diff_air*-1)} \% of reads) were  more abundant in air samples and \Sexpr{deseq_n_diff_earth} OTUs (\Sexpr{round(deseq_sum_diff_earth)} \% of reads) were more abundant in soil samples. Details for a subset of these OTUs are shown in Tables \ref{tab:deseq1} and \ref{tab:deseq2} (extended results provided in Table S4). Results from these analyses were used to colour code Figures \ref{fig:waterp} and \ref{fig:waterc} to indicate whether taxa easily become airborne under windy conditions or not. It can be seen from these figures that some phyla and particualr cyanobacterial genera differed in abundance between the dune flank soil and airborne dust liberated from the same soil by wind tunnel treatment. For example phylum \textit{Cyanobacteria} was more abundant in soil (resist becoming airborne) and phylum \textit{Firmicutes} was more abundant in dust (easily become airborne). Detail of cyanobacterial distribution given in Figure \ref{fig:waterc} shows that \textit{Phormidium} and \textit{Microcoleus} species were more common in soil compared to airborne dust, whereas \textit{Leptolyngbya} species were more common in the soil compared to dust. \textit{Nodularia} species and \textit{Synechococcus} species were more common in water channels compared to soils, whilst the other genera mentioned were more common in soils. 


<<deseq_out1, echo=FALSE, cache=FALSE>>=
kable(outtab, digits=2, caption = "OTUs which are significantly more abundant in soil compared to dust blown from the soil. This is a subset of the DESeq2 analysis output, sorted by magnitude of difference in abundance. The padj column indicates the probability of the observed distribution including adjustment for multiple testing. Extended results are provided in Table S4. \\label{tab:deseq1}")
@


<<deseq_out2, echo=FALSE, cache=FALSE>>=
kable(outtab2, digits=2,caption = "OTUs which are significantly more abundant in dust blown from the soil compared to the whole soil. This is a subset of the DESeq2 analysis output, sorted by magnitude of difference in abundance. Extended results are provided in Table S4. \\label{tab:deseq2}")
@

<<deseq_out3, echo=FALSE, cache=FALSE>>=
otus.df.out <- otus.df[1:10,c(rank_names(ab)[c(2,6)],sample_names(ab.rel.merged)[-6],"sig")]
colnames(otus.df.out)[colnames(otus.df.out)=="dune flank"] <- "d.flank"
# class(otus.df$dune)
# otus.df.out <- as.numeric(otus.df.out)
kable(otus.df.out,row.names = F, digits=2,caption = "OTU abundance in each sampling zone (pc), in order of overall abundance. Sig column indicates if the OTU was significantly more abundant in soil or wind tunnel filter after running the wind tunnel on dune flanks. The 10 most abundant OTUs overall are shown, extended results are provided in Table S4. \\label{tab:deseq3}")
@

<<deseq_out4, echo=FALSE, cache=FALSE>>=
# merge otu differential abundance results from sigtab into otus.df for supplementary data
wanted_cols <- colnames((sigtab))[1:6]
otus.df <- merge(otus.df,sigtab[,wanted_cols],by="row.names",all.x=TRUE)
#colnames(otus.df)[1] <- "OTU"

# to make output clearer, properly identify which samples the deseq2 analysis was not done on:
otus.df$sig[is.na(otus.df$baseMean)] <- NA

write.csv(otus.df[,c("OTU",rank_names(ab)[1:7],sample_names(ab.rel.merged),"sig",wanted_cols)],"results/Elliott_Diamantina_Microbiome_Table_S4[otus].csv",row.names=FALSE)
@



\section{Discussion}
\label{sec:disc}

%% probably use parts of this in discussion:
% Landscape stability and dust production are geomorphic processes of the utmost importance to communities in drylands and neighbouring areas who face hazards such as dust storms and mobile sand dunes. An unstable landscape is one where soil surface protection is no longer in place. Loss of vegetation, biocrusts and soil surface disturbance increases the susceptibility to wind or water erosion (Webb and Strong, 2011\footnote{need citation}). Loss of soil at the site of erosion degrades the soil physical and chemical properties \citep{leys1994soil,leys2002erosion}. Transport of dust can result in downwind impacts to other terrestrial \citep{kohfeld2001dirtmap,reheis1995quaternary,herrmann1996identification,cattle2002aeolian}, aquatic \citep{leys1999dust,beraldi2013early}, marine \citep{Jickells2001atmospheric,shinn2000african} and human populations \citep{tozer2013dust,huszar1986estimating,miri2009environmental,saxton1995wind}. Further afield from dust sources there are continental scale impacts including the movement of sediment which is of geomorphic significance, and the movement of nutrients which can have effects on the biota of water bodies in particular \citep{beraldi2013early}.

We have performed the first detailed characterisation of the prokaryotic community in a lanscape-scale dryland system comprising terrestrial, fluvial, and aeolian components. This is also one of only a few studies to identify the microbes present in the central Australian drylands which cover over xx\footnote{can anyone provide figure and ref?} KM$^{2}$. Whilst determining the microbial composition of soils was not our primary objective, our results in addition to those presented are available in sequence databases\footnote{DE to submit sequences and provide accession numbers in results section} and supplementary materials. This is an important contribution as such data on drylands are lacking in the global literature \citep{ferrenberg2017biological}. Our novel approach used landscape sampling combined with model based differential abundance analysis, enabling us to test hypotheses about the ecology of biocrust organisms in drylands, particularly focusing on the implications of dispersal strategies for landscape stability and dust production. 

%\subsection{Co-occurrence of taxa in biocrusts and river channels}
% The higher proportion of DNA sequence reads shared between dune and river compared to river and pan, may be explained by the fact that flooding of the pan is a rare event (last occurring x years previously) but mobilisation of dust from sand dunes is occurring continuously. 

%The question of dispersal direction could be tackled in future work by monitoring microbial populations over time and relating this to weather conditions and occurrence of flood events. Whilst we are satisfied in the present work to uncover clues about microbial dispersal in dryland systems, the next challenge is explicitly linking this to function at landscape scale. 

%Due to the dune being at higher elevation than all other locations and above the flood line, it might be expected to harbour fewest OTUs of aquatic origin, however this does not seem to be the case (Figure \ref{fig:waterp}). In fact our data are not informative with respect to the direction of dispersal in this recycling system. 

%Dune and pan sites found throughout the world's drylands present similar harsh environments but different challenges for survival at the soil surface, especially in relation to disperal and colonisation. Claypans can be completely engulfed by flood events lasting ??\footnote{give range and citation if possible} and often leading to extensive replacement of the soil surface with new or overturned sediment, but in contrast dunes rarely if ever experience flooding. Due to the extensive vacant niche with abundant resources when floodwaters subside, organisms colonising the claypan soil will face extreme competition. Vegetative growth is therefore not expected to be an effective strategy for colonising the recently flooded claypan because it would be too slow, hence we expected that more generalised taxa already established in the rivers are most likely to be primary colonisers of the pan surface. We also expect due to the relatively short flooding interval at our field site that succession will not proceed far, therefore the claypan biocrusts are most likely to conatain organisms that are adapted for this lifecycle including both aquatic and terrestrial phases. Whilst it is possible that the pan biocrusts are mostly composed of ill-adapted aquatic species, we find this unlikely given the long geological timescales and widespread long-term global presence of similar systems having provided ample opportunity for organisms to efficiently exploit their periodicity.\footnote{need to say if our results support these ideas.}

\subsection{Microbial community of biocrusts in a dryland ephemeral lake bed and surrounding dunes}
Previous research on biocrust microbial communities in drylands is still quite rare in the literature \citep{ferrenberg2017biological} and has focused mainly on sandy locations, similar to the dunes here. We found that the dunes harboured typical biocrust communities with over 30 \% of sequences being of cyanobacterial origin, along with much of the remainder belonging to the phyla \textit{Actinobacteria}, \textit{Acidobacteria}, \textit{Chloroflexi}, and \textit{Proteobacteria} (Figure \ref{fig:waterp}). The same phyla in similar proportions have previously been found in biocrusts of the Kalahari in Botswana \citep{elliott2014niche}.

Dune and dune flank locations had very similar community structure, but these were clearly distinct from the pan and the nebkha locations (Figure \ref{fig:waterp}). It is normal to find different microbial communities in different situations so this result is as expected, however the question lies in the nature and reason for the differences observed. Edaphic factors like grain size and soil chemistry are commonly linked to such observations \footnote{perhaps I should do an analysis to check in this case}... This section should be expanded a bit...

Different communities in linked compartments of this recycling system indicate the outcome of different selective pressures leading to establishment of organisms with different adaptations. In the context of geomorphology adaptations relating to adhesion are of particular interest because they provide a potential mechanism for biotic control over a range of processes relating to the properties and movement of sediments. It is certain that such biotic control is occurring\citep[e.g.][]{bowker2007biological}\footnote{check that the ref is suitable. I think this is self-evident anyway.}, but details are scarce.

\subsection{Wind preferentially lifts certain taxa from biocrusts}
We have found strong evidence that microbes are differentially liberated from the dryland soil surface by moderate wind (e.g. Figure \ref{fig:waterp}; Tables \ref{tab:deseq1} and \ref{tab:deseq2}). These results show that the adhesive properties of biocrust organisms vary in a real-world situation, and importantly they identify taxa which may contribute to particular geomorphic processes as a result of their biological adaptations. It is not possible to say at this stage whether such adaptations were selected because of their geomorphic influence, or whether geomorphic influence is incidental to some other selective pressure. Both mechanisms of selection seem likely.

Our data show that sand dunes have the a high proportion of taxa that resist wind erosion (Figure \ref{fig:waterp}), thus sand dune surface communities are adapted in a way that promotes stability in the dune. This stabilising effect of the microbiota can be regarded as ecosystem engineering, as it modifies the environment to provide the stable soil surface required for long-term biocrust survival. Many OTUs were found to resist becoming airborne (Table S4), including most \textit{Cyanobacteria} which have long been recognised for their stabilising role in biocrusts \citep{belnap1998vulnerability}. This kind of adaptation may also inhibit dispersal by the wind, limiting the ability of biocrusts to colonise new areas or regenerate after elimination for example as a result of over-grazing, because the regenerating inoculum must come from a local source (e.g. vegetative growth or transport by an animal). Possible difficulties for biocrusts to regenerate after extreme sustained disturbance is something we suggested as a risk in \citet{elliott2014niche}, and is supported by the present results which confirm the resistance of many biocrust taxa to becoming airborne.

The nebkha sites within the claypan were the only place where a high proportion of the biota was identified as being easily mobilised by wind (xx \%; Figure \ref{fig:waterp}). This was not a direct measurement, but based upon taxa from the dune flank that were easily mobilised and measuring their abundance in the nebkha. We anticipate that a direct measurement on the nebkha sites would yield a higher proportion but this would be extremely difficult to do because the nebkha morphology is incompatible with wind tunnel apparatus. It should also be noted that our techniques do not establish direction of transport, therefore a proportion (possibly the majority) of the taxa mobilised from the dune flank by wind treatment are likely to have originated from the pan and nebkha sites. The reason for a high proportion of wind-mobile biota on the nebkha is unlikely to be a purely physical phenomenon, since the nebkha physico-chemical properties are similar\footnote{debatable though, await grain size results and any further info} to the dune and pan compartments (Figure \ref{fig:chem}). A biological explanation may be that nebkha are inhabited by biocrusts that rely principally on aerial disperal, because they are adapted as primary colonisers of newly exposed niches. Such organisms would need to have the capacity for adhesion to establish a colony in the new niche, but also an ability to disperse from the colony. This could be achieved by phenotypic switching which is a well known phenomenon in biofilms with a variety of mechanisms available (REF)\footnote{DE provide this}. Furthermore, biocrust organisms adapted for aerial dispersal would also need adaptations for survival in the air, such as small size and resistance to dessication and radiation. Alternatively, it is possible that the main niche of these microbes is in fact the atmosphere, and their presence in biocrusts is transitory, for instance to acquire nutrients before moving on.

\subsection{Microbial dispersal and landscape stability}

Biological adaptations for survival in fluvial-aeolian systems are likely to include mechanisms for modulating attachment to surfaces which may have profound geomorphic consequences. Attachment to the soil will result in binding of particles, reducing the potential for dust emission both by increasing particle sizes and by physical entrapment of particles. Whilst this is clearly advantageous to soil-surface dwelling biocrust communities, it is also a potential hazard to be physically constrained in this way, because soils may be buried by dust or submerged under flood waters for instance. Some organisms will surely overcome such hazards with adaptations for aquatic life or vegetative migration in the soil, and there is evidence for both in the biocrust literature (e.g. REFS). Another likely adaptation is to promote destabilisation of the biocrust to enable dispersal by the wind, which would lead to dust production and the transport of nutrients out of the soil. Once dispersed on the wind, microbes would have the opportunity to colonise new areas, thus ensuring survival even if the source area became inhospitable. A parallel situation is well documented in the biofilms literature which deals mainly with microbial cells living as attached communities in aqueous environments such as rivers and catheters. Even single-species biofilms exhibit both attachment and detachment phenotypes, triggered by genetic switches that respond to changing conditions (REF)\footnote{DE provide this}. Macroscopic organisms also exhibit this behaviour, for instance scallops are normally regarded as sesslie creatures but they will swim away in response to environmental triggers \citep{caddy1968underwater}. 

We propose that biocrust microbes in common with other organisms also have a variety of dispersal mechanisms, and that their dispersal adaptations have geomorphic consequences such as modulating particle size and dust availability. At least three distinct dispersal strategies are identified which may be employed by biocrust inhabitants, each strategy having particular phenotypic features and related geomorphic potential (Table \ref{tab:concept}). Whilst we propose a typical niche associated with each dispersal strategy, we do not expect this to be exclusive. Rather, we suggest that these strategies all co-exist in each biocrust community, and the relative abundance of organisms exhibiting each phenotype will vary depending on the life history of each particular biocrust.

% add in somewhere:
% Whilst the fluvial compartment of the Lake Eyre basin is largely confined by internal drainage, other systems have external drainage (e.g. xx). Aeolian transport of finer sediments (dust) provides a continuous opportunity for transport within the system, and importantly also with distant systems all over the world.

% Table generated by Excel2LaTeX from sheet 'Sheet1'
\begin{sidewaystable}
  \centering
    \caption{Summary of hypothesised dispersal strategies for biocrust microbes and their possible geomorphic effects. Example taxa for each mode of dispersal are given based on results from this study. Further candidate taxa can be identified from supplementary data Table S4}%
    \begin{tabular}{p{3.5cm}p{4cm}p{4cm}p{4.5cm}}
    \textbf{Dispersal strategy} & \textbf{Niche} & \textbf{Likely phenotype} & \textbf{Geomorphic effect} \\
    \hline
    Vegetative growth\newline{}e.g. \textit{Phormidium} and \textit{Microcoleus} spp (cyanobacteria) & Long-term stable soils\newline{}e.g. stabilised sand dune & oligotrophic\newline{}mutualistic\newline{}EPS production\newline{}Can recover from localised disturbance\newline{}Low water requirement & Strongly stabilise soil\newline{}Suppress dust production\newline{}Trapping dust\newline{}Enhance soil fertility \\
    \hline
    Aerial\newline{}e.g. \textit{Deinococcus} sp. (Thermi); \textit{Hymenobacter} sp. (Bacteroidetes) & Recently exposed soil and transient landscape features. Possibly active whilst airborne.\newline{}e.g. nebkha & copiotrophic\newline{}exploitative\newline{}Limited recovery from localised disturbance\newline{}Medium water requirement & Partially stabilise soil\newline{}Promote moderate dust production including removal of soil nutrients \\
    \hline
    Fluvial\newline{}e.g. \textit{Microcoleus} spp. & Water bodies, flood plains and pans\newline{}e.g. claypan & Copiotrophic\newline{}Excludes aerial invaders\newline{}Poor ability to recover from localised disturbance\newline{}High water requirement & Partially stabilise soil\newline{}Suppress dust production\newline{}Neutral soil fertility influence \\
    \end{tabular}%
    \label{tab:concept}%
\end{sidewaystable}%

\subsubsection{Fluvial dispersal strategy}
Periodic cycles of wetting and drying are a common feature of the dryland soil surface which all biocrust organisms must adapt to in order to exploit occasional rainfall and dew. Adaptation to flooding cycles requires in addition an ability to persist long-term in the aqueous environment, and the terrestrial environment. When floods subside in drylands the soil is rich in resources, and the aquatic organisms already present are likely to benefit from a founder effect that will prevent significant establishment of vegetatively or aerially dispersed microbes. This is likely to have happened in the claypan, where the aquatic organisms best placed to exploit the founder effect would be R strategists that can quickly consume the available resources before the soil dries out. In the dry times that follow these organisms are unlikely to be able to function effectively in the absence of abundant water and in the face of dwindling resources. If they are adapted for fluvial dispersal then their main task between floods is to survive and exclude competitors until further floodwaters arrive, therefore they are likely to lie dormant unless activated by prolonged exposure to water. In terms of landscape stability they will be an immediate benefit as floods subside due to the extensive production of biomass, however during dry times they may suffer erosion and are unlikely to have the capacity for repair. \footnote{what do our data say about this?}

\subsubsection{Vegetative dispersal strategy}
Unlike pan surfaces, stabilised sand dunes do not experience wholesale disruption via flooding, therefore succession can take place over long timescales and un-crusted sites exposed by localised disturbances can likely be re-colonised by vegetative growth from nearby areas. Large scale burial of sand dunes does occur as a consequence of aeolian deposition (REF), but these events do not disrupt the crust or bury it deeply, and it is known that biocrust biota can move vertically up to xx mm in a day (REF) so shallow burial is not expected to threaten biocrust survival. This occasional partial burial followed by upward migration of biocrust biota could contribute significantly to the stability of sand dunes. 

The vegetative dispersal strategy is implicitly assumed in the majority of biocrust literature which holds that biocrusts are slow to establish and they gradually succeed to stronger forms \citep[e.g][]{belnap1998vulnerability}. In the present work the sand dune biocrusts are a classic example of this standard view of biocrust development. Organisms relying on vegetative growth for dispersal are expected to be highly invested in maintaining a stable soil surface which facilitates the establishment of long-term extensive colonies that are highly resilient to harsh environmental conditions. This investment would likely include the production of EPS which stabilises the soil, and production of protective pigments in upper layers. Another expected feature is high resource efficiency and ability to maintain activity including primary production at very low water potential, thus providing the capacity for diverting resources to soil stabilisation. Our data suggested cyanobacteria belonging to the genera \textit{Phormidium} and \textit{Microcoleus} as likely candidates for vegetative dispersal, and indeed these are recognised as keystone taxa delivering stability in dryland biocrusts (REF)\footnote{need to check this}.

\subsubsection{Aeolian dispersal strategy}
Biocrusts may also be established from airborne inocula, and the nebkha biocrusts are most likely an example of this. Because strongly adhesive biocrust taxa are less likely to become airborne, biocrusts established from airborne sediments may lack the strength of vegetatively established biocrusts at least initially until more stabilising vegetative taxa move in (and possibly ultimately out-compete them). They must therefore rely on other stabilising landscape features such as stones and plants, enabling them to settle without investing strongly in promoting landscape stability although they probably do this incidentally. Taxa adapted for aerial dispersal must achieve high numbers in the atmosphere to be successful, thus they are likely to be R-strategists or copiotrophs that exploit islands of fertility to produce more airborne inocula. This means that although they have some capacity to stabilise soil, they may also contribute to dust production and export of nutrients from the soil. 

Our data suggest \textit{Erwinia}, \textit{Deinococcus}, and \textit{Hymenobacter} species may be adapted for aeolian dispersal as part of their life cycle (Table \ref{tab:deseq2}). \textit{Erwinia} species include plant pathogens which are known to disperse on the wind \citep{graham1975potential}, however they are not commonly reported or expected to reside in biocrust soils so we shall not consider them further here. \textit{Hymenobacter} species are recognised as extremophiles and the  \textit{H. arizonensis} was first isolated from a biocrust in Arizona \citep{reddy2013description}. Described properties align with our expectations for aerially dispersing biocrust microbes, including capability to use a wide range of carbon sources and posessing pigmentation possibly enabling resistance to radiation and dessication which is a known feature of the genus \citep{rainey2005extensive}. \textit{Deinococcus} species are also well known for their remarkable ability to withstand dessication and all kinds of radiation \citep{makarova2001genome}. Like \textit{Hymenobacter} this genus has been found in desert soils before \citep[e.g][]{rainey2005extensive} but according to the latest version of Bergey's manual \citep{whitman2015bergey} the natural habitat of \textit{Deinococcus} species remains unknown. We suggest then that the habitat of this ancient lineage is in fact the biocrust top surface, and it is adapted for wind dispersal. With such ecology \textit{Deinococcus} and other wind dispersed biocrust species may serve a protective function to the biocrust, and can also ensure that they quickly colonise any newly emerging biocrust which will immediately benefit from such protection early in their development. 

%\subsection{Geomorphic predictions from microbial ecology}
%...
%\begin{enumerate}
%\item Nitrogen deposition - Stable dune crusts with vegetative dispersal mechanism will experience increased erosion because the oligotrophic crust will face competition from copiotrophic aerial taxa which deplete soil resources
%\item More frequent flooding - Claypans will experience increased stability because the biocrust which is not well adapted for repair under dry conditions will be replenished more often
%\item More small rainfall events - Claypan biocrusts will be degraded due to activation of heterotrophs but insufficient water to initiate photosynthesis in R strategist phototrophs, resulting in increased dust production from claypans
%\end{enumerate}

\section{Conclusions}
\label{sec:conc}
In conclusion, we have shown that biocrust taxa vary in their capacity to be mobilised by the wind and we have suggested that this is inversely related to the taxon potential to promote landscape stability. We have also shown that biocrust microbes are easily found in watercourses, and proposed that some biocrust taxa of recycling fluvial-aeolian geomorphic systems are principally dispersed by fluvial deposition. These results provide new evidence supporting the microbial role in landscape stability and suppresion of dust production in drylands. By interpreting results from an ecological perspective, we provide a theoretical basis for predicting how microbes will influence landscape stability under different scenarios. Such predictions can be tested in future work, for instance by monitoring establishment and succession in biocrusts over time, in tandem with physical assessments. An ecological approach informed by microbiology can then be developed to help predict landscape responses to climate change and land use change, and also to plan mitigation strategies for maximiasing soil stability. 


%%% End of body of article

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Optional Appendix goes here



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%
%  ACKNOWLEDGMENTS



\bibliography{references}
%\bibliographystyle{plain}
%%  REFERENCE LIST AND TEXT CITATIONS
%
% Either type in your references using
%
% \begin{thebibliography}{}
% \bibitem{}
% Text
% \end{thebibliography}
%
% Or, to use BibTeX:
%
% Follow these steps
%
% 1. Type in \bibliography{<name of your .bib file>}
%    Run LaTeX on your LaTeX file.
%
% 2. Run BiBTeX on your LaTeX file.
%
% 3. Open the new .bbl file containing the reference list and
%   copy all the contents into your LaTeX file here.
%
% 4. Run LaTeX on your new file which will produce the citations.
%
% AGU does not want a .bib or a .bbl file. Please copy in the contents of your .bbl file here.


%\begin{thebibliography}{}

%\bibitem[{\textit{Bell and Munoz}}(2008)]{Boug10} Bell, A.~H., and
%Munoz, D.~P.  (2008). Activity in the superior colliculus reflects
%dynamic interactions between voluntary and involuntary influences on
%orienting behaviour. \textit{Eur. J. Neurosci.} 28, 1654--1660.

%\end{thebibliography}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Track Changes:
% To add words, \added{<word added>}
% To delete words, \deleted{<word deleted>}
% To replace words, \replace{<word to be replaced>}{<replacement word>}

% At the end of the document, use \listofchanges, which will list the
% changes and the page and line number where the change was made.

% When final version, \listofchanges will not produce anything,
% \added{} word will be printed, \deleted{} will take away the word,
% \replaced{}{} will print only the 2nd argument.

%%%
\listofchanges
%%%

\end{document}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Supporting Information
%% (Optional) See AGUSuppInfoSamp.tex/pdf for requirements
%% for Supporting Information.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
