---
title: "Diamantina biocrusts"
author: "David Elliott"
date: "`r format(Sys.time(), '%d %B, %Y')`" 
output:
  pdf_document:
    fig_caption: yes
    fig_crop: no
    fig_height: 8
    fig_width: 11
    keep_tex: yes
header-includes: \usepackage{multicol}\usepackage{graphicx}\graphicspath{img/}\usepackage{subfig}   
# \usepackage{subfig}
# classoption: landscape
---

```{r setup,echo=FALSE,message=FALSE,results='hide', warning=FALSE, cache=FALSE}
options(width = 120)
library(phyloseq)
library("ggplot2")
library(vegan)
library(grid)
library(reshape2)
library(scales)   # needed for formatting y-axis labels to non-scientific type
library(RColorBrewer)
library("DESeq2")
library(knitr)
set.seed(42)

theme_set(theme_bw())

# setup grid
g0 <- grid.layout(nrow=2, ncol=1)
vp.01 <- viewport(layout.pos.col=1, layout.pos.row=1) 
vp.02 <- viewport(layout.pos.col=1, layout.pos.row=2)

gl <- grid.layout(nrow=1, ncol=2)
vp.1 <- viewport(layout.pos.col=1, layout.pos.row=1) 
vp.2 <- viewport(layout.pos.col=2, layout.pos.row=1)
```

```{r load,echo=FALSE,message=FALSE,results='hide', warning=FALSE, cache=TRUE}
bac <- import_biom("data/otu_table_mc2_w_tax.biom", "data/rep_set.tre")

map <- import_qiime(mapfilename = "data/map.txt")
bac <- merge_phyloseq(bac,map)

  # Fix ranks. 
  colnames(tax_table(bac)) <- c("Kingdom", "Phylum", "Class", "Order", "Family", "Genus", "Species")
  # fix taxa names
  tax_table(bac)[,colnames(tax_table(bac))] <- gsub(tax_table(bac)[,colnames(tax_table(bac))],pattern="[a-z]__",replacement="")

# keep combined kingdom objects
expt.combined <- bac

# separate archaea and bacteria into different objects
arc <- subset_taxa(bac,Kingdom=="Archaea")
bac <- subset_taxa(bac,Kingdom=="Bacteria" )

# convert OTU observations into relative abundance per sample
arc.rel <- transform_sample_counts(arc, function(x) 100*x/(sum(x)))
bac.rel <- transform_sample_counts(bac, function(x) 100*x/(sum(x)))
# cyano.rel <- transform_sample_counts(cyano, function(x) 100*x/(sum(x)))
expt.combined.rel <- merge_phyloseq(otu_table(bac.rel),tax_table(bac.rel),otu_table(arc.rel),tax_table(arc.rel),sample_data(bac.rel))

# make cyano object
cyano.rel <- subset_taxa(bac.rel,Phylum=="Cyanobacteria")
#cyano.g <- tax_glom(cyano,"Genus")
cyano.rel.g <- tax_glom(cyano.rel,taxrank = "Genus")

# make combined kingdom phylum plots
expt.combined.rel.phylum <- tax_glom(expt.combined.rel,taxrank = "Phylum")
name <- "archaeal and bacterial phyla"

# Save a .csv of all the phylum abundances
# phylumdf.all = psmelt(tax_glom(expt.combined.rel,taxrank = "Phylum"))
phylumdf.all = psmelt(tax_glom(expt.combined.rel.phylum,taxrank = "Phylum"))
write.csv(phylumdf.all,"phylum_breakdown.csv")

# To simplify plots
# Remove taxa not seen at more than 1 % in at least 10% of the samples.
expt.combined.rel.phylum.rr = filter_taxa(expt.combined.rel.phylum, function(x) sum(x > 1) > (0.1*length(x)), TRUE)

cyano.rel.g.rr <- filter_taxa(cyano.rel.g, function(x) sum(x > 0.1) > (0.1*length(x)), TRUE)
# melt phyloseq object for plotting
phylumdf.rr = psmelt(expt.combined.rel.phylum.rr)
cyanodf = psmelt(cyano.rel.g.rr)

bac.earthair <- subset_samples(bac,element!="water")
arc.earthair <- subset_samples(arc,element!="water")
```

```{r analyse1,echo=FALSE,message=FALSE,results='hide', warning=FALSE, cache=TRUE}
# plot phyla (combined archea, bacteria, and fungi)
phylumdf.rr$Phylum <- paste(substr(phylumdf.rr$Kingdom,1,1),phylumdf.rr$Phylum,sep=";")

phyla.combined <- ggplot(phylumdf.rr, aes(x=zone_code, y=Abundance, color = element,shape = element)) + geom_boxplot() + facet_wrap(~Phylum, nrow=3) + scale_y_log10(name="relative abundance (%)",labels=comma) 
# plot cyanos

cyano.plot <- ggplot(cyanodf, aes(x=zone_code, y=Abundance, color = element, shape = element)) + geom_boxplot() + facet_wrap(~Genus, nrow=1) + scale_y_log10(name="relative abundance (%)",labels=comma)

name <- "bacteria"

rich.b <- plot_richness(bac,measures = "Chao1",color = "element", shape="element",x="zone") + geom_point(size=5,alpha=0.7) 
ord.b <- ordinate(bac.earthair, "NMDS", "bray")
NMDS.b <- plot_ordination(bac.earthair, ord.b, type = "sites", color = "zone", shape="element") + geom_point(size=5,alpha=0.7) + theme(aspect.ratio=1) + theme(legend.position="bottom")

name <- "archaea"

rich.a <- plot_richness(arc,measures = "Chao1",color = "element", shape="element",x="zone") + geom_point(size=5,alpha=0.7) 
ord.a <- ordinate(arc.earthair, "NMDS", "bray")
NMDS.a <- plot_ordination(arc.earthair, ord.a, type = "sites", color = "zone", shape="element") + geom_point(size=5,alpha=0.7) + theme(aspect.ratio=1) + theme(legend.position="bottom")

m_reads_total <- sum(get_variable(bac,'n_reads'))/1000000
m_reads_bac_qc <- sum(otu_table(bac))/1000000
m_reads_arc_qc <- sum(otu_table(arc))/1000000

```

```{r locations, echo=FALSE, eval=FALSE, cache=TRUE}
## DISABLED - see block below
library(ggmap)
library(sp)
library(rgdal) # needed by sp
library(RColorBrewer)
# Coordinates conversion to locations on the Earth in the Universal Transverse Mercator (UTM) coordinate system using the sp (spatial points) package. 

# Calculate spatial points based on the lat/lon information
# based on http://grokbase.com/t/r/r-sig-geo/136h9k8e9y/lat-long-utm-again

samples_latlon <- sample_data(bac)[,c("lat","lon")]
samples_latlon <- data.frame(samples_latlon)
#convert to sp object:
coordinates(samples_latlon) <- ~ lon + lat #longitude first

#adding a coordinate reference system:
proj4string(samples_latlon) <- CRS("+proj=longlat +ellps=WGS84 +datum=WGS84")

#project using sp transform:
UTM <- spTransform(samples_latlon, CRS=CRS("+proj=utm +zone=53 +ellps=WGS84 +datum=WGS84 +units=m +no_defs +towgs84=0,0,0"))

# Now get the spatial points back into the main dataframe:
UTM.df <- as.data.frame(UTM)
UTM.df$easting <- UTM.df$lon
UTM.df$northing <- UTM.df$lat
sample_data(bac)
sample_data(bac) <- cbind(sample_data(bac),UTM.df[,c("easting","northing")])
```

```{r map2,echo=FALSE, eval=FALSE, cache=TRUE}
# this block disabled because of error:
### > basemap <- ggmap(dMap)
### Error: GeomRasterAnn was built with an incompatible version of ggproto.
### Please reinstall the package that provides this extension.
# instead just including the graphic

#Create a custom color scale
myColors <- brewer.pal(6,"Set1")[c(6,1,2,5,4,3)]
names(myColors) <- levels(get_variable(bac,"zone"))
colScale <- scale_colour_manual(name = "zone",values = myColors)

# plot using ggmap. This just uses lat/lon, not the spatial points.

loc <- c(lon = mean(get_variable(bac,"lon")), lat = mean(get_variable(bac,"lat")))
locZoom <- c(lon = mean(get_variable(bac,"lon")[get_variable(bac,"zone")!="channel"]), lat = mean(get_variable(bac,"lat")[get_variable(bac,"zone")!="channel"]))

dMap <- get_map(location=loc, source="google", maptype = "satellite", zoom = 11)
dMapZoom <- get_map(location=locZoom, source="google", maptype = "satellite", zoom = 14)

basemap <- ggmap(dMap)
basemapZoom <- ggmap(dMapZoom)

basemap + geom_point(aes(x = lon, y = lat, colour=zone), data = dd.loc, size = 4) + geom_text(aes(label=index),size=2.5, data = dd.loc) + colScale

basemapZoom + geom_point(aes(x = lon, y = lat, colour=zone), data = subset(dd.loc,dd.loc$zone!="channel"), size = 4) + geom_text(aes(label=index),size=2.5, data = subset(dd.loc,dd.loc$zone!="channel")) + colScale

```

\section{Introduction}

Hypotheses:\label{hypotheses}
\begin{enumerate}
\item River flooding deposits microbes on the claypan, some of which persist as viable biocrusts
\item Dune erosion provides additional microbial inoculum to the claypan
\item Wind differentially mobilises certain microbes from the dunes
\begin{enumerate}
\item Easily mobilised organisms are more likely to become established in the claypan, especially on nebka
\item Easily mobilised organisms will be more abundant in river channels compared to less easily mobilised organisms 
\item Less easily mobilised organisms are better adapted to forming dune biocrusts (i.e. they will be rare elsewhere)
\end{enumerate}
\end{enumerate}

\section{methods}
\subsection{field site}
The dune runs approximately North-South and the claypan is on the East side (Figure \ref{fig:map}). Soil was sampled from five transects perpendicular to the dune, and water/sediment was sampled from water channels in the surrounding area.

\begin{figure}[p]%
\centering
\caption{Sampling locations}
\subfloat[General area of claypan including surrounding water channels.]{{ \includegraphics[width=15cm]{img/DNA1.png} }}%
\hspace{0.5cm}
\subfloat[Closer view of dune and pan area.]{{\includegraphics[width=15cm]{img/DNA2.png} }}%
\label{fig:map}%
\end{figure}

The following locations were targeted for soil sampling:

- Dune (D). Dune biological soil crust (BSC)
- Dune flank (DF). Dune flank BSC
- Nebka (N). BSC situated on nebka in the claypan to the East of the dune
- Pan (P). BSC platelets in vegetation interspaces further onto the claypan
- Channel (C). Sediment at the water's edge in water channels surrounding the dune/claypan complex

Transects were planned to be approximtely 500 m apart to achieve even spacing along the dune, however some zones to the South were quite heavily impacted by cattle and possibly rabbits. These impacted zones were excluded because the dune flanks did not have suitable areas to run the wind tunnel. The transects are numbered 1-5 starting at the South end. The different sampling areas are expected to have different grain size characteristics.

\subsection{Sampling}
Sampled areas were 10x10 cm and were selected to represent the diversity of biocrust in the area. In areas where differing crust types were noticeable the sampled areas were selected to have both types where applicable (e.g. biological/physical, light/dark). In badly disturbed areas remnants of pre-disturbance crust were selected (2x dune sites).

Water channels were sampled by dragging a 5 ml sterile bijoux along the bank where it meets the water. Approximately 2.5 ml bank sediment and 2.5 ml standing water were collected in each bottle from the same location. 

\subsection{wind tunnel}
In addition to the natural crust samples a wind tunnel was run on the dune flank zones within 3 m of where the natural sampling was done. The wind tunnel was run on the dune flanks because they are naturally eroded by saltation impact from the dune sand above when it is windy, therefore the potential as an inoculum source to surrounding areas is greater than the higher dune areas. A saltation source was not added to these experiments for the sake of simplicity, as there will be saltation occurring within the wind tunnel anyway. At each site the wind tunnel was run three times (on adjacent areas) for 5 minutes at 10 m/s for 5 minutes. This is a moderate treatment intended to be realistic and to collect only the easily extracted particles / organisms. 

(Helene collected data to determine the exact run conditions)

\section{results}
\subsection{Descriptive and exploratory analyses}
Blue-green platelet crusts were common but not the dominant surface cover on the claypan. They appear to have once been very widespread but have now been largely eroded away. Underneath them was a similar looking crust that seems to be physically formed, and is more brown. In many places there were nebka atop the blue-green platelet crusts and the platelets ran up the nebka slightly, however the crusts on nebka appeared distinct. Platelet crusts appeared to be more common near nebka. 

In summary, three distinct surface layers were observed in the claypan:
1. basal physical crust (not sampled)
2. platelet formation biological crust (sampled)
3. nebka (sampled)

```{r chem, echo=FALSE,fig.height=3,fig.cap='Soil properties\\label{fig:chem}', warning=FALSE, cache=TRUE}
# melt for plotting
chemistry.melt <- melt(sample_data(bac.earthair),measure.vars=c("N_pc","C_pc","pH"))

# make the plot 
print(ggplot(chemistry.melt, aes(x=zone, y=value, shape=zone, colour=zone)) + stat_summary(fun.data = "mean_se", size=1) + facet_wrap(~variable,scales = "free_y",nrow=1) + labs(x = "zone") + theme(aspect.ratio=1,legend.position="none") )

```

DNA was successfully extracted and sequenced from `r nsamples(bac)` samples, yielding `r round(m_reads_total,1)` million reads. After quality control and taxonomic assignment `r round(m_reads_bac_qc,1)` million bacterial reads and `r round(m_reads_arc_qc,1)` million archaeal reads remained. There were `r ntaxa(bac)` bacterial OTUs and `r ntaxa(arc)` archael OTUs detected (OTUs defined by 97 % similarity which is approximately species-level difference between OTUs). This analysis was planned to target bacteria, and as archaea make up only `r round(100*m_reads_arc_qc / m_reads_bac_qc,1)` \% of sequences, bacteria will be the focus of this analysis.

A total of `r length(unique(tax_table(bac)[,"Phylum"]))` bacterial phyla and `r length(unique(tax_table(arc)[,"Phylum"]))` archael phyla were identified. Frequencies of the most abundant phyla are shown in Figure \ref{fig:phyla}. It can be seen that water channel sampels are dissimilar to the soil-based samples at the phylum level although all abundant phyla are represented in both soil and water. 

```{r phyla, echo=FALSE, warning=FALSE, cache=TRUE,fig.cap='Relative abundance of bacterial and archael phyla in soil, water and airborne dust lifted by wind tunnel treatment. Bacterial and archael phyla abundances were calculated individually per kingdom.\\label{fig:phyla}'}
print(phyla.combined)
```

Cyanobacteria were abundant in all samples, and were more abundant in dune flank soil compared to airborne dust liberated by wind-tunnel treatment of the same soil. Relative abundance of most other phyla was not significantly different in soil and airborne dust (assumed: statistical tests not done yet). This result is explored further in Figure \ref{fig:cyano} which shows the frequency of the most abundant cyanobaterial genera in each sample. Of the three genera which were common in soil samples, Microcoleus and Phormidium species were more common in the soil compared to airborne dust, whereas Leptoglybna species were more common in the soil compared to dust (statistical tests not done yet). Phormidium was by far the most abundant cyanobacterial phylum, in common with findings from the Kalahari in southern Botswana.

```{r cyano, echo=FALSE, fig.height=4,warning=FALSE, cache=TRUE,fig.cap='Relative abundance of cyanobacterial genera in soil, water and airborne dust lifted by wind tunnel treatment. Abundances are relative wo the whole bacterial community.\\label{fig:cyano}'}
print(cyano.plot)
```

The Chao1 species richness estimation (Figure \ref{fig:rich}) indicated in excess of 10,000 species present in each sample, without striking differences with respect to sample type. These estimates may reduce after additional quality control measures are taken into account.

```{r rich, echo=FALSE,fig.width=4,fig.height=3,fig.cap='Chao1 richness estimate\\label{fig:rich}', cache=TRUE}
#fig.subcap=c('one plot', 'the other one'),
#grid.newpage()
#pushViewport(viewport(layout=gl))
#print(rich.a,vp=vp.1)
#print(rich.b,vp=vp.2)
print(rich.b)
```

Microbial community structure was compared with respect to sample source using nonmetric multidimensional scaling (NMDS). Initial results showed that the water samples differed extremely compared to all other samples, and this dominated all other inter-sample differences. The water samples were therefore removed and the community stucutre relationships between the soil and airborne dust samples was revealed as shown in Figure \ref{fig:nmds}. This plot shows differences in community structure between all sample sites. The dune community differs from the dune flank community, and the airborne dust community derived from the dune flank differs from the dune flank community. Notably the dust community has a community structure intermediate between the dune / dune flank and nebka communities.

```{r nmds, echo=FALSE,fig.height=6,fig.cap='NMDS visualisation of bacterial community structure\\label{fig:nmds}', cache=TRUE}

#grid.newpage()
#pushViewport(viewport(layout=gl))
#print(NMDS.a,vp=vp.1)
#print(NMDS.b,vp=vp.2)
print(NMDS.b)

```

\subsection{Addressing the hypotheses}

```{r hyp1, echo=FALSE,fig.height=6,fig.cap='Shared taxa between channel and pan (left), and between channel and dune (right). \\label{fig:shared}', cache=TRUE}
# no output from this block but kept for now as it calculates some useful results/objects.
# Might be suitable for output as small table or in text mention.

channel <- subset_samples(bac.rel,zone=="channel")
dune <- subset_samples(bac.rel,element=="air")
pan <- subset_samples(bac.rel,zone=="pan")

channel <- prune_taxa(taxa_sums(channel)>0,channel)
dune <- prune_taxa(taxa_sums(dune)>0,dune)
pan <- prune_taxa(taxa_sums(pan)>0,pan)

channel.ntaxa <- ntaxa(channel)
dune.ntaxa <- ntaxa(dune)
pan.ntaxa <- ntaxa(pan)

channel_dune.intersection <- intersect(taxa_names(channel),taxa_names(dune))
channel_dune.intersection.n <- length(channel_dune.intersection)  

channel_pan.intersection <- intersect(taxa_names(channel),taxa_names(pan))
channel_pan.intersection.n <- length(channel_pan.intersection)  

dune_pan.intersection <- intersect(taxa_names(dune),taxa_names(pan))
dune_pan.intersection.n <- length(dune_pan.intersection)  

all.intersection <- intersect(channel_dune.intersection, channel_pan.intersection)
all.intersection <- length(all.intersection)

# Now we have the intersections
# make objects for the pairs and subset to just the intersecting taxa
channel_pan <- subset_samples(bac.rel,zone%in%c("channel","pan"))
channel_pan <- prune_taxa(channel_pan.intersection, channel_pan)
otu_table(channel_pan) <- otu_table(channel_pan)/5

channel_dune <- subset_samples(bac.rel,zone%in%c("channel","dune"))
channel_dune <- prune_taxa(channel_dune.intersection, channel_dune)
otu_table(channel_dune) <- otu_table(channel_dune)/5

# create a common colour palette for plotting
# not used now but might be useful later
#allphyla <- unique(c(tax_table(channel_dune)[,"Phylum"],tax_table(channel_pan)[,"Phylum"]))
#allphyla.col <- as.vector(colorRampPalette(brewer.pal(12,"Set3"))(length(allphyla)))
#names(allphyla.col)  <- allphyla

# plot
#channel_pan.shared <- plot_bar(channel_pan,x = "zone",fill = "Phylum") + geom_bar(aes(color=Phylum, fill=Phylum), stat="identity", position="stack") + scale_y_continuous(name="relative abundance (%)") + scale_fill_manual("Phylum", values = allphyla.col) + scale_colour_manual("Phylum", values = allphyla.col)

# work out OTU abundance for those only in soil
bac.rel.earth <- subset_samples(bac.rel,element=="earth")
bac.rel.earth.merged <- merge_samples(bac.rel.earth,rep(T,nsamples(bac.rel.earth)))
otu_table(bac.rel.earth.merged) <- otu_table(bac.rel.earth.merged)/nsamples(bac.rel.earth)
inoc_source_otus <- unique(c(taxa_names(channel),taxa_names(dune)))

soil_only_otus.sum <- round(sum(otu_table(bac.rel.earth.merged)) -  sum(otu_table(bac.rel.earth.merged)[,inoc_source_otus]),1)

```

```{r hyp2v2, echo=FALSE,fig.height=6,fig.cap='Phylum abundance and OTU source in soil samples and water samples. Source indicates whether individual OTUs (97 % similarity) contributing to the phylum abundance were found in airborne dust from the dune flank, or in water and sediment in water channels. Only phyla present at > 1% of all reads are shown for clarity. Source designation is not indicative of the ultimate origin of OTUs. \\label{fig:source}', cache=TRUE}
# Add to tax table info on where OTU if found
# working with merged objects to reduce processing time
bac.rel.merged <- merge_samples(subset_samples(bac.rel,element!="air"),"zone")
otu_table(bac.rel.merged) <- otu_table(bac.rel.merged)/5

# Keep only phyla that are > 1% overall:
# to simplify plot and protect against unequal sampling bias
bac.rel.merged.phylum <- tax_glom(bac.rel.merged,"Phylum")
keep_phyla <- (taxa_sums(bac.rel.merged.phylum)/5)>1
keep_phyla <- as.character(tax_table(bac.rel.merged.phylum)[keep_phyla,"Phylum"])
keep_phyla_index <- tax_table(bac.rel.merged)[,"Phylum"]%in%keep_phyla

bac.rel.merged <- subset_taxa(bac.rel.merged, keep_phyla_index)
sample_data(bac.rel.merged)$zone <- as.factor(sample_names(bac.rel.merged))

airborne <- subset_samples(bac.rel,element=="air")
airborne <- prune_taxa(taxa_sums(airborne)>0,airborne)
waterborne <- subset_samples(bac.rel.merged,zone=="channel")
waterborne <- prune_taxa(taxa_sums(waterborne)>0,waterborne)

airborne_otus <- taxa_names(airborne)[taxa_sums(airborne)>0]
waterborne_otus <- taxa_names(waterborne)[taxa_sums(waterborne)>0]
both_otus <- intersect(airborne_otus,waterborne_otus)

# simplify data (for testing only)
bac.rel2 <- prune_taxa(taxa_sums(bac.rel.merged) > 0.1, bac.rel.merged) 
# bac.rel2 <- bac.rel.merged
airborne_otus <- airborne_otus[airborne_otus%in%taxa_names(bac.rel2)]
waterborne_otus <- waterborne_otus[waterborne_otus%in%taxa_names(bac.rel2)]
both_otus <- both_otus[both_otus%in%taxa_names(bac.rel2)]

colnames(tax_table(bac.rel2))[1] <- "source"

tax_table(bac.rel2)[,"source"] <- "neither"
tax_table(bac.rel2)[airborne_otus,"source"] <- "airborne"
tax_table(bac.rel2)[waterborne_otus,"source"] <- "waterborne"
tax_table(bac.rel2)[both_otus,"source"] <- "both"

otu_source <- plot_bar(bac.rel2,x = "zone",fill = "source") + geom_bar(aes(color=source, fill=source), stat="identity", position="stack") + scale_y_continuous(name="relative abundance (%)") 
otu_source + facet_wrap(~Phylum, nrow=2)

```

Of the `r pan.ntaxa` OTUs found in the claypan, `r channel_pan.intersection.n` were also found in water channels, `r dune_pan.intersection.n` were found in airborne dust from the dune flank, and `r all.intersection` were found in both of these hypothesised inoculum sources. The dune and channel samples had `r channel_dune.intersection.n` OTUs in common. 

Whilst many soil OTUs were not detected in the air or water samples, most of the OTUs found exclusively in soil were rare, collectively accounting for only `r soil_only_otus.sum ` % of reads in the soil samples. A breakdown of OTU co-occurrence in all samples summarised by phylum read frequency is shown in Figure \ref{fig:source}. It can be seen that co-occurrence of bacteria between soils and hypothesised inoculum sources is very common (assessed by number of reads not number of OTUs), and the relative number of organisms from different inoculum sources varies with respect to soil type and phylum. 

```{r deseq, echo=FALSE, message=FALSE, cache=TRUE}

expt.deseq <- subset_samples(bac,zone=="dune flank")
  expt.deseq = prune_taxa(taxa_sums(expt.deseq) > 0, expt.deseq)
  
  # add 1 to the whole OTU table, to avoid infinity problems
  # this will reduce the fold changes
  otu_table(expt.deseq) <- otu_table(expt.deseq)+1
  
  typedds = phyloseq_to_deseq2(expt.deseq, ~ element)
  typedds = DESeq(typedds, test="Wald",fitType="parametric")
  # DESeq docs:
  #On p-values:
  # By default, independent filtering is performed to select a set of genes for multiple test correction which will optimize the number of adjusted p-values less than a given critical value alpha (by default 0.1). The adjusted p-values for the genes which do not pass the filter threshold are set to NA. 
  
# When using local regression fit, the user should examine plotDispEsts(dds)
# to make sure the fitted line is not sharply curving up or down based on
# the position of individual points.

  #  plotDispEsts(typedds,main="model check")

  res = results(typedds, cooksCutoff = FALSE, alpha = 0.05)
  alpha = 0.05
  sigtab_type = res

  rel.abundance <- apply(otu_table(bac)[rownames(sigtab_type), ],FUN = max,MARGIN = 1)
  sigtab_type = cbind(as(sigtab_type, "data.frame"), as(tax_table(expt.deseq)[rownames(sigtab_type), ], "matrix"))
  sigtab_type$max <- rel.abundance
  
  # record the OTU name
  sigtab_type$otu <- row.names(sigtab_type)
  
  ind <- sigtab_type$padj<0.05&!is.na(sigtab_type$padj)
  sigtab_type$otu.orig <- sigtab_type$otu
  sigtab_type[ind,"otu2"] <- gsub(pattern = "OTU_",replacement = "",x = sigtab_type[ind,"otu"])
  sigtab_type[ind,"otu"] <- paste("*",sigtab_type[ind,"otu"],"*",sep="")

  sigtab_type$sig <- ""
  sigtab_type[ind,"sig"] <- "*"


  # Reverse the sign of the fold change so that it represents crust:subsoil not subsoil:crust
  # sigtab_type$log2FoldChange <- sigtab_type$log2FoldChange * -1
  # sigtab_type$fold <- sign(sigtab_type$log2FoldChange) * (2^abs(sigtab_type$log2FoldChange))
  
  scale_fill_discrete <- function(palname = "Set1", ...) {
    scale_fill_brewer(palette = palname, ...)
  }
  # Phylum order
  x = tapply(sigtab_type$log2FoldChange, sigtab_type$Phylum, function(x) max(x))
  x = sort(x, TRUE)
  sigtab_type$Phylum = factor(as.character(sigtab_type$Phylum), levels=names(x))
  # Class order
  x = tapply(sigtab_type$log2FoldChange, sigtab_type$Class, function(x) max(x))
  x = sort(x, TRUE)
  sigtab_type$Class = factor(as.character(sigtab_type$Class), levels=names(x))
  # Genus order
  x = tapply(sigtab_type$log2FoldChange, sigtab_type$Genus, function(x) max(x))
  x = sort(x, TRUE)
  sigtab_type$Genus = factor(as.character(sigtab_type$Genus), levels=names(x))
  # Family order
  x = tapply(sigtab_type$log2FoldChange, sigtab_type$Family, function(x) max(x))
  x = sort(x, TRUE)
  sigtab_type$Family = factor(as.character(sigtab_type$Family), levels=names(x))
  # Order order
  x = tapply(sigtab_type$log2FoldChange, sigtab_type$Order, function(x) max(x))
  x = sort(x, TRUE)
  sigtab_type$Order = factor(as.character(sigtab_type$Order), levels=names(x))
  
  sd <- as.data.frame(sample_data(expt.deseq))

  a <- levels(sample_data(expt.deseq)$type)[2]
  a.desc <- sd[sd$element=="air","element"][1]
  a.desc <- as.character(unclass(a.desc)$element)

  b <- levels(sample_data(expt.deseq)$element)[1]
  b.desc <- sd[sd$element=="earth","element"][1]
  b.desc <- as.character(unclass(b.desc)$element)

  tbase <- paste(c(a.desc,b.desc),collapse=" compared to ")
  
  inclab <- paste("increased in ",a.desc,sep="")
  declab <- paste("decreased in ",a.desc,sep="")
  t <- tbase

deseq_plot1 <-     ggplot(sigtab_type) + ggtitle(t) +
    geom_point(aes(x=Class, y=log2FoldChange, group=otu,color=Phylum, label=otu),alpha=0.9) + 
    theme(axis.text.x = element_text(angle = -40, hjust = 0, vjust=1,size=10)) +
    geom_text(aes(x=Class, y=log2FoldChange, group=otu,label=sig),fontface="bold",size=12) +
    geom_text(aes(x=nlevels(Class)/3,y=max(log2FoldChange)),hjust=1) +
    geom_text(aes(x=nlevels(Class)/3,y=min(log2FoldChange)),hjust=1)   


  t <- paste(tbase," p <",alpha)

deseq_plot2 <-   ggplot(subset(sigtab_type,sigtab_type$padj<alpha)) + ggtitle(t) +
    geom_point(aes(x=Class, y=log2FoldChange, group=otu,color=Phylum, label=otu2),alpha=0.9) + 
    geom_text(aes(x=Class, y=log2FoldChange, group=otu2,label=otu2),fontface="bold",angle=45 ,hjust = 0, vjust=0.5, size=3) + 
    theme(axis.text.x = element_text(angle = -40, hjust = 0, vjust=1,size=10))+ facet_wrap(~Kingdom,scales = "free_x") +
  geom_text(aes(x=3,y=max(log2FoldChange)),label=inclab,hjust=0) +
    geom_text(aes(x=3,y=min(log2FoldChange)),label=declab,hjust=0)   

# manipulations to the sigtab_type table - for later output as graphics and tables

wanted_cols <- c("Phylum",  "Class",  "Order",	"Family",	"Genus",	"Species","padj","log2FoldChange")
wanted_rows <- (sigtab_type$padj<0.05) & (!is.na(sigtab_type$padj))
wanted_otus <- sigtab_type[wanted_rows,"otu.orig"]

expt.rel.df <- subset_samples(bac.rel, zone=="dune flank")

otus.rel.df.by.type <- otu_table(merge_samples(expt.rel.df,group = "element"))/5
orbt <- as.data.frame(t(otus.rel.df.by.type))
sigtab_type.subset <- sigtab_type[wanted_rows,wanted_cols]
sigtab_type.subset <- cbind(sigtab_type.subset[wanted_otus,],orbt[wanted_otus,])
# add a difference column
sigtab_type.subset$diff <- sigtab_type.subset$earth - sigtab_type.subset$air

# Sanity check on the significant OTUs
sane <- sign(sigtab_type.subset$log2FoldChange)==(sign(sigtab_type.subset$diff))

# identify the removed OTUs (none)
# sigtab_type.subset.removed <- sigtab_type.subset[!sane,]

# remove the OTUs not passing sanity check (none)
# sigtab_type.subset <- sigtab_type.subset[sane,]

# order by diff
sigtab_type.subset <- sigtab_type.subset[order(sigtab_type.subset$diff,decreasing = T),] 
# save to file
write.csv(sigtab_type.subset,"windtunnel_otus.csv")

outcols <- c("Phylum",		"Family",	"Genus","padj","air",	"earth") #"Class",	"Order",
outtab <- sigtab_type.subset[1:10,outcols]

outtab2 <- sigtab_type.subset[nrow(sigtab_type.subset):(nrow(sigtab_type.subset)-10),outcols]

kable(outtab, digits=2,caption = "OTUs which are significantly more abundant in soil compared to dust blown from the soil\\label{tab:deseq1}")
kable(outtab2, digits=2,caption = "OTUs which are significantly more abundant in dust blown from the soil compared to the whole soil.\\label{tab:deseq2}")
#outtab

otus_earth <- row.names(sigtab_type.subset[sigtab_type.subset$diff>0,])
otus_air <- row.names(sigtab_type.subset[sigtab_type.subset$diff<0,])

n_otu_earth <- length(otus_earth)
n_otu_air <- length(otus_air)


```

```{r airborne_dest, echo=FALSE,fig.height=6,fig.cap='Representation of taxa easily mobilised from the dune flank by wind in all sample types. \\label{fig:airborne}', cache=TRUE}
# TODO
# make a plot showing where otus_earth and otus_air show up


#bac.rel.ea <- subset_taxa(bac.rel,taxa_names(bac.rel)%in%row.names(sigtab_type.subset))
bac.rel.ea <- bac.rel
bac.rel.ea <- subset_samples(bac.rel.ea,element!="air")
#bac.rel.air <- subset_taxa(bac.rel,taxa_names(bac.rel)%in%otus_air)
#bac.rel.air <- subset_samples(bac.rel.air,element!="air")

# bac.rel.ea <- merge_samples(bac.rel.ea,"zone")
otu_table(bac.rel.ea) <- otu_table(bac.rel.ea)/5


colnames(tax_table(bac.rel.ea))[1] <- "source"
tax_table(bac.rel.ea)[,"source"] <- "intermediate"
tax_table(bac.rel.ea)[row.names(tax_table(bac.rel.ea))%in%otus_air,"source"] <- "easily becomes airborne"
tax_table(bac.rel.ea)[row.names(tax_table(bac.rel.ea))%in%otus_earth,"source"] <- "resists becoming airborne"

# keep only abundant phyla to simplify plot
keep_phyla_index <- tax_table(bac.rel.ea)[,"Phylum"]%in%keep_phyla
bac.rel.ea <- subset_taxa(bac.rel.ea, keep_phyla_index)
# simplify data (for testing only)
bac.rel.ea <- prune_taxa(taxa_sums(bac.rel.ea) > 0.01, bac.rel.ea) 

wind_otus_plot <- plot_bar(bac.rel.ea,x = "zone",fill = "source")
wind_otus_plot <- wind_otus_plot + geom_bar(aes(color=source, fill=source), stat="identity", position="stack") 
wind_otus_plot <- wind_otus_plot + scale_y_continuous(name="relative abundance (%)") + scale_fill_brewer() + scale_color_brewer()

wind_otus_plot + facet_wrap(~Phylum, nrow=2)

```


\section{discussion}
... see Table \ref{tab:deseq1} and Table \ref{tab:deseq2} . Due to the dune being at higher elevation than all other locations, it might be expected to harbour fewer OTUs of aquatic origin, however this does not seem to be the case. In fact our data are not informative with respect to the direction of dispersal. This question could be tackled in future work monitoring microbial populations over time and relating this to weather conditions and occurrence of flood events. We expect that co-occurrence between dune and river channel may be driven principally by wind dispersal (direction dune>river), whereas co-occurrence between river channel and claypan may be driven principally by flood events (direction river>claypan). The higher proportion of DNA sequence reads shared between dune and river compared to river and pan, may be explained by the fact that flooding of the pan is a rare event (last occurring x years previously) but mobilisation of dust from sand dunes is occurring continuously.
