---
title: "Diamantina biocrusts"
author: "David Elliott"
date: "`r format(Sys.time(), '%d %B, %Y')`" 
bibliography: references.bib
output:
  pdf_document:
    fig_caption: yes
    fig_crop: no
    fig_height: 8
    fig_width: 11
    keep_tex: yes

header-includes: \usepackage{multicol}\usepackage{graphicx}\graphicspath{img/}\usepackage{subfig}\usepackage{pdflscape}\usepackage{natbib}\usepackage{attachfile}
---

```{r setup,echo=FALSE,message=FALSE,results='hide', warning=FALSE, cache=FALSE}
options(width = 120)
library(phyloseq)
library("ggplot2")
library(vegan)
library(grid)
library(reshape2)
library(scales)   # needed for formatting y-axis labels to non-scientific type
library(RColorBrewer)
library("DESeq2")
library(knitr)
set.seed(42)

theme_set(theme_bw())

# setup grid
g0 <- grid.layout(nrow=2, ncol=1)
vp.01 <- viewport(layout.pos.col=1, layout.pos.row=1) 
vp.02 <- viewport(layout.pos.col=1, layout.pos.row=2)

gl <- grid.layout(nrow=1, ncol=2)
vp.1 <- viewport(layout.pos.col=1, layout.pos.row=1) 
vp.2 <- viewport(layout.pos.col=2, layout.pos.row=1)
```

```{r load,echo=FALSE,message=FALSE,results='hide', warning=FALSE, cache=TRUE}
bac <- import_biom("data/otu_table_mc2_w_tax.biom", "data/rep_set.tre",parseFunction = parse_taxonomy_greengenes)

map <- import_qiime(mapfilename = "data/map.txt")
bac <- merge_phyloseq(bac,map)

  # remove chloroplast
  bac <- subset_taxa(bac,Class!="Chloroplast")

  # standardise unknown taxon entries
tax_table(bac)[is.na(tax_table(bac))] <- "Unknown"
tax_table(bac)[tax_table(bac) %in% c("", "unclassified", "Unclassified")] <- "Unknown"

# keep combined kingdom objects
expt.combined <- bac

# separate archaea and bacteria into different objects
arc <- subset_taxa(bac,Kingdom=="Archaea")
bac <- subset_taxa(bac,Kingdom=="Bacteria" )

expt.deseq <- subset_samples(bac,zone=="dune flank")
expt.deseq = prune_taxa(taxa_sums(expt.deseq) > 0, expt.deseq)
# add 1 to the whole OTU table, to avoid infinity problems in deseq analysis
# this will reduce the fold changes
otu_table(expt.deseq) <- otu_table(expt.deseq)+1
  
# convert OTU observations into relative abundance per sample
arc.rel <- transform_sample_counts(arc, function(x) 100*x/(sum(x)))
bac.rel <- transform_sample_counts(bac, function(x) 100*x/(sum(x)))
expt.combined.rel <- merge_phyloseq(otu_table(bac.rel),tax_table(bac.rel),otu_table(arc.rel),tax_table(arc.rel),sample_data(bac.rel))

expt.combined.rel.phylum <- tax_glom(expt.combined.rel,taxrank = "Phylum")
cyano.rel <- subset_taxa(bac.rel,Phylum=="Cyanobacteria")

# Identify top phyla and cyano genera - so they can be selectively plotted/analysed
top.phyla = names(sort(tapply(taxa_sums(bac.rel), tax_table(bac.rel)[, "Phylum"], sum), TRUE)[1:10])
top.cyano.genus = names(sort(tapply(taxa_sums(cyano.rel), tax_table(cyano.rel)[, "Genus"], sum), TRUE)[1:6])
top.cyano.genus <- top.cyano.genus[top.cyano.genus!="Unknown"]

# keep only the top phyla, but retain all in .all object for when needed
bac.rel.all <- bac.rel
bac.rel <- subset_taxa(bac.rel, Phylum%in%top.phyla)

# keep only the more abundant cyanobacterial genera
cyano.rel <- subset_taxa(cyano.rel, Genus%in%top.cyano.genus)
cyano.rel.g <- tax_glom(cyano.rel,taxrank = "Genus")

# simplify data (for testing only)
bac.rel <- prune_taxa(taxa_sums(bac.rel) > 0.2, bac.rel) 

# To simplify plots
# Remove phyla not seen at more than 1 % in at least 10% of the samples.
expt.combined.rel.phylum.rr = filter_taxa(expt.combined.rel.phylum, function(x) sum(x > 1) > (0.1*length(x)), TRUE)

# Save a .csv of all the phylum abundances
#phylumdf.all = psmelt(tax_glom(expt.combined.rel.phylum,taxrank = "Phylum"))
#write.csv(phylumdf.all,"phylum_breakdown.csv")

expt.combined.rel.phylum.merged <- merge_samples(subset_samples(expt.combined.rel.phylum,element!="air"),"zone")
otu_table(expt.combined.rel.phylum.merged) <- otu_table(expt.combined.rel.phylum.merged)/5

tp <- t(otu_table(expt.combined.rel.phylum.merged))
tp <- as.data.frame(tp)
phyla.df <- as.data.frame(cbind(tax_table(expt.combined.rel.phylum.merged),tp))
phyla.order <- taxa_names(expt.combined.rel.phylum.merged)[order(taxa_sums(expt.combined.rel.phylum.merged),decreasing = T)]
phyla.df <- phyla.df[phyla.order,]
write.csv(phyla.df[ ,c(rank_names(bac)[1:2],sample_names(expt.combined.rel.phylum.merged))],"phyla.csv")

cyano.rel.g.rr <- subset_taxa(cyano.rel.g, Genus%in%top.cyano.genus)
# melt phyloseq object for plotting
phylumdf.rr = psmelt(expt.combined.rel.phylum.rr)
cyanodf = psmelt(cyano.rel.g.rr)

bac.earthair <- subset_samples(bac,element!="water")
arc.earthair <- subset_samples(arc,element!="water")
ab.earthair <- subset_samples(expt.combined,element!="water")
```

```{r analyse1,echo=FALSE,message=FALSE,results='hide', warning=FALSE, cache=TRUE}
# plot phyla (combined archea, bacteria, and fungi)
phylumdf.rr$Phylum <- paste(substr(phylumdf.rr$Kingdom,1,1),phylumdf.rr$Phylum,sep=";")

phyla.combined <- ggplot(phylumdf.rr, aes(x=zone_code, y=Abundance, color = element,shape = element)) + geom_boxplot() + facet_wrap(~Phylum, nrow=3) + scale_y_log10(name="relative abundance (%)",labels=comma) 
# plot cyanos

cyano.plot <- ggplot(cyanodf, aes(x=zone_code, y=Abundance, color = element, shape = element)) + geom_boxplot() + facet_wrap(~Genus, nrow=1) + scale_y_log10(name="relative abundance (%)",labels=comma)

rich.ab <- plot_richness(expt.combined,measures = "Chao1",color = "element", shape="element",x="zone") + geom_point(size=5,alpha=0.7) 
ord.ab <- ordinate(ab.earthair, "NMDS", "bray")
NMDS.ab <- plot_ordination(ab.earthair, ord.ab, type = "sites", color = "zone", shape="element") + geom_point(size=5,alpha=0.7) + theme(aspect.ratio=1) + theme(legend.position="bottom")

#rich.b <- plot_richness(bac,measures = "Chao1",color = "element", shape="element",x="zone") + geom_point(size=5,alpha=0.7) 
#ord.b <- ordinate(bac.earthair, "NMDS", "bray")
#NMDS.b <- plot_ordination(bac.earthair, ord.b, type = "sites", color = "zone", shape="element") + geom_point(size=5,alpha=0.7) + theme(aspect.ratio=1) + theme(legend.position="bottom")

#rich.a <- plot_richness(arc,measures = "Chao1",color = "element", shape="element",x="zone") + geom_point(size=5,alpha=0.7) 
#ord.a <- ordinate(arc.earthair, "NMDS", "bray")
#NMDS.a <- plot_ordination(arc.earthair, ord.a, type = "sites", color = "zone", shape="element") + geom_point(size=5,alpha=0.7) + theme(aspect.ratio=1) + theme(legend.position="bottom")

m_reads_total <- sum(get_variable(bac,'n_reads'))/1000000
m_reads_bac_qc <- sum(otu_table(bac))/1000000
m_reads_arc_qc <- sum(otu_table(arc))/1000000

```

```{r locations, echo=FALSE, eval=FALSE, cache=TRUE}
## DISABLED - see block below
library(ggmap)
library(sp)
library(rgdal) # needed by sp
library(RColorBrewer)
# Coordinates conversion to locations on the Earth in the Universal Transverse Mercator (UTM) coordinate system using the sp (spatial points) package. 

# Calculate spatial points based on the lat/lon information
# based on http://grokbase.com/t/r/r-sig-geo/136h9k8e9y/lat-long-utm-again

samples_latlon <- sample_data(bac)[,c("lat","lon")]
samples_latlon <- data.frame(samples_latlon)
#convert to sp object:
coordinates(samples_latlon) <- ~ lon + lat #longitude first

#adding a coordinate reference system:
proj4string(samples_latlon) <- CRS("+proj=longlat +ellps=WGS84 +datum=WGS84")

#project using sp transform:
UTM <- spTransform(samples_latlon, CRS=CRS("+proj=utm +zone=53 +ellps=WGS84 +datum=WGS84 +units=m +no_defs +towgs84=0,0,0"))

# Now get the spatial points back into the main dataframe:
UTM.df <- as.data.frame(UTM)
UTM.df$easting <- UTM.df$lon
UTM.df$northing <- UTM.df$lat
sample_data(bac)
sample_data(bac) <- cbind(sample_data(bac),UTM.df[,c("easting","northing")])
```

```{r map2,echo=FALSE, eval=FALSE, cache=TRUE}
# this block disabled because of error:
### > basemap <- ggmap(dMap)
### Error: GeomRasterAnn was built with an incompatible version of ggproto.
### Please reinstall the package that provides this extension.
# instead just including the graphic

#Create a custom color scale
myColors <- brewer.pal(6,"Set1")[c(6,1,2,5,4,3)]
names(myColors) <- levels(get_variable(bac,"zone"))
colScale <- scale_colour_manual(name = "zone",values = myColors)

# plot using ggmap. This just uses lat/lon, not the spatial points.

loc <- c(lon = mean(get_variable(bac,"lon")), lat = mean(get_variable(bac,"lat")))
locZoom <- c(lon = mean(get_variable(bac,"lon")[get_variable(bac,"zone")!="channel"]), lat = mean(get_variable(bac,"lat")[get_variable(bac,"zone")!="channel"]))

dMap <- get_map(location=loc, source="google", maptype = "satellite", zoom = 11)
dMapZoom <- get_map(location=locZoom, source="google", maptype = "satellite", zoom = 14)

basemap <- ggmap(dMap)
basemapZoom <- ggmap(dMapZoom)

basemap + geom_point(aes(x = lon, y = lat, colour=zone), data = dd.loc, size = 4) + geom_text(aes(label=index),size=2.5, data = dd.loc) + colScale

basemapZoom + geom_point(aes(x = lon, y = lat, colour=zone), data = subset(dd.loc,dd.loc$zone!="channel"), size = 4) + geom_text(aes(label=index),size=2.5, data = subset(dd.loc,dd.loc$zone!="channel")) + colScale

```

```{r hyp_analysis,echo=FALSE, cache=TRUE}
# no output from this block but kept for now as it calculates some useful results/objects.
# Might be suitable for output as small table or in text mention.

channel <- subset_samples(bac.rel,zone=="channel")
dune <- subset_samples(bac.rel,element=="air")
pan <- subset_samples(bac.rel,zone=="pan")

channel <- prune_taxa(taxa_sums(channel)>0,channel)
dune <- prune_taxa(taxa_sums(dune)>0,dune)
pan <- prune_taxa(taxa_sums(pan)>0,pan)

channel.ntaxa <- ntaxa(channel)
dune.ntaxa <- ntaxa(dune)
pan.ntaxa <- ntaxa(pan)

channel_dune.intersection <- intersect(taxa_names(channel),taxa_names(dune))
channel_dune.intersection.n <- length(channel_dune.intersection)  

channel_pan.intersection <- intersect(taxa_names(channel),taxa_names(pan))
channel_pan.intersection.n <- length(channel_pan.intersection)  

dune_pan.intersection <- intersect(taxa_names(dune),taxa_names(pan))
dune_pan.intersection.n <- length(dune_pan.intersection)  

all.intersection <- intersect(channel_dune.intersection, channel_pan.intersection)
all.intersection <- length(all.intersection)

# Now we have the intersections
# make objects for the pairs and subset to just the intersecting taxa
channel_pan <- subset_samples(bac.rel,zone%in%c("channel","pan"))
channel_pan <- prune_taxa(channel_pan.intersection, channel_pan)
otu_table(channel_pan) <- otu_table(channel_pan)/5

channel_dune <- subset_samples(bac.rel,zone%in%c("channel","dune"))
channel_dune <- prune_taxa(channel_dune.intersection, channel_dune)
otu_table(channel_dune) <- otu_table(channel_dune)/5

# create a common colour palette for plotting
# not used now but might be useful later
#allphyla <- unique(c(tax_table(channel_dune)[,"Phylum"],tax_table(channel_pan)[,"Phylum"]))
#allphyla.col <- as.vector(colorRampPalette(brewer.pal(12,"Set3"))(length(allphyla)))
#names(allphyla.col)  <- allphyla

# plot
#channel_pan.shared <- plot_bar(channel_pan,x = "zone",fill = "Phylum") + geom_bar(aes(color=Phylum, fill=Phylum), stat="identity", position="stack") + scale_y_continuous(name="relative abundance (%)") + scale_fill_manual("Phylum", values = allphyla.col) + scale_colour_manual("Phylum", values = allphyla.col)

# work out OTU abundance for those only in soil
bac.rel.earth <- subset_samples(bac.rel,element=="earth")
bac.rel.earth.merged <- merge_samples(bac.rel.earth,rep(T,nsamples(bac.rel.earth)))
otu_table(bac.rel.earth.merged) <- otu_table(bac.rel.earth.merged)/nsamples(bac.rel.earth)
inoc_source_otus <- unique(c(taxa_names(channel),taxa_names(dune)))

soil_only_otus.sum <- round(sum(otu_table(bac.rel.earth.merged)) -  sum(otu_table(bac.rel.earth.merged)[,inoc_source_otus]),1)


```

```{r hyp_analysis2,echo=FALSE, cache=TRUE}
# working with merged objects to reduce processing time
bac.rel.merged <- merge_samples(subset_samples(bac.rel,element!="air"),"zone")
otu_table(bac.rel.merged) <- otu_table(bac.rel.merged)/5
bac.rel.merged.phylum <- tax_glom(bac.rel.merged,"Phylum")

# make dataframes for output later
t <- t(otu_table(bac.rel.merged))
t <- as.data.frame(t)
otus.df <- as.data.frame(cbind(tax_table(bac.rel.merged),t))
otus.order <- taxa_names(bac.rel.merged)[order(taxa_sums(bac.rel.merged),decreasing = T)]
otus.df <- otus.df[otus.order,]
# more info will be added to this DF after DESEQ analysis, then it will be output

# Keep only phyla that are > 1% overall:
# to simplify plot and protect against unequal sampling bias

keep_phyla <- (taxa_sums(bac.rel.merged.phylum)/5)>1
keep_phyla <- as.character(tax_table(bac.rel.merged.phylum)[keep_phyla,"Phylum"])
keep_phyla_index <- tax_table(bac.rel.merged)[,"Phylum"]%in%keep_phyla

bac.rel.merged <- subset_taxa(bac.rel.merged, keep_phyla_index)
sample_data(bac.rel.merged)$zone <- as.factor(sample_names(bac.rel.merged))

airborne <- subset_samples(bac.rel,element=="air")
airborne <- prune_taxa(taxa_sums(airborne)>0,airborne)
waterborne <- subset_samples(bac.rel.merged,zone=="channel")
waterborne <- prune_taxa(taxa_sums(waterborne)>0,waterborne)

airborne_otus <- taxa_names(airborne)[taxa_sums(airborne)>0]
waterborne_otus <- taxa_names(waterborne)[taxa_sums(waterborne)>0]
both_otus <- intersect(airborne_otus,waterborne_otus)

# make an object to be modified for plots
bac.rel.ea <- subset_samples(bac.rel,element!="air")
otu_table(bac.rel.ea) <- otu_table(bac.rel.ea)/5

# keep only the more abundant phyla (for clarity of plots)
# this now done globally so shouldn't be needed here
# bac.rel.ea.p <- subset_taxa(bac.rel.ea, Phylum%in%top.phyla)
bac.rel.ea.p <- bac.rel.ea

# same again but looking at cyanobacterial genera
bac.rel.ea.cyano <- subset_taxa(bac.rel.ea, Phylum == "Cyanobacteria")
# select only the more abundant cyanobacterial genera
bac.rel.ea.cyano <- subset_taxa(bac.rel.ea.cyano, Genus%in%top.cyano.genus)

airborne_otus <- airborne_otus[airborne_otus%in%taxa_names(bac.rel.ea)]
waterborne_otus <- waterborne_otus[waterborne_otus%in%taxa_names(bac.rel.ea)]
both_otus <- both_otus[both_otus%in%taxa_names(bac.rel.ea)]

```

```{r deseq_analysis1, echo=FALSE, message=FALSE, cache=TRUE}

  typedds = phyloseq_to_deseq2(expt.deseq, ~ element)
  typedds = DESeq(typedds, test="Wald",fitType="parametric")
  # DESeq docs:
  #On p-values:
  # By default, independent filtering is performed to select a set of genes for multiple test correction which will optimize the number of adjusted p-values less than a given critical value alpha (by default 0.1). The adjusted p-values for the genes which do not pass the filter threshold are set to NA. 
  
# When using local regression fit, the user should examine plotDispEsts(dds)
# to make sure the fitted line is not sharply curving up or down based on
# the position of individual points.

  #  plotDispEsts(typedds,main="model check")

  res = results(typedds, cooksCutoff = FALSE, alpha = 0.05)
  alpha = 0.05
  sigtab = res

  rel.abundance <- apply(otu_table(bac)[rownames(sigtab), ],FUN = max,MARGIN = 1)
  sigtab = cbind(as(sigtab, "data.frame"), as(tax_table(expt.deseq)[rownames(sigtab), ], "matrix"))
  sigtab$max <- rel.abundance
  
  # record the OTU name
  sigtab$otu <- row.names(sigtab)
  
  ind <- sigtab$padj<0.05&!is.na(sigtab$padj)
  sigtab$otu.orig <- sigtab$otu
  sigtab[ind,"otu2"] <- gsub(pattern = "OTU_",replacement = "",x = sigtab[ind,"otu"])
  sigtab[ind,"otu"] <- paste("*",sigtab[ind,"otu"],"*",sep="")

  sigtab$sig <- ""
  sigtab[ind,"sig"] <- "*"
  
  sd <- as.data.frame(sample_data(expt.deseq))

# manipulations to the sigtab table - for later output as graphics and tables

wanted_cols <- c("Phylum",  "Class",  "Order",	"Family",	"Genus",	"Species","padj","log2FoldChange")
wanted_rows <- (sigtab$padj<0.05) & (!is.na(sigtab$padj))
wanted_otus <- sigtab[wanted_rows,"otu.orig"]

expt.rel.df <- subset_samples(bac.rel.all, zone=="dune flank")
expt.rel.df.merged <- merge_samples(expt.rel.df,group = "element")
otus.rel.df.by.type <- otu_table(expt.rel.df.merged)/5

orbt <- as.data.frame(t(otus.rel.df.by.type))
sigtab.subset <- sigtab[wanted_rows,wanted_cols]
sigtab.subset <- cbind(sigtab.subset[wanted_otus,],orbt[wanted_otus,])
# add a difference column
sigtab.subset$diff <- sigtab.subset$earth - sigtab.subset$air

# Sanity check on the significant OTUs
sane <- sign(sigtab.subset$log2FoldChange)==(sign(sigtab.subset$diff))

# identify the removed OTUs (none)
# sigtab.subset.removed <- sigtab.subset[!sane,]

# remove the OTUs not passing sanity check (none)
# sigtab.subset <- sigtab.subset[sane,]

# order by diff
sigtab.subset <- sigtab.subset[order(sigtab.subset$diff,decreasing = T),] 
# save to file
write.csv(sigtab.subset,"windtunnel_otus.csv")

outcols <- c("Phylum",		"Family",	"Genus","padj","air",	"earth") #"Class",	"Order",
outtab <- sigtab.subset[1:10,outcols]
outtab2 <- sigtab.subset[nrow(sigtab.subset):(nrow(sigtab.subset)-10),outcols]
# there are NAs in outtab2 because of something wrong in working out the mean abundances
# as can be seen by this line
# otu_table(bac)["4397479"]
# otu_table(expt.rel.df.merged)["4397479"]
# there are observations for filter but it comes out as 0 in the table.
# This is just because the OTU "4397479" is present in bac used for deseq analysis, but not in the subset object.

otus_earth <- row.names(sigtab.subset[sigtab.subset$diff>0,])
otus_air <- row.names(sigtab.subset[sigtab.subset$diff<0,])

n_otu_earth <- length(otus_earth)
n_otu_air <- length(otus_air)

# record significant OTUs in otus.df for output file / table
otus.df$sig <- NA
otus.df[otus_earth,"sig"] <- "soil"
otus.df[otus_air,"sig"] <- "air"
```

\section{Introduction}

Hypotheses:\label{Hypotheses}
\begin{enumerate}
\item River flooding is an occasional source of microbes to the claypan, some of which persist as viable biocrusts
\item Microbes mobilised from sand dunes by wind erosion are a constant source of inoculum to the claypan
\item Wind differentially mobilises certain microbes from the dunes
\item Easily mobilised organisms are more likely to become established in the claypan, especially on nebkha
\end{enumerate}

\section{Methods}
\subsection{field site}
The dune runs approximately North-South and the claypan is on the East side (Figure \ref{fig:map}). Soil was sampled from five transects perpendicular to the dune, and water/sediment was sampled from water channels in the surrounding area.

\begin{figure}[p]%
\centering
\caption{Sampling locations}
\subfloat[General area of claypan including surrounding water channels.]{{ \includegraphics[width=15cm]{img/DNA1.png} }}%
\hspace{0.5cm}
\subfloat[Closer view of dune and pan area.]{{\includegraphics[width=15cm]{img/DNA2.png} }}%
\label{fig:map}%
\end{figure}

The following locations were targeted for soil sampling:

- Dune (D). Dune biocrust
- Dune flank (DF). Dune flank biocrust
- Nebkha (N). Biocrust situated on nebkha in the claypan to the East of the dune
- Pan (P). Biocrust platelets in vegetation interspaces further onto the claypan
- Channel (C). Sediment and water from the banks of water channels surrounding the dune/claypan complex

Transects were planned to be approximtely 500 m apart to achieve even spacing along the dune, however some zones to the south were quite heavily impacted by cattle and possibly rabbits. These impacted zones were excluded because the dune flanks did not have suitable areas to run the wind tunnel. The transects are numbered 1-5 starting at the South end. The different sampling areas are expected to have different grain size characteristics.

\subsection{Sampling}
Sampled areas were 10x10 cm and were selected to represent the diversity of biocrust in the area. In areas where differing crust types were noticeable the sampled areas were selected to have both types where applicable (e.g. biological/physical, light/dark). In badly disturbed areas remnants of pre-disturbance crust were selected (2x dune sites).

Water channels were sampled by dragging a 5 ml sterile bijoux along the bank where it meets the water. Approximately 2.5 ml bank sediment and 2.5 ml standing water were collected in each bottle from the same location. 

\subsection{Wind tunnel}
In addition to the natural crust samples a wind tunnel was run on the dune flank zones within 3 m of where the natural sampling was done. The wind tunnel was run on the dune flanks because they are naturally eroded by saltation impact from the dune sand above when it is windy, therefore the potential as an inoculum source to surrounding areas is greater than the higher dune areas. A saltation source was not added to these experiments for the sake of simplicity, as there will be saltation occurring within the wind tunnel anyway. At each site the wind tunnel was run three times (on adjacent areas) for 5 minutes at 10 m/s for 5 minutes. This is a moderate treatment intended to be realistic and to collect only the easily mobilised particles / organisms. 

(Helene collected data to determine the exact run conditions)

\section{Results}
DNA sequencing was performed by the Centre for Genomic Research (CGR) NERC Facility at the University of Liverpool. Raw data, summary statistics, and methods description are available via the CGR LIMS at <http://cgr.liv.ac.uk/illum/LIMS10796_978f1f02f3c84a18/>

\subsection{Descriptive and exploratory analyses}
Blue-green platelet crusts were common but not the dominant surface cover on the claypan. They appear to have once been very widespread but have now been largely eroded away. Underneath them was a similar looking crust that seems to be physically formed, and is more brown. In many places there were nebkha atop the blue-green platelet crusts and the platelets ran up the nebkha slightly, however the crusts on nebkha appeared distinct. Platelet crusts appeared to be more common near nebkha. 

In summary, three distinct surface layers were observed in the claypan:
\begin{enumerate}
\item basal physical crust (not sampled)
\item platelet formation biological crust (sampled: P)
\item nebkha (sampled: N)
\end{enumerate}

```{r chem_out, echo=FALSE,fig.height=3,fig.cap='Soil properties. Source data and additional information about each sample are \\textattachfile[description=Mapping file relating samples and collected data,color=0 0 1]{data/map.txt}{available here (click or double-click to access)}\\label{fig:chem}', warning=FALSE, cache=TRUE}
# melt for plotting
chemistry.melt <- melt(sample_data(bac.earthair),measure.vars=c("N_pc","C_pc","pH"))

# make the plot 
print(ggplot(chemistry.melt, aes(x=zone, y=value, shape=zone, colour=zone)) + stat_summary(fun.data = "mean_se", size=1) + facet_wrap(~variable,scales = "free_y",nrow=1) + labs(x = "zone") + theme(aspect.ratio=1,legend.position="none") )

```

DNA was successfully extracted and sequenced from `r nsamples(bac)` samples, yielding `r round(m_reads_total,1)` million reads. After quality control and taxonomic assignment `r round(m_reads_bac_qc,1)` million bacterial reads and `r round(m_reads_arc_qc,1)` million archaeal reads remained. Please note that this is a preliminary analysis only and results might change slightly after additional QC and bioinformatics adjustments.

There were `r ntaxa(bac)` bacterial OTUs and `r ntaxa(arc)` archael OTUs detected (OTUs defined by 97 % similarity which is approximately species-level difference between OTUs). This analysis was planned to target bacteria, and as archaea make up only `r round(100*m_reads_arc_qc / m_reads_bac_qc,1)` \% of sequences, bacteria are the main focus of this analysis.

A total of `r length(unique(tax_table(bac)[,"Phylum"]))` bacterial phyla and `r length(unique(tax_table(arc)[,"Phylum"]))` archael phyla were identified. Frequencies of the most abundant phyla are shown in Figure \ref{fig:phyla}. It can be seen that water channel sampels are dissimilar to the soil-based samples at the phylum level although all abundant phyla are represented in both soil and water. Some phyla differed in abundance between the dune flank soil and airborne dust liberated from the same soil by wind tunnel treatment, for example cyanobacteria (more abundant in soil) and firmicutes (more abundant in dust). This result is explored further in Figure \ref{fig:cyano} which shows the frequency of the most abundant cyanobaterial genera in each sample. 


```{r phyla_out, echo=FALSE, warning=FALSE, cache=TRUE,fig.cap='Relative abundance of bacterial and archael phyla in soil, water and airborne dust lifted by wind tunnel treatment. Bacterial and archael phyla abundances were calculated individually per kingdom. A summary table of phylum abundance by sample type is \\textattachfile[description=Bacterial phylum abundance in biocrusts and water channels in Diamantina National Park Australia,color=0 0 1]{phyla.csv}{available here (click or double-click to access)} \\label{fig:phyla}'}
print(phyla.combined)
```

Phormidium was by far the most abundant cyanobacterial genus (Figure \ref{fig:cyano}), in common with our previous findings from the Kalahari in southern Botswana [@elliott2014niche]. Phormidium and Microcoleus species appeared to be more common in soil compared to airborne dust, whereas Leptoglybna species appeared more common in the soil compared to dust. Nodularia species and Synechococcus species appeared to be more common in water channels compared to soils, whilst the other genera mentioned were more common in soils. 

```{r cyano_out, echo=FALSE, fig.height=3,warning=FALSE, cache=TRUE,fig.cap='Relative abundance of cyanobacterial genera in soil, water and airborne dust lifted by wind tunnel treatment. Abundances are relative wo the whole bacterial community.\\label{fig:cyano}'}
print(cyano.plot)
```

The Chao1 species richness estimation (Figure \ref{fig:rich}) indicated in excess of 10,000 species present in each sample, without striking differences with respect to sample type. These estimates may reduce after additional quality control measures are taken into account.

```{r rich_out, echo=FALSE,fig.width=4,fig.height=3,fig.cap='Chao1 richness estimate (all prokaryotes). \\label{fig:rich}', cache=TRUE}

print(rich.ab)
```

Microbial community structure was compared with respect to sample source using nonmetric multidimensional scaling (NMDS). Initial results showed that the water samples differed extremely compared to all other samples (see also Figure \ref{fig:phyla}), and this dominated all other inter-sample differences. The water samples were therefore removed and the community stucutre relationships between the soil and airborne dust samples were revealed as shown in Figure \ref{fig:nmds}. The dune community differs from the dune flank community, and the airborne dust community derived from the dune flank differs from the dune flank community. Notably the dust community has a community structure intermediate between the dune / dune flank and nebkha communities.

```{r nmds_out, echo=FALSE,fig.height=6,fig.cap='NMDS visualisation of prokaryotic community structure in soil (sand dunes, claypan, and nebkha) and airborne dust (from sand dunes). \\label{fig:nmds}', cache=TRUE}

print(NMDS.ab)

```

```{r source_analysis1, echo=FALSE, message=FALSE, cache=TRUE}

colnames(tax_table(bac.rel.ea.p))[1] <- "source"
tax_table(bac.rel.ea.p)[,"source"] <- "not found in water channel"
tax_table(bac.rel.ea.p)[taxa_names(bac.rel.ea.p)%in%waterborne_otus,"source"] <- "found in water channel"

water_otus_plot <- plot_bar(bac.rel.ea.p,x = "zone",fill = "source")
water_otus_plot <- water_otus_plot + geom_bar(aes(color=source, fill=source), stat="identity", position="stack") 
water_otus_plot <- water_otus_plot + scale_y_continuous(name="relative abundance (%)") + scale_fill_brewer(palette = "Set1",direction = -1) + scale_color_brewer(palette = "Set1",direction = -1)
water_otus_plot <- water_otus_plot + facet_wrap(~Phylum, nrow=2) + theme(legend.position="bottom")

###

colnames(tax_table(bac.rel.ea.cyano))[1] <- "source"
tax_table(bac.rel.ea.cyano)[,"source"] <- "not found in water channel"
tax_table(bac.rel.ea.cyano)[taxa_names(bac.rel.ea.cyano)%in%waterborne_otus,"source"] <- "found in water channel"

water_otus_plot.c <- plot_bar(bac.rel.ea.cyano,x = "zone",fill = "source")
water_otus_plot.c <- water_otus_plot.c + geom_bar(aes(color=source, fill=source), stat="identity", position="stack") 
water_otus_plot.c <- water_otus_plot.c + scale_y_continuous(name="relative abundance (%)") + scale_fill_brewer(palette = "Set1",direction = -1) + scale_color_brewer(palette = "Set1",direction = -1)
water_otus_plot.c <- water_otus_plot.c + facet_wrap(~Genus, nrow=1) + theme(legend.position="bottom")

```

```{r deseq_analysis2, echo=FALSE, message=FALSE, cache=TRUE}
# see whether OTU is easy or not to become airborne by wind tunnel treatment
# this is used later to make the wind_otus_plot
# using object bac.rel.ea.p - which has only the more abundant phyla

colnames(tax_table(bac.rel.ea.p))[1] <- "source"
tax_table(bac.rel.ea.p)[,"source"] <- "intermediate"
tax_table(bac.rel.ea.p)[row.names(tax_table(bac.rel.ea.p))%in%otus_air,"source"] <- "easily becomes airborne"
tax_table(bac.rel.ea.p)[row.names(tax_table(bac.rel.ea.p))%in%otus_earth,"source"] <- "resists becoming airborne"

colnames(tax_table(bac.rel.ea.cyano))[1] <- "source"
tax_table(bac.rel.ea.cyano)[,"source"] <- "intermediate"
tax_table(bac.rel.ea.cyano)[row.names(tax_table(bac.rel.ea.cyano))%in%otus_air,"source"] <- "easily becomes airborne"
tax_table(bac.rel.ea.cyano)[row.names(tax_table(bac.rel.ea.cyano))%in%otus_earth,"source"] <- "resists becoming airborne"

deseq_n_test <- nrow(sigtab)

deseq_sum_diff_air <- sum(sigtab.subset[sigtab.subset$diff<0,"diff"] )
deseq_sum_diff_earth <- sum(sigtab.subset[sigtab.subset$diff>0,"diff"] )
deseq_n_diff_air <- length(sigtab.subset[sigtab.subset$diff<0,"diff"] )
deseq_n_diff_earth <- length(sigtab.subset[sigtab.subset$diff>0,"diff"] )

```

```{r source_analysis2, echo=FALSE, message=FALSE, cache=TRUE}
###

wind_otus_plot <- plot_bar(bac.rel.ea.p,x = "zone",fill = "source")
wind_otus_plot <- wind_otus_plot + geom_bar(aes(color=source, fill=source), stat="identity", position="stack") 
wind_otus_plot <- wind_otus_plot + scale_y_continuous(name="relative abundance (%)") + scale_fill_brewer(palette = "Accent") + scale_color_brewer(palette = "Accent")
wind_otus_plot <- wind_otus_plot + facet_wrap(~Phylum, nrow=2) + theme(legend.position="bottom")

###

wind_otus_plot.c <- plot_bar(bac.rel.ea.cyano,x = "zone",fill = "source")
wind_otus_plot.c <- wind_otus_plot.c + geom_bar(aes(color=source, fill=source), stat="identity", position="stack") 
wind_otus_plot.c <- wind_otus_plot.c + scale_y_continuous(name="relative abundance (%)") + scale_fill_brewer(palette = "Accent") + scale_color_brewer(palette = "Accent")
wind_otus_plot.c <- wind_otus_plot.c + facet_wrap(~Genus, nrow=1) + theme(legend.position="bottom")
```

```{r OTU_source_out, echo=FALSE,fig.height=11,fig.cap='Relative abundance of bacterial phyla. Source indicates whether individual OTUs (97 % similarity) contributing to the phylum abundance were found in the water channels (top), and whether they were significantly more abundant in soil or wind tunnel collectors at the dune flank sites (bottom). Only the 10 most abundant phyla are shown for clarity. Source designation is not indicative of the ultimate origin of OTUs.\\label{fig:waterp}', cache=FALSE}

grid.newpage()
pushViewport(viewport(layout=g0))
print(water_otus_plot,vp=vp.01)
print(wind_otus_plot,vp=vp.02)

#print(water_otus_plot)
#print(wind_otus_plot )
```

Of the `r pan.ntaxa` OTUs found in the claypan, `r channel_pan.intersection.n` were also found in water channels, `r dune_pan.intersection.n` were found in airborne dust from the dune flank, and `r all.intersection` were found in both of these hypothesised inoculum sources. The dune and channel samples had `r channel_dune.intersection.n` OTUs in common.

```{r OTU_source_out2, echo=FALSE,fig.height=8,fig.cap='Relative abundance of cyanobacterial genera. Source indicates whether individual OTUs (97 % similarity) contributing to the genus abundance were found in the water channels (top), and whether they were significantly more abundant in soil or wind tunnel collectors at the dune flank sites (bottom). Unidentified OTUs were excluded. Only the 5 most abundant identified cyanobacterial genera are shown for clarity. Source designation is not indicative of the ultimate origin of OTUs.\\label{fig:waterc}', cache=FALSE}

grid.newpage()
pushViewport(viewport(layout=g0))
print(water_otus_plot.c,vp=vp.01)
print(wind_otus_plot.c,vp=vp.02)

#print(water_otus_plot.c  )
#print(wind_otus_plot.c)
```

Whilst many soil OTUs were not detected in the air or water samples, most of the OTUs found exclusively in soil were rare, collectively accounting for only `r soil_only_otus.sum ` % of reads in the soil samples. A breakdown of OTU co-occurrence in all samples summarised by phylum read frequency is shown in Figure \ref{fig:waterp}, with detail on cyanobacterial genera given in Figure \ref{fig:waterc}. It can be seen that co-occurrence of bacteria between soils and hypothesised inoculum sources is very common (assessed by number of reads\footnote{The number of reads is assumed to be indicative of the abundance of the organism to which the DNA sequence belongs.} not number of OTUs), and the relative abundance of organisms from different inoculum sources varies with respect to soil type and phylum. 

A model based approach was used to check if any OTUs had significantly different abundance between the dune flank soil and the collected dust derived from the same soil obtained by wind tunnel treatment. This technique avoids problems associated with uneven sampling effort and includes correction for multiple testing. The results (p<0.05) indicated that `r deseq_n_diff_air` OTUs (`r round(deseq_sum_diff_air*-1)` % of reads) were  more abundant in air samples and `r deseq_n_diff_earth` OTUs (`r round(deseq_sum_diff_earth)` % of reads) were more abundant in soil samples. Details for a subset of these OTUs are shown in Tables \ref{tab:deseq1} and \ref{tab:deseq2}.

```{r deseq_out1, echo=FALSE, cache=FALSE}
kable(outtab, digits=2,caption = "OTUs which are significantly more abundant in soil compared to dust blown from the soil. This is a subset of the DESEQ2 analysis output, sorted by magnitude of difference in abundance. Full details of significant results from this analysis are \\textattachfile[description=DESEQ 2 analysis of differential abundance,color=0 0 1]{windtunnel_otus.csv}{available here (click or double-click to access)} \\label{tab:deseq1}")

kable(outtab2, digits=2,caption = "OTUs which are significantly more abundant in dust blown from the soil compared to the whole soil. This is a subset of the DESEQ2 analysis output, sorted by magnitude of difference in abundance. Full details of significant results from this analysis are available via the file linked in the legend to Table \\ref{tab:deseq1} (these two tables show the top and bottom 10 lines from the same file) \\label{tab:deseq2}")

otus.df.out <- otus.df[1:10,c(rank_names(bac)[c(2,5,6)],sample_names(bac.rel.merged),"sig")]
colnames(otus.df.out)[colnames(otus.df.out)=="dune flank"] <- "d.flank"
# class(otus.df$dune)
# otus.df.out <- as.numeric(otus.df.out)
kable(otus.df.out,row.names = F, digits=2,caption = "OTU abundance in each sampling zone, in order of overall abundance. Sig column indicates if the OTU was significantly more abundant in soil or wind tunnel filter after running the wind tunnel on dune flanks. The 10 most abundant OTUs overall are shown, full table with all OTUs is \\textattachfile[description=DESEQ 2 analysis of differential abundance,color=0 0 1]{all_otus.csv}{available here (click or double-click to access)} \\label{tab:deseq3}")

write.csv(otus.df[ ,c(rank_names(bac)[1:7],sample_names(bac.rel.merged))],"all_otus.csv")
```


```{r otus_out}
```


\section{Discussion}
A few previous studies have been published on the microbial content of airborne sediments, and there was recently a review on the subject by @acosta2015microbiology. This review pointed out the fact that most previous work has focused on long-distance transport and the movement of pathogens, which are quite different subjects compared to the present work here. The only study in @acosta2015microbiology combining use of a wind tunnel with collection of airborne particles and their microbial charaterisation is @gardner2012pyrosequencing, which used a pyrosequencing approach. Prior to wind-tunnel treatments the soils in @gardner2012pyrosequencing were tilled, raked, and flattened using a lawn roller which may have considerably affected the results. The main focus of that paper was determining the association of particular microbes or diversity metrics with organic matter and particle size fractions.

\subsection{Microbes in dunes, claypan, and nebkha}
The microbial community structure of the soil surface varied markedly in relation to soil type (e.g. Figure \ref{fig:nmds}. 

\subsection{Dispersal and ecology of biocrust microbes}
Due to the dune being at higher elevation than all other locations, it might be expected to harbour fewer OTUs of aquatic origin, however this does not seem to be the case. In fact our data are not informative with respect to the direction of dispersal. This question could be tackled in future work monitoring microbial populations over time and relating this to weather conditions and occurrence of flood events. We expect that co-occurrence between dune and river channel may be driven principally by wind dispersal (direction dune>river), whereas co-occurrence between river channel and claypan may be driven principally by flood events (direction river>claypan). The higher proportion of DNA sequence reads shared between dune and river compared to river and pan, may be explained by the fact that flooding of the pan is a rare event (last occurring x years previously) but mobilisation of dust from sand dunes is occurring continuously..

\section{References}
\bibliography{references}