%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Supporting Information
%% (Optional)
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% OVERVIEW
%
% Please note that all supporting information will be peer reviewed with your manuscript.
% In general, the purpose of the supporting information is to enable
% authors to provide and archive auxiliary information such as data
% tables, method information, figures, video, or computer software,
% in digital formats so that other scientists can use it.

% The key criteria are that the data:
% 1. supplement the main scientific conclusions of the paper but are not essential to the conclusions (with the exception of
%    including data so the experiment can be reproducible);
% 2. are likely to be usable or used by other scientists working in the field;
% 3. are described with sufficient precision that other scientists can understand them, and
% 4. are not exe files.
%

% All Supporting text and figures should be included in this document.

% Data sets, large tables, movie files,
% and audio files should be uploaded separately, following AGU naming
% conventions. Include their captions in this document and list the
% file name with the caption. You will be prompted to upload these
% files on the Upload Files tab during the submission process, using
% file type “Supporting Information (SI)”

\documentclass[draft]{agujournal}

% Please type in the journal name: \journalname{<Journal Name>}
% ie,
\journalname{JGR-Biogeosciences}


\begin{document}

%% This command needs article title as argument to \supportinginfo{}:
\supportinginfo{Surface Stability in Drylands is Influenced by Dispersal Strategy of Soil Bacteria}


\authors{David R. Elliott\affil{1},
Andrew D. Thomas\affil{2}, Craig L. Strong\affil{3},
and Joanna Bullard\affil{4}}

\affiliation{1}{Environmental Sustainability Research Centre, University of Derby, Derby, UK.}
\affiliation{2}{Department of Geography and Earth Sciences, Aberystwyth University, Aberystwyth, UK.}
\affiliation{3}{The Fenner School of Environment \& Society, ANU College of Sciences, The Australian National University, Canberra   ACT 2601,  Australia.}
\affiliation{4}{Department of Geography, School of Social, Political and Geographical Sciences, Loughborough University, Leicestershire, UK.}

%% Corresponding Author
%(include name and email addresses of the corresponding author.  More
%than one corresponding author is allowed in this Word file and for
%publication; but only one corresponding author is allowed in our
%editorial system.)

\correspondingauthor{D. R. Elliott}{d.r.elliott@derby.ac.uk}

%% ------------------------------------------------------------------------ %%
%
%  TEXT
%
%% ------------------------------------------------------------------------ %%

\section*{Contents}
%%%Remove or add items as needed%%%
%\begin{enumerate}

%\end{enumerate}

\section*{Additional Supporting Information (Files uploaded separately)}

\begin{enumerate}
\item Caption for Figures S1 and S2
\item Captions for large Tables S1 to S4
\end{enumerate}

\section*{Figure S1} % Elliott_Diamantina_Microbiome_Figure_S1[nmds].pdf
NMDS visualisation of prokaryotic microbial community structure (beta diversity) of soil (sand dunes, claypan, and nebkha), water channels, and airborne dust (from sand dunes). Due to extreme difference of the channel sample community structure compared to the soils, these samples were excluded from a similar figure shown in the main manuscript.

\section*{Figure S2} % Elliott_Diamantina_Microbiome_Figure_S2[RDA].pdf
Redundancy analysis used to identify variables explaining microbial community structure via stepwise model building approach. The only significant variable as determined by ANOVA (p$<$0.05) was particle mass median diameter (d50) as indicated by arrow on the plot.

\section*{Large Table S1.} % Elliott_Diamantina_Microbiome_Table_S1[sample-metadata].txt
Sample metadata including GPS locations and soil properties for all samples.

\section*{Large Table S2.} % Elliott_Diamantina_Microbiome_Table_S2[phyla].csv
Relative abundance of all prokaryotic phyla. 

\section*{Large Table S3.} % Elliott_Diamantina_Microbiome_Table_S3[cyanobacteria].csv
Relative abundance of cyanobacteria

\section*{Large Table S4.} % Elliott_Diamantina_Microbiome_Table_S4[otus].csv
OTU taxonomy and relative abundance in each sampling zone (\%). This is an extended version of tables 1 and 2 presented in the main manuscript manuscript. Sig column indicates if the OTU was significantly more abundant in soil or wind tunnel filter after running the wind tunnel on dune flanks. Columns to the right of this show the statistical results from DESEQ2 analysis which are the basis for the interpretation of differential mobilisation of OTUs by wind tunnel treatment. OTUs with an overall relative abundance less than 0.2 \% are excluded.

\section*{Data Set S1.} % all.otutab.biom
Contingency table recording the frequency of OTU observations in each sample.

\section*{Data Set S2.} % all.otus.fasta
Fasta file recording the DNA sequence for each OTU.

\section*{Data Set S3.} % all.otus_tax_assignments
Taxonomy assignments for each OTU.

\section*{Method S1} % Elliott_Diamantina_Microbiome_Method_S1[R].R
R script containing a record of the analyses performed. In conjunction with the supplementary data provided this script can be used to re-create all analyses and data figures presented in this research.


\end{document}



